<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Carl Murie" />

<meta name="date" content="2018-12-11" />

<title>GSEA analysis - T5 disease status</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<script src="site_libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">VaheyProject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">GSEA analysis - T5 disease status</h1>
<h4 class="author"><em>Carl Murie</em></h4>
<h4 class="date"><em>December 11, 2018</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-12-11</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<p><details> <summary> <strong style="color:red;">✖</strong> <strong>R Markdown file:</strong> uncommitted changes </summary> The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20181008)</code> </summary></p>
<p>The command <code>set.seed(20181008)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> 1ed83ed </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    data/.Rhistory

Untracked files:
    Untracked:  VaheyReport.tar.gz
    Untracked:  analysis/DEG_T0.Rmd
    Untracked:  analysis/DEG_T0_Interaction.Rmd
    Untracked:  analysis/DEG_T1.Rmd
    Untracked:  analysis/DEG_T1_Interaction.Rmd
    Untracked:  analysis/GSEA_T0.Rmd
    Untracked:  analysis/GSEA_T0_Interaction.Rmd
    Untracked:  analysis/GSEA_T1.Rmd
    Untracked:  analysis/GSEA_T1_Interaction.Rmd
    Untracked:  analysis/analysis_T5_disease.Rmd
    Untracked:  analysis/pamr.Rmd
    Untracked:  code/processGEOFiles.R
    Untracked:  data/BTM_for_GSEA_20131008.gmt
    Untracked:  data/GPL570.soft
    Untracked:  data/GPL571.soft
    Untracked:  data/GSE18323-GPL570_series_matrix.txt.gz
    Untracked:  data/GSE18323-GPL571_series_matrix.txt.gz
    Untracked:  data/gpl_eset.Rds
    Untracked:  data/h.all.v5.0.symbols.gmt
    Untracked:  docs/figure/DEG_T0.Rmd/
    Untracked:  docs/figure/DEG_T0_Interaction.Rmd/
    Untracked:  docs/figure/DEG_T1.Rmd/
    Untracked:  docs/figure/DEG_T1_Interaction.Rmd/
    Untracked:  docs/figure/pamr.Rmd/
    Untracked:  output/dmso_M3_both_disease.csv
    Untracked:  output/dmso_M3_old_disease.csv
    Untracked:  output/dmso_M3_young_disease.csv
    Untracked:  output/pam_res.Rda

Unstaged changes:
    Modified:   analysis/analysis_PCA.Rmd
    Modified:   analysis/experimental_design.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/processVahey.Rmd

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes. </details>
</li>
</ul>
<hr />
<div id="analysis" class="section level1">
<h1>Analysis</h1>
<p>The Blood Transcript Modules (BTM) from Shuzhao et al. Molecular signatures of antibody responses derived from a systems biological study of 5 human vaccines, 2014 were used for the GSEA analysis.</p>
<p>Apply camera to samples in T5 to test for differentially expressed gene-sets for Protection - NonProtection.</p>
</div>
<div id="sample-sizes" class="section level1">
<h1>Sample sizes</h1>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px;">
disease
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
total
</th>
<th style="text-align:center;">
NonProtected
</th>
<th style="text-align:center;">
Protected
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
T5
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
13
</td>
</tr>
</tbody>
</table>
</div>
<div id="t5-disease-protection-protected---nonprotected" class="section level1">
<h1>T5: disease protection (Protected - NonProtected)</h1>
<p>linear model: ~disease filtering: T5</p>
<div id="htmlwidget-9c23e70c978f4c72e429" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9c23e70c978f4c72e429">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117"],["cell cycle (I) (M4.1)","mitotic cell division (M6)","cell division (stimulated CD4+ T cells) (M46)","TBA (M233)","TBA (M205)","E2F transcription factor network (M8)","TBA (M26.1)","cell cycle (II) (M4.10)","enriched in NK cells (KIR cluster) (M61.1)","NK cell surface signature (S1)","mitotic cell cycle - DNA replication (M4.4)","enriched in NK cells (I) (M7.2)","C-MYC transcriptional network (M4.12)","TBA (M26.0)","leukocyte migration (M88.0)","PLK1 signaling events (M4.2)","TBA (M72.0)","enriched in NK cells (II) (M61.0)","TBA (M148)","TBA (M137)","TBA (M214)","mitotic cell cycle in stimulated CD4 T cells (M4.5)","TBA (M184.0)","TBA (M244)","TBA (M184.1)","TBA (M153)","cell cycle (III) (M103)","putative targets of PAX3 (M89.0)","cell division in stimulated CD4 T cells (M4.6)","TBA (M26.2)","TBA (M174)","Ran mediated mitosis (M15)","extracellular matrix (II) (M2.1)","intracellular transport (M147)","TBA (M180)","phosphatidylinositol signaling system (M101)","TBA (M185)","TBA (M72.1)","enriched in NK cells (receptor activation) (M61.2)","TBA (M131)","amino acid metabolishm and transport (M154.0)","leukocyte differentiation (M160)","golgi membrane (II) (M237)","TBA (M141)","MHC-TLR7-TLR8 cluster (M146)","TBA (M70.0)","muscle contraction, SRF targets (M195)","TBA (M72.2)","extracellular matrix (I) (M2.0)","enriched for ubiquitination (M138)","interferon alpha response (I) (M158.0)","TBA (M125)","extracellular matrix (III) (M2.2)","inositol phosphate metabolism (M129)","signaling in T cells (I) (M35.0)","signaling in T cells (II) (M35.1)","TBA (source: naive B cells) (M152.1)","enriched in antigen presentation (II) (M95.0)","innate antiviral response (M150)","cell cell adhesion (M133.1)","putative targets of PAX3 (M89.1)","cell cycle and transcription (M4.0)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","cell adhesion (M51)","regulation of transcription, transcription factors (M213)","TBA (M128)","signal transduction, plasma membrane (M82)","chaperonin mediated protein folding (I) (M204.0)","complement and other receptors in DCs (M40)","plasma cells, immunoglobulins (M156.1)","CD1 and other DC receptors (M50)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","TBA (M203)","extracellular matrix, collagen (M210)","TBA (M194)","myeloid cell cytokines, metallopeptidases and laminins (M78)","TBA (M108)","integrin cell surface interactions (II) (M1.1)","cytoskeleton/actin (SRF transcription targets) (M145.1)","transmembrane and ion transporters (I) (M142)","enriched in antigen presentation (I) (M71)","chromosome Y linked (M240)","TBA (M242)","extracellular matrix, complement (M140)","DNA repair (M76)","extracellular region cluster (GO) (M189)","integrin cell surface interactions (I) (M1.0)","mitotic cell cycle in stimulated CD4 T cells (M4.9)","TBA (M177.0)","mitotic cell cycle (M4.7)","innate activation by cytosolic DNA sensing (M13)","TBA (M190)","E2F1 targets (Q4) (M10.1)","RA, WNT, CSF receptors network (monocyte) (M23)","enriched in plasma membrane proteins (I) (M135.0)","regulation of antigen presentation and immune response (M5.0)","transcription regulation in cell development (M49)","TBA (M90)","cytoskeletal remodeling (enriched for SRF targets) (M34)","mitotic cell cycle in stimulated CD4 T cells (M4.11)","heme biosynthesis (I) (M171)","TBA (M70.1)","enriched for TF motif PAX3 (M179)","type I interferon response (M127)","enriched in membrane proteins (M124)","transmembrane and ion transporters (II) (M224)","blood coagulation (M11.1)","BCR signaling (M54)","TBA (M48)","TBA (M66)","antiviral IFN signature (M75)","suppression of MAPK signaling (M56)","TBA (M220)","enriched in monocytes (IV) (M118.0)","E2F1 targets (Q3) (M10.0)","cell division - E2F transcription network (M4.8)","TBA (M120)"],[127,25,24,12,11,12,18,14,13,41,26,39,11,26,45,31,21,23,11,15,12,23,14,7,11,12,51,15,13,10,21,13,44,16,11,13,14,16,14,11,23,15,13,23,16,17,8,12,30,8,16,10,14,12,14,10,8,20,11,10,10,286,15,35,14,11,12,12,12,11,10,10,10,28,13,11,8,12,13,9,16,9,10,13,21,15,29,13,21,20,11,10,19,11,12,73,45,11,10,10,10,9,9,11,16,11,19,12,12,16,20,11,10,44,29,18,10],["Up","Up","Up","Up","Down","Up","Up","Up","Down","Down","Up","Down","Up","Up","Up","Up","Down","Down","Down","Down","Down","Up","Up","Up","Up","Down","Up","Down","Up","Up","Down","Up","Up","Down","Down","Down","Down","Down","Down","Up","Up","Down","Down","Up","Down","Up","Up","Down","Up","Down","Up","Up","Up","Down","Down","Down","Down","Down","Down","Up","Down","Up","Down","Up","Down","Down","Up","Down","Down","Up","Down","Down","Down","Up","Down","Up","Up","Up","Up","Up","Down","Down","Down","Up","Up","Up","Up","Up","Down","Up","Down","Down","Up","Down","Up","Down","Up","Up","Up","Up","Up","Up","Down","Down","Up","Up","Down","Down","Down","Down","Down","Down","Up","Down","Up","Up","Down"],[3.54815577809484e-11,5.35374647998738e-07,1.32769304134197e-05,2.12879083985861e-05,2.5582012266867e-05,2.91918982874889e-05,6.08942375511187e-05,8.64764742873817e-05,9.52833367892979e-05,0.000100365464722413,0.000149840133967355,0.00018739038361007,0.000195727217747734,0.000216436061758314,0.000236119238319287,0.000246941039132547,0.000255144452352932,0.000269175853494265,0.000308902507572443,0.000322789992908415,0.000446088453223236,0.000577172746578618,0.000624300155892217,0.000698484949734868,0.0007244965619752,0.000783759894972173,0.000825991664307424,0.000941601778385938,0.00106254528776385,0.00140710938115997,0.00158755175102861,0.00169149535995363,0.00175561710600329,0.00208010344460012,0.00243454278045373,0.0027861546903525,0.00292295017296332,0.002959487118071,0.00326384961830961,0.0033140867981518,0.00372752468939898,0.0037926942849603,0.00384311734479534,0.00417964040020068,0.00430303765934879,0.00454698123555797,0.00504080766936886,0.00524515872557147,0.00524713642421494,0.00604771423455775,0.00666127145312489,0.00682525755019159,0.00785931411027826,0.00812657582657564,0.00857987933498508,0.00941276484836044,0.0104449459899453,0.0106361739128613,0.012287900666475,0.0133773440980939,0.0135273978200815,0.0136002613729107,0.0139916337272969,0.0144611274691235,0.0145754430921202,0.0156460334347326,0.0167759582074773,0.0179023100535947,0.0179203183944657,0.0202596893061788,0.0203809419575942,0.0204855357201201,0.0219492956881883,0.0219968660077283,0.0222702078607405,0.0224341341390232,0.0228891161800491,0.0238975635841796,0.0257762377346399,0.0271100885052395,0.0284607636638656,0.0289576677760092,0.0292123648489474,0.0304522858492057,0.0314878374250986,0.0322300145968566,0.03490220244675,0.035117249510138,0.036302846064235,0.036352681055729,0.0364096264330572,0.0369837353469974,0.0380302749002015,0.0382203679828313,0.0384892738994656,0.0385725694772024,0.038784955048319,0.0390456977693232,0.0408122821523283,0.0409298593084396,0.0409478424852079,0.0423063196980606,0.0423116406234786,0.0442163544845835,0.0460127072785625,0.0501912972023115,0.052633250567481,0.0531498771576182,0.0539460029754637,0.0554529233469045,0.0556212103697101,0.0574662361946541,0.0589621337025658,0.0618132923809185,0.066025295339956,0.0664442862804435,0.066948691436053],[1.20637296455224e-08,9.10136901597855e-05,0.00150471878018756,0.00165420756962437,0.00165420756962437,0.00165420756962437,0.00295772010962576,0.00341242580056204,0.00341242580056204,0.00341242580056204,0.00463142232262734,0.00508443278822501,0.00508443278822501,0.00508443278822501,0.00508443278822501,0.00508443278822501,0.00508443278822501,0.00508443278822501,0.00548742987944306,0.00548742987944306,0.00722238448075715,0.00891994244712409,0.00922878491318929,0.00985315324286272,0.00985315324286272,0.0102491678573284,0.0104013765135009,0.0114337358804007,0.012457427511714,0.0159472396531463,0.0174118579145074,0.0179721381995073,0.0180881762436703,0.0208010344460012,0.0236498441529791,0.0263136831866625,0.0264796215827406,0.0264796215827406,0.0281697377842903,0.0281697377842903,0.0303874394704748,0.0303874394704748,0.0303874394704748,0.032297221274278,0.0325118400928575,0.0336081221758632,0.0364087017190424,0.0364087017190424,0.0364087017190424,0.0411244567949927,0.044408476354166,0.0446266839820219,0.0504182414621624,0.0511673292784392,0.0530392540708168,0.0571489294364741,0.0623031866066911,0.0623499850064281,0.0708116309593474,0.074582078496607,0.074582078496607,0.074582078496607,0.0755104042425547,0.0762407792510905,0.0762407792510905,0.0806007783001378,0.0851317282170489,0.0883030181756283,0.0883030181756283,0.0967372520116785,0.0967372520116785,0.0967372520116785,0.100363231674578,0.100363231674578,0.100363231674578,0.100363231674578,0.101068824691126,0.104168866905398,0.110935706706045,0.115217876147268,0.119464933897707,0.119665109019785,0.119665109019785,0.123259252246785,0.125951349700395,0.127420987941061,0.135464665730305,0.135464665730305,0.135464665730305,0.135464665730305,0.135464665730305,0.135464665730305,0.135464665730305,0.135464665730305,0.135464665730305,0.135464665730305,0.135464665730305,0.135464665730305,0.137844222227432,0.137844222227432,0.137844222227432,0.139669493320221,0.139669493320221,0.144553466584215,0.148993528330583,0.160990953290433,0.167245842924706,0.167323687348057,0.168271935886768,0.170371275006319,0.170371275006319,0.174451074162343,0.177408189901525,0.184355433416774,0.194551752891094,0.194551752891094,0.194551752891094]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="compare-significant-gene-sets-to-mal067-assay-for-m3-dmso-disease-effects." class="section level1">
<h1>Compare significant gene-sets to mal067 assay for M3, DMSO disease effects.</h1>
<p>Overlapping gene-sets between Vahey significant gene-sets and mal067 significant gene-sets. FDR cutoff &lt;= 0.2</p>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<caption>
both
</caption>
<thead>
<tr>
<th style="text-align:center;">
name
</th>
<th style="text-align:center;">
value
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<caption>
young
</caption>
<thead>
<tr>
<th style="text-align:center;">
name
</th>
<th style="text-align:center;">
value
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
1
</td>
<td style="text-align:center;">
extracellular matrix (II) (M2.1)
</td>
</tr>
<tr>
<td style="text-align:center;">
2
</td>
<td style="text-align:center;">
TBA (M131)
</td>
</tr>
<tr>
<td style="text-align:center;">
3
</td>
<td style="text-align:center;">
MHC-TLR7-TLR8 cluster (M146)
</td>
</tr>
<tr>
<td style="text-align:center;">
4
</td>
<td style="text-align:center;">
cell adhesion (M51)
</td>
</tr>
<tr>
<td style="text-align:center;">
5
</td>
<td style="text-align:center;">
CD1 and other DC receptors (M50)
</td>
</tr>
<tr>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
RA, WNT, CSF receptors network (monocyte) (M23)
</td>
</tr>
<tr>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
TBA (M66)
</td>
</tr>
<tr>
<td style="text-align:center;">
8
</td>
<td style="text-align:center;">
suppression of MAPK signaling (M56)
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<caption>
old
</caption>
<thead>
<tr>
<th style="text-align:center;">
name
</th>
<th style="text-align:center;">
value
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.5 LTS

Matrix products: default
BLAS/LAPACK: /app/easybuild/software/OpenBLAS/0.2.18-GCC-5.4.0-2.26-LAPACK-3.6.1/lib/libopenblas_prescottp-r0.2.18.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] magrittr_1.5         DT_0.4               limma_3.36.1        
 [4] RNASeqUtilities_1.1  here_0.1             data.table_1.11.8   
 [7] forcats_0.3.0        stringr_1.3.1        dplyr_0.7.99.9000   
[10] purrr_0.2.5          readr_1.1.1          tidyr_0.8.0         
[13] tibble_1.4.99.9006   ggplot2_3.1.0        tidyverse_1.2.1     
[16] GSEABase_1.42.0      graph_1.58.0         annotate_1.58.0     
[19] XML_3.98-1.11        AnnotationDbi_1.42.1 IRanges_2.14.6      
[22] S4Vectors_0.18.1     Biobase_2.40.0       BiocGenerics_0.26.0 
[25] knitr_1.20          

loaded via a namespace (and not attached):
 [1] nlme_3.1-137      bitops_1.0-6      lubridate_1.7.4  
 [4] bit64_0.9-7       httr_1.3.1        rprojroot_1.3-2  
 [7] tools_3.5.0       backports_1.1.2   R6_2.3.0         
[10] DBI_1.0.0         lazyeval_0.2.1    colorspace_1.3-2 
[13] withr_2.1.2       tidyselect_0.2.5  gridExtra_2.3    
[16] bit_1.1-14        compiler_3.5.0    git2r_0.23.0     
[19] cli_1.0.1         rvest_0.3.2       xml2_1.2.0       
[22] scales_1.0.0      digest_0.6.18     rmarkdown_1.10   
[25] R.utils_2.7.0     pkgconfig_2.0.2   htmltools_0.3.6  
[28] highr_0.7         htmlwidgets_1.2   rlang_0.3.0.9000 
[31] readxl_1.1.0      rstudioapi_0.8    RSQLite_2.1.1    
[34] shiny_1.0.5       jsonlite_1.5      crosstalk_1.0.0  
[37] R.oo_1.22.0       RCurl_1.95-4.10   kableExtra_0.9.0 
[40] Rcpp_1.0.0        munsell_0.5.0     R.methodsS3_1.7.1
[43] stringi_1.2.4     whisker_0.3-2     yaml_2.2.0       
[46] plyr_1.8.4        grid_3.5.0        blob_1.1.1       
[49] promises_1.0.1    crayon_1.3.4      lattice_0.20-35  
[52] haven_1.1.2       hms_0.4.2         pillar_1.3.0.9001
[55] glue_1.3.0        evaluate_0.12     modelr_0.1.1     
[58] httpuv_1.4.2      cellranger_1.1.0  gtable_0.2.0     
[61] assertthat_0.2.0  mime_0.6          xtable_1.8-2     
[64] broom_0.5.0       later_0.7.2       viridisLite_0.3.0
[67] memoise_1.1.0     workflowr_1.1.1  </code></pre>
</div>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
