<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Carl Murie" />

<meta name="date" content="2019-04-18" />

<title>GSEA analysis - day of vaccination (T1) vs all other later time periods</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<script src="site_libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.5/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">VaheyProject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">GSEA analysis - day of vaccination (T1) vs all other later time periods</h1>
<h3 class="subtitle">time effect</h3>
<h4 class="author">Carl Murie</h4>
<h4 class="date">April 18, 2019</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-04-18
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>VaheyProject/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.3.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20181008code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20181008)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20181008code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20181008)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong1ed83ed"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 1ed83ed </a>
</p>
</div>
<div id="strongRepositoryversionstrong1ed83ed" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    data/.Rhistory

Untracked files:
    Untracked:  VaheyReport.tar.gz
    Untracked:  analysis/DEG_T0.Rmd
    Untracked:  analysis/DEG_T0_Interaction.Rmd
    Untracked:  analysis/DEG_T1.Rmd
    Untracked:  analysis/DEG_T1_Interaction.Rmd
    Untracked:  analysis/GSEA_T0.Rmd
    Untracked:  analysis/GSEA_T0_Interaction.Rmd
    Untracked:  analysis/GSEA_T1.Rmd
    Untracked:  analysis/GSEA_T1_Interaction.Rmd
    Untracked:  analysis/analysis_T4_disease.Rmd
    Untracked:  analysis/analysis_T5_disease.Rmd
    Untracked:  analysis/pamr.Rmd
    Untracked:  code/processGEOFiles.R
    Untracked:  data/BTM_for_GSEA_20131008.gmt
    Untracked:  data/GPL570.soft
    Untracked:  data/GPL571.soft
    Untracked:  data/GSE18323-GPL570_series_matrix.txt.gz
    Untracked:  data/GSE18323-GPL571_series_matrix.txt.gz
    Untracked:  data/gpl_eset.Rds
    Untracked:  data/h.all.v5.0.symbols.gmt
    Untracked:  docs/figure/DEG_T0.Rmd/
    Untracked:  docs/figure/DEG_T0_Interaction.Rmd/
    Untracked:  docs/figure/DEG_T1.Rmd/
    Untracked:  docs/figure/DEG_T1_Interaction.Rmd/
    Untracked:  docs/figure/GSEA_T0.Rmd/
    Untracked:  docs/figure/GSEA_T0_Interaction.Rmd/
    Untracked:  docs/figure/pamr.Rmd/
    Untracked:  output/dmso_M3_both_disease.csv
    Untracked:  output/dmso_M3_old_disease.csv
    Untracked:  output/dmso_M3_young_disease.csv
    Untracked:  output/pam_res.Rda

Unstaged changes:
    Modified:   analysis/analysis_PCA.Rmd
    Modified:   analysis/experimental_design.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/processVahey.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with <code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="gsea-analysis" class="section level1">
<h1>GSEA Analysis</h1>
<p>The Blood Transcript Modules (BTM) from Shuzhao et al. Molecular signatures of antibody responses derived from a systems biological study of 5 human vaccines, 2014 were used for the GSEA analysis.</p>
<div id="day-of-vaccine-t1-vs-visits-t2-to-t5-time-effect" class="section level2">
<h2>Day of vaccine (T1) vs visits T2 to T5: time effect</h2>
<ul>
<li><strong>visit</strong>
<ul>
<li>T0 - study entry (baseline)</li>
<li>T1 - day of third vaccination</li>
<li>T2 - 24h after T1</li>
<li>T3 - 72h after T1</li>
<li>T4 - 2 weeks after T1</li>
<li>T5 - 5 days after challenge</li>
</ul></li>
</ul>
<p>A FDR cutoff of &lt;= 0.2 was used.</p>
<p>The formula used in the camera function is: ~visit</p>
<div id="sample-size" class="section level3">
<h3>Sample size</h3>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
disease
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
total
</th>
<th style="text-align:center;">
NonProtected
</th>
<th style="text-align:center;">
Protected
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
T0
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
25
</td>
<td style="text-align:center;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
T1
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
25
</td>
<td style="text-align:center;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
T2
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
T3
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
T4
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
T5
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="comparisons" class="section level2 tabset tabset-fade tabset-pills">
<h2>Comparisons</h2>
<div id="t1-versus-t2-t2---t1" class="section level3">
<h3>T1 versus T2 (T2 - T1)</h3>
<div id="htmlwidget-9c23e70c978f4c72e429" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9c23e70c978f4c72e429">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130"],["enriched in monocytes (II) (M11.0)","TLR and inflammatory signaling (M16)","enriched in T cells (I) (M7.0)","enriched in activated dendritic cells (II) (M165)","enriched in NK cells (I) (M7.2)","Monocyte surface signature (S4)","T cell activation (I) (M7.1)","innate antiviral response (M150)","antiviral IFN signature (M75)","myeloid cell enriched receptors and transporters (M4.3)","type I interferon response (M127)","TBA (M66)","enriched in monocytes (IV) (M118.0)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","T cell activation (III) (M7.4)","enriched in activated dendritic cells/monocytes (M64)","formyl peptide receptor mediated neutrophil response (M11.2)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","enriched in NK cells (II) (M61.0)","cell cycle and transcription (M4.0)","activated dendritic cells (M67)","T cell differentiation (M14)","cell junction (GO) (M4.13)","T cell differentiation via ITK and PKC (M18)","enriched in NK cells (receptor activation) (M61.2)","T cell differentiation (Th2) (M19)","enriched in monocytes (III) (M73)","T cell surface signature (S0)","RIG-1 like receptor signaling (M68)","T cell activation (II) (M7.3)","blood coagulation (M11.1)","IL2, IL7, TCR network (M65)","enriched in neutrophils (I) (M37.1)","T cell activation and signaling (M5.1)","lysosomal/endosomal proteins (M139)","enriched in monocytes (I) (M4.15)","translation initiation factor 3 complex (M245)","enriched in neutrophils (II) (M163)","recruitment of neutrophils (M132)","regulation of signal transduction (M3)","enriched in activated dendritic cells (I) (M119)","TBA (M48)","lysosome (M209)","transmembrane transport (I) (M87)","MHC-TLR7-TLR8 cluster (M146)","enriched in antigen presentation (III) (M95.1)","enriched in T cells (II) (M223)","NK cell surface signature (S1)","enriched in NK cells (KIR cluster) (M61.1)","Activated (LPS) dendritic cell surface signature (S11)","TBA (M198)","complement and other receptors in DCs (M40)","RA, WNT, CSF receptors network (monocyte) (M23)","inflammatory response (M33)","enriched in antigen presentation (II) (M95.0)","endoplasmic reticulum (M37.2)","innate activation by cytosolic DNA sensing (M13)","immune activation - generic cluster (M37.0)","complement activation (I) (M112.0)","antigen processing and presentation (M200)","regulation of antigen presentation and immune response (M5.0)","DC surface signature (S5)","inflammasome receptors and signaling (M53)","B cell development (M9)","Resting dendritic cell surface signature (S10)","TBA (M55)","suppression of MAPK signaling (M56)","TBA (M79)","enriched in NK cells (III) (M157)","cell cycle (III) (M103)","CD4 T cell surface signature Th2-stimulated (S7)","double positive thymocytes (M126)","TLR8-BAFF network (M25)","enriched in G-protein coupled receptors (M130)","enriched in B cells (I) (M47.0)","cell adhesion (GO) (M117)","proteasome (M226)","CCR1, 7 and cell signaling (M59)","enriched in myeloid cells and monocytes (M81)","enriched in antigen presentation (I) (M71)","TBA (M211)","platelet activation (III) (M42)","plasma cells &amp; B cells, immunoglobulins (M156.0)","cell activation (IL15, IL23, TNF) (M24)","proinflammatory cytokines and chemokines (M29)","chemokines and inflammatory molecules in myeloid cells (M86.0)","enriched in monocytes (surface) (M118.1)","G protein coupled receptors cluster (M155)","enriched in B cells (VI) (M69)","T cell signaling and costimulation (M44)","mitotic cell cycle - DNA replication (M4.4)","E2F1 targets (Q3) (M10.0)","golgi membrane (II) (M237)","extracellular matrix (II) (M2.1)","CD28 costimulation (M12)","proinflammatory dendritic cell, myeloid cell response (M86.1)","TBA (M151)","TBA (M214)","mitotic cell cycle in stimulated CD4 T cells (M4.5)","nucleotide metabolism (M181)","AP-1 transcription factor network (M20)","enriched in B cells (II) (M47.1)","cell cycle, mitotic phase (M230)","TBA (M180)","immuregulation - monocytes, T and B cells (M57)","enriched for TF motif PAX3 (M179)","extracellular matrix, complement (M140)","mismatch repair (I) (M22.0)","T &amp; B cell development, activation (M62.0)","E2F1 targets (Q4) (M10.1)","TBA (M136)","lymphocyte generic cluster (M60)","transcription regulation in cell development (M49)","TBA (M192)","TBA (M148)","cell cycle (I) (M4.1)","enriched in dendritic cells (M168)","TBA (M141)","cell cycle and growth arrest (M31)","enriched in B cells (V) (M47.4)","leukocyte activation and migration (M45)","plasma cells, immunoglobulins (M156.1)","TBA (M236)","CORO1A-DEF6 network (I) (M32.2)","regulation of localization (GO) (M63)","enriched for promoter motif NATCACGTGAY (putative SREBF1 targets) (M178)","T cell surface, activation (M36)","myeloid, dendritic cell activation via NFkB (I) (M43.0)","chemokine cluster (II) (M27.1)","TBA (M203)"],[173,43,51,32,39,65,43,11,20,23,11,16,44,10,13,15,9,15,23,286,11,11,10,11,14,16,11,22,9,26,19,11,39,21,10,10,9,12,9,40,10,12,7,24,16,12,9,41,13,27,10,12,11,9,20,15,11,266,16,6,73,65,10,11,61,11,11,8,12,51,10,10,8,10,37,21,10,10,34,16,10,10,24,14,9,18,9,9,17,9,26,29,13,44,8,12,8,12,23,9,14,28,8,11,12,9,13,25,30,19,16,15,45,12,11,127,13,23,11,8,11,11,10,19,12,9,12,13,15,10],["Up","Up","Down","Up","Down","Up","Down","Up","Up","Up","Up","Up","Up","Up","Down","Up","Up","Up","Down","Up","Up","Down","Up","Down","Down","Down","Up","Down","Up","Down","Up","Down","Up","Down","Up","Up","Down","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Down","Down","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Down","Down","Down","Up","Down","Down","Down","Up","Up","Up","Up","Down","Up","Down","Up","Up","Up","Up","Up","Down","Down","Down","Down","Up","Up","Down","Up","Down","Up","Down","Down","Up","Down","Down","Up","Down","Down","Up","Down","Down","Down","Up","Down","Up","Up","Up","Down","Up","Up","Up","Down","Down","Down","Up","Down","Up","Down","Up","Up","Up","Down"],[2.00173049446853e-53,7.36612383959768e-37,3.02342815453221e-29,3.14318671374941e-23,5.78999132249046e-22,2.00345502563248e-21,4.17203445517886e-21,1.74454082238203e-17,3.17825183351251e-17,4.34069661236131e-17,7.05532827002858e-17,2.23108166969293e-16,6.80657674627945e-16,4.49559129365836e-15,2.78669660510076e-14,5.30157012053549e-14,6.47124888231432e-14,2.47908398125101e-13,4.69764931185676e-13,7.06769722556514e-13,9.61285331984085e-13,7.68950324558416e-12,2.04678640622259e-11,5.2105929164123e-11,1.07650110917982e-10,1.07783666695087e-10,2.0704666175365e-10,2.22519340384699e-10,8.08566748282877e-10,1.11848694281879e-09,1.92626932881004e-09,2.00955371068594e-09,3.03089669250258e-09,8.22700953117996e-09,1.63464593594862e-08,2.03153275201015e-08,3.37126865318657e-08,4.73110116716309e-08,5.10335082692756e-08,7.65378946313675e-08,9.48663783497728e-08,1.21854674402976e-07,2.7863483007433e-07,4.70655690672326e-07,6.15460496581525e-07,6.91979430245024e-07,7.47038088554451e-07,9.48689409517796e-07,1.32661210532376e-06,1.50183021740702e-06,1.78052393211222e-06,2.87209914427955e-06,3.45488375114302e-06,4.97031696296241e-06,5.40824056036744e-06,1.22188590084794e-05,1.2534392693811e-05,1.49213861646102e-05,1.64676443793668e-05,2.05942007098917e-05,2.30102816156057e-05,2.51585472874429e-05,2.72075748682555e-05,3.04419000694072e-05,3.12980829701204e-05,3.88986562433606e-05,8.3435407099984e-05,0.000112960461329088,0.000113556418360199,0.000129994399926803,0.000149378251540056,0.000150802933776727,0.000168944734815521,0.000205740550126009,0.000207161460113547,0.000215818884667798,0.000257333182845259,0.000269417988436477,0.00028246131334305,0.000444423395071474,0.000492674082636005,0.000602250368769615,0.000698997905572823,0.000707970759069706,0.00087886563767194,0.000985095368953803,0.00105087472289215,0.00106304900302329,0.00180900147310056,0.00198693100782933,0.00245469209426034,0.00266763184900141,0.00317736344722407,0.00318346839661849,0.00426297577872011,0.00438217195068017,0.00613640099712567,0.00784720813057697,0.0110481830722146,0.012097157307677,0.0131506414246083,0.0143321273370782,0.0151089669307323,0.0152222983146047,0.0171518230781958,0.0173196425245112,0.0182806370268831,0.0202262608252163,0.0206800857934047,0.0213120394850865,0.0251096402338439,0.0282051309535735,0.0289919225039373,0.0312425618772086,0.0312597143920861,0.0350608382814455,0.039235812911594,0.0411873790372682,0.041840270790827,0.0425068246755934,0.0443108318126547,0.0456837100950704,0.0506818385187683,0.0543084405603412,0.0590319272573961,0.0626069678118015,0.0632597318970997,0.0658806662945966,0.0672507907693049,0.071518362777288],[6.80588368119301e-51,1.25224105273161e-34,3.42655190846984e-27,2.671708706687e-21,3.93719409929351e-20,1.13529118119174e-19,2.02641673537259e-19,7.41429849512361e-16,1.20067291488251e-15,1.47583684820284e-15,2.18073782891793e-15,6.32139806412995e-15,1.78018161056539e-14,1.09178645703132e-13,6.31651230489506e-13,1.12658365061379e-12,1.29424977646286e-12,4.68271418680746e-12,8.40631982121736e-12,1.20150852834607e-11,1.55636672797423e-11,1.18837777431755e-10,3.02568425267688e-10,7.38167329825076e-10,1.40947871832037e-09,1.40947871832037e-09,2.60725425912004e-09,2.7020205618142e-09,9.47974808331649e-09,1.26761853519463e-08,2.11268248966262e-08,2.13515081760381e-08,3.12274204682084e-08,8.22700953117996e-08,1.58794176635009e-07,1.91866982134292e-07,3.09792254617144e-07,4.23309051798803e-07,4.44907507988557e-07,6.50572104366624e-07,7.86696796071287e-07,9.86442602309804e-07,2.20315912151796e-06,3.63688488246797e-06,4.65014597417152e-06,5.11463057137626e-06,5.40410532145773e-06,6.71988331741772e-06,9.20506358796076e-06,1.02124454783677e-05,1.18701595474148e-05,1.87791097895202e-05,2.21634051960118e-05,3.12945882853189e-05,3.3432759827726e-05,7.41859296943391e-05,7.47665529104517e-05,8.747019475806e-05,9.48982896438085e-05,0.000116700470689386,0.000128254028677146,0.000137966227060171,0.000146834531035029,0.000161722594118726,0.000163713049382168,0.000200387017011251,0.000423403558417829,0.000559553365832863,0.000559553365832863,0.000631401371073042,0.000712124965056766,0.000712124965056766,0.000786865888181879,0.000939131952514747,0.000939131952514747,0.000965505536671729,0.00113627639178426,0.00117438610344105,0.00121565628527388,0.00188879942905376,0.00206801466785484,0.00249713567538621,0.0028633649143947,0.00286559592956786,0.00351546255068776,0.00389456308656155,0.00410686673314173,0.00410723478440816,0.0069107921444291,0.00750618380735524,0.0091713770554782,0.00985863944196172,0.0115146729239392,0.0115146729239392,0.015256965944893,0.0155201923253256,0.0215090344229147,0.0272250077999609,0.0379432549954844,0.0411303348461019,0.04426948598383,0.0477737577902608,0.0497652060285152,0.0497652060285152,0.0555392366341578,0.0555535703616395,0.0580880055059836,0.0636752655608662,0.0645066896308037,0.0658735765902673,0.0769124115270895,0.0856227189662052,0.0872323331976875,0.0924200251592109,0.0924200251592109,0.10276452599734,0.114018601623435,0.118675498920942,0.119543630830934,0.120436003247515,0.124509775341344,0.127315257641999,0.140096138994969,0.148910240246097,0.160566842140117,0.16893943695248,0.169356762559165,0.174995519845022,0.177250146213672,0.187048025725215]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t1-versus-t3-t3---t1" class="section level3">
<h3>T1 versus T3 (T3 - T1)</h3>
<div id="htmlwidget-a0e6efde7f4990d8641e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-a0e6efde7f4990d8641e">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145"],["cell cycle and transcription (M4.0)","enriched in B cells (I) (M47.0)","enriched in monocytes (II) (M11.0)","cell cycle (I) (M4.1)","myeloid cell enriched receptors and transporters (M4.3)","putative targets of PAX3 (M89.0)","enriched in B cells (VI) (M69)","TLR and inflammatory signaling (M16)","TBA (M236)","TBA (M66)","enriched in B cells (II) (M47.1)","plasma cells &amp; B cells, immunoglobulins (M156.0)","TBA (M177.1)","mitotic cell cycle in stimulated CD4 T cells (M4.9)","TBA (M70.0)","Monocyte surface signature (S4)","putative targets of PAX3 (M89.1)","cell cycle (II) (M4.10)","complement and other receptors in DCs (M40)","CCR1, 7 and cell signaling (M59)","mitotic cell cycle in stimulated CD4 T cells (M4.5)","mitotic cell cycle (M4.7)","growth factor induced, enriched in nuclear receptor subfamily 4 (M94)","TBA (M194)","phosphatidylinositol signaling system (M101)","PLK1 signaling events (M4.2)","proinflammatory cytokines and chemokines (M29)","cell division (stimulated CD4+ T cells) (M46)","MHC-TLR7-TLR8 cluster (M146)","golgi membrane (II) (M237)","blood coagulation (M11.1)","enriched in antigen presentation (II) (M95.0)","TBA (M153)","TBA (M125)","TBA (M184.1)","enriched in monocytes (IV) (M118.0)","cell cycle and growth arrest (M31)","TBA (M184.0)","DNA repair (M76)","mitosis (TF motif CCAATNNSNNNGCG) (M169)","respiratory electron transport chain (mitochondrion) (M231)","formyl peptide receptor mediated neutrophil response (M11.2)","enriched in naive and memory B cells (M83)","mismatch repair (I) (M22.0)","TBA (M214)","NK cell surface signature (S1)","inflammatory response (M33)","E2F1 targets (Q3) (M10.0)","lysosomal/endosomal proteins (M139)","B cell surface signature (S2)","receptors, cell migration (M109)","TBA (M177.0)","AP-1 transcription factor network (M20)","Resting dendritic cell surface signature (S10)","cell junction (GO) (M4.13)","mitotic cell cycle - DNA replication (M4.4)","TBA (M218)","enriched in monocytes (I) (M4.15)","enriched in neutrophils (I) (M37.1)","enriched in activated dendritic cells (II) (M165)","enriched in NK cells (III) (M157)","E2F transcription factor network (M8)","enriched in neutrophils (II) (M163)","cell division in stimulated CD4 T cells (M4.6)","TBA (M233)","TBA (M186)","nuclear pore complex (mitosis) (M106.1)","regulation of antigen presentation and immune response (M5.0)","lysosome (M209)","Hox cluster I (M17.0)","TBA (M244)","enriched in B cell differentiation (M123)","respiratory electron transport chain (mitochondrion) (M219)","immuregulation - monocytes, T and B cells (M57)","cell cycle, ATP binding (M144)","TBA (M70.1)","extracellular matrix (I) (M2.0)","plasma cells, immunoglobulins (M156.1)","TBA (M180)","TBA (M48)","proteasome (M226)","transcription regulation in cell development (M49)","signaling in T cells (II) (M35.1)","regulation of signal transduction (M3)","RA, WNT, CSF receptors network (monocyte) (M23)","Hox cluster VI (M107)","BCR signaling (M54)","TBA (M26.0)","antiviral IFN signature (M75)","T cell activation (III) (M7.4)","activated dendritic cells (M67)","leukocyte migration (M88.0)","cell division (M37.3)","G protein coupled receptors cluster (M155)","enriched in hepatocyte nuclear factors (I) (M88.1)","cell movement, Adhesion &amp; Platelet activation (M30)","TBA (M131)","inositol phosphate metabolism (M129)","cell division - E2F transcription network (M4.8)","innate antiviral response (M150)","enriched in B cells (III) (M47.2)","enriched in monocytes (III) (M73)","TBA (M128)","TBA (M72.0)","TBA (M41.0)","enriched in antigen presentation (III) (M95.1)","enriched in G-protein coupled receptors (M130)","antigen processing and presentation (M200)","Naive B cell surface signature (S8)","respiratory electron transport chain (mitochondrion) (M216)","TBA (M198)","TBA (M102)","lymphocyte generic cluster (M60)","intracellular transport (M147)","cell cycle (III) (M103)","regulation of transcription, transcription factors (M213)","B cell development (M9)","proinflammatory dendritic cell, myeloid cell response (M86.1)","small GTPase mediated signal transduction (M215)","enriched in membrane proteins (M124)","enriched for TF motif PAX3 (M179)","enriched in NK cells (I) (M7.2)","mitotic cell cycle in stimulated CD4 T cells (M4.11)","enriched in extracellular matrix &amp; associated proteins (M202)","platelet activation (III) (M42)","TBA (M201)","TBA (M41.2)","complement activation (II) (M112.1)","enriched in activated dendritic cells (I) (M119)","Hox cluster III (M17.2)","muscle contraction, SRF targets (M195)","translation initiation (M227)","Rho GTPase cycle (M4.14)","enriched for ubiquitination (M138)","TBA (M72.2)","TBA (M90)","enriched in plasma membrane proteins (I) (M135.0)","endoplasmic reticulum (M37.2)","glycerophospholipid metabolism (M114.1)","TBA (M203)","enriched in plasma membrane proteins (II) (M135.1)","regulation of localization (GO) (M63)","TBA (M141)","Hox cluster IV (M17.3)","respiratory electron transport chain (mitochondrion) (M238)"],[286,37,173,127,23,15,17,43,10,16,28,24,8,13,17,65,10,14,12,10,23,20,12,13,13,31,9,24,16,13,19,20,12,10,11,44,11,14,21,14,11,9,10,25,12,41,9,29,10,137,14,21,14,61,10,26,11,10,39,32,12,12,12,13,12,9,11,73,7,10,7,8,17,12,15,9,30,11,11,12,10,45,10,40,11,9,12,26,20,13,11,45,10,9,11,17,11,12,18,11,23,11,11,21,15,12,10,6,38,11,10,11,15,16,51,14,11,12,11,16,9,39,10,11,10,12,12,10,10,10,8,10,8,8,12,11,12,15,8,10,11,12,23,8,12],["Up","Down","Up","Up","Up","Down","Down","Up","Up","Up","Down","Down","Up","Up","Down","Up","Down","Up","Up","Up","Up","Up","Down","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Down","Up","Down","Down","Up","Up","Up","Up","Down","Up","Up","Up","Up","Up","Up","Down","Up","Up","Down","Up","Up","Up","Up","Up","Down","Up","Up","Up","Up","Up","Down","Down","Up","Up","Up","Down","Down","Down","Up","Down","Up","Down","Down","Down","Up","Up","Up","Up","Up","Up","Up","Down","Up","Down","Up","Down","Up","Down","Up","Up","Down","Down","Down","Up","Up","Up","Down","Up","Up","Up","Down","Up","Up","Up","Down","Up","Up","Down","Down","Up","Up","Up","Up","Up","Up","Down","Up","Up","Up","Down","Up","Down","Down","Down","Up","Down","Down","Up","Up","Up","Up","Down","Down","Up","Up","Up","Down","Up","Down","Down","Up"],[8.92784770304458e-14,4.75925482265341e-12,6.81250201752707e-12,5.06818942902097e-09,1.46929450603778e-08,2.63532918277402e-08,2.701936315295e-08,6.73300466258347e-07,7.21533122668516e-07,1.35623265825999e-06,2.18790691300436e-06,2.21523298622586e-06,2.37440935529082e-06,2.83782992873167e-06,5.60239926946748e-06,6.11604645460154e-06,1.40396888987136e-05,1.74307992014517e-05,1.80266688613886e-05,3.18284312001283e-05,4.5640390642336e-05,4.59810728946325e-05,5.71529414294348e-05,6.42756282231388e-05,6.87665852648214e-05,7.81350325737991e-05,9.33114486650463e-05,0.00010237454711972,0.000107277727423608,0.000126771928482919,0.000132081453999522,0.000149749499806641,0.000184667516651284,0.000253104078411932,0.000401399745213933,0.000422994652573566,0.000551674861028475,0.000832627885835508,0.000911646105325684,0.00117004487913254,0.00117203358951925,0.00125317668263538,0.0013468589382807,0.00141610711485331,0.00145043346807902,0.0014709969414304,0.00159278745889396,0.00195157971163725,0.00207175469968386,0.00208041989644613,0.002193424636671,0.00230912767936207,0.0024558853862568,0.00278600041493494,0.00295200930950111,0.00352256148616661,0.00358824367313518,0.00385544210522258,0.00417795144400366,0.00429093396660014,0.0053952121745475,0.00561416963751585,0.00591578725843911,0.00615138017532806,0.00640270314431724,0.00641722304632972,0.00674334349089072,0.00717702286107322,0.00723681375870584,0.00862435266812258,0.00869790333333998,0.00952921068512642,0.00981591271582654,0.0099771302042562,0.0105738065598531,0.0106150933643758,0.0107027219889199,0.0107237079141025,0.0113288574666581,0.0113563514976423,0.0120097334496139,0.0124669137137436,0.0126830355272119,0.0127872748686258,0.0128775760403107,0.0134549182207591,0.0135168139069138,0.0137986537120641,0.0164265642882784,0.0166963198637816,0.0176727119802678,0.0198366332510454,0.0202087499027176,0.0205324262935778,0.0207525786581009,0.0219293436626845,0.0234845116202598,0.0246431488983075,0.0248163484981192,0.02623863530774,0.0272245419703309,0.0277264589393487,0.0278726486806054,0.0279268266241688,0.0281325814271795,0.0291751683751796,0.0293871776680925,0.0294114046191998,0.0298977085519922,0.0369608825932629,0.0374006741412072,0.0384256762550347,0.0389894188031695,0.0392181839448287,0.0398631388306316,0.0401443691820386,0.0403246450816748,0.041619065254997,0.0435801174408595,0.0436312145946946,0.0437836439851011,0.0464943278002379,0.0480504409509352,0.0483001757970385,0.0496991656465293,0.0505781409008493,0.0506304924703891,0.0518923083706846,0.0561439381794901,0.0567780315887641,0.0587026281073826,0.0613605570223736,0.0621117375883597,0.0623892724226704,0.0636025619514738,0.0648495178391296,0.0660222054731179,0.0668208790016572,0.0693024124404199,0.0709775247940969,0.0735050235849968,0.0743055106525301,0.080634400117553,0.0807035643107231,0.0850251883857084],[3.03546821903516e-11,7.72083561986402e-10,7.72083561986402e-10,4.30796101466782e-07,9.9912026410569e-07,1.312369067429e-06,1.312369067429e-06,2.72579179674773e-05,2.72579179674773e-05,4.61119103808396e-05,6.20999369845291e-05,6.20999369845291e-05,6.20999369845291e-05,6.89187268406263e-05,0.000126987716774596,0.000129965987160283,0.000280793777974272,0.000322582495414322,0.000322582495414322,0.000541083330402181,0.000710616581098866,0.000710616581098866,0.000844869568956863,0.0009105713998278,0.000935225559601572,0.00102176581058045,0.00117503305726355,0.00124311950073946,0.0012577388732423,0.00143674852280641,0.00144863530193024,0.00159108843544556,0.00190263502004354,0.00253104078411932,0.00389931181064963,0.00399494949652812,0.00506944466891031,0.00744982845221244,0.00794768399514699,0.00971930293747674,0.00971930293747674,0.010144763621334,0.0106495823026846,0.0108725860888334,0.0108725860888334,0.0108725860888334,0.0115222922558287,0.0138236896240972,0.0141468552958337,0.0141468552958337,0.01462283091114,0.0150981425189058,0.015754736440138,0.0175414840940348,0.0182487848223705,0.0213869804517259,0.0214035587520344,0.0226008675133737,0.0240763303552753,0.0243152924774008,0.0300716744155107,0.0307873818831514,0.0319264709185603,0.0326792071814303,0.0330584217538198,0.0330584217538198,0.034219952043326,0.0356596619994201,0.0356596619994201,0.0416519314554309,0.0416519314554309,0.0449990504575414,0.0457179496353565,0.045840868506042,0.0467443678307033,0.0467443678307033,0.0467443678307033,0.0467443678307033,0.0482644938649797,0.0482644938649797,0.0504112268255398,0.0515103041612427,0.0515103041612427,0.0515103041612427,0.0515103041612427,0.0528243302109274,0.0528243302109274,0.0533129802511567,0.0627531669439851,0.0630749861520638,0.0660299128933081,0.0733092967973415,0.0738814512572472,0.0742662227640048,0.074272386776361,0.0776664254720076,0.0823168448545188,0.0852278635288941,0.0852278635288941,0.0892113600463161,0.0910959779546765,0.0910959779546765,0.0910959779546765,0.0910959779546765,0.0910959779546765,0.0925914589863698,0.0925914589863698,0.0925914589863698,0.093258907409884,0.11424272801554,0.114560623495589,0.116649374345641,0.116966513519665,0.116966513519665,0.117182729297175,0.117182729297175,0.117182729297175,0.119919340565245,0.12302842111516,0.12302842111516,0.12302842111516,0.129574356164597,0.132435965895105,0.132435965895105,0.13518173055856,0.135546200314428,0.135546200314428,0.137838944109631,0.147976271170749,0.148496390309075,0.15235796608023,0.158049919603083,0.158301138982895,0.158301138982895,0.160184230100008,0.162123794597824,0.163850728911387,0.164631151163503,0.169516692300308,0.172373988785664,0.177246156162404,0.177914602970847,0.190550082400319,0.190550082400319,0.199369407249247]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t1-versus-t4-t4---t1" class="section level3">
<h3>T1 versus T4 (T4 - T1)</h3>
<div id="htmlwidget-c47734cf8f874295dd41" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-c47734cf8f874295dd41">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46"],["TBA (M70.0)","TBA (M125)","TBA (M70.1)","TBA (M186)","TBA (M184.0)","TBA (M184.1)","intracellular transport (M147)","TBA (M148)","TBA (M214)","phosphatidylinositol signaling system (M101)","TBA (M153)","TBA (M137)","golgi membrane (II) (M237)","TBA (M174)","TBA (M244)","E2F1 targets (Q3) (M10.0)","TBA (M72.1)","TBA (M233)","TBA (M72.2)","inositol phosphate metabolism (M129)","regulation of transcription, transcription factors (M213)","TBA (M180)","TBA (M72.0)","TBA (M128)","nuclear pore, transport; mRNA splicing, processing (M143)","TBA (M203)","interferon alpha response (I) (M158.0)","DNA repair (M76)","cell division (M37.3)","nuclear pore complex (M106.0)","mitosis (TF motif CCAATNNSNNNGCG) (M169)","complement activation (II) (M112.1)","enriched in activated dendritic cells (II) (M165)","cell cycle, ATP binding (M144)","TBA (M229)","translation initiation factor 3 complex (M245)","nuclear pore complex (mitosis) (M106.1)","enriched in DNA interacting proteins (M182)","E2F1 targets (Q4) (M10.1)","Plasma cell surface signature (S3)","mismatch repair (I) (M22.0)","TBA (M243)","cell junction (M162.1)","cell division (stimulated CD4+ T cells) (M46)","cell division - E2F transcription network (M4.8)","TBA (M98.0)"],[17,10,9,9,14,11,16,11,12,13,12,15,13,21,7,29,16,12,12,12,14,11,21,11,11,10,16,21,10,16,14,10,32,15,7,9,11,12,19,16,25,9,11,24,18,14],["Down","Down","Down","Down","Down","Down","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Up","Down","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Up","Up","Up","Down","Up","Up","Up","Down","Up","Up","Up","Down","Up","Up","Down","Up","Up","Down"],[3.46974069875726e-19,6.40793912531405e-18,2.06749849148325e-11,5.40496265566476e-08,1.72823314617449e-07,4.98757648685535e-07,1.52214222680576e-05,1.56941197173014e-05,6.15530336424707e-05,0.000118589548708732,0.000129150681419403,0.000183501539855338,0.000241616968223644,0.000649750901647389,0.000709630822513761,0.000803742782937691,0.00114582581024022,0.0011793464114778,0.0011916667360567,0.00133917108640901,0.00141884343046585,0.00142750306006153,0.00167177684290963,0.00171991267248452,0.00180019799930048,0.00220378908788923,0.00242728393176771,0.00460704047997842,0.00503445707220828,0.00535387079930734,0.00547412061184835,0.00565356203674223,0.00592224340830559,0.00622764944676213,0.00685303166712337,0.00714687859185051,0.00885700796549312,0.0090197971131902,0.0120946563937752,0.0182642950037527,0.0204521319455041,0.0216793039994686,0.0234463106798807,0.0246267945796204,0.0262131988073376,0.0268252021326595],[1.17971183757747e-16,1.08934965130339e-15,2.34316495701434e-09,4.59421825731505e-06,1.17519853939866e-05,2.82629334255136e-05,0.000667000087985308,0.000667000087985308,0.00232533682649334,0.00399193015296338,0.00399193015296338,0.00519921029590124,0.00631921301507991,0.0157796647542937,0.0160849653103119,0.0170795341374259,0.0213245626452251,0.0213245626452251,0.0213245626452251,0.0220614109282236,0.0220614109282236,0.0220614109282236,0.0243654295268641,0.0243654295268641,0.0244826927904865,0.0288187803800899,0.0305657976592971,0.0559426343997379,0.0590246691224419,0.0600387421944658,0.0600387421944658,0.0600690966403862,0.0610170532976939,0.0622764944676213,0.0665723076234842,0.0674982978119215,0.0807034478548597,0.0807034478548597,0.105440594202143,0.155246507531898,0.169603045401741,0.175499127614746,0.185389433282778,0.190297958115249,0.198055279877662,0.198273233154439]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t1-versus-t5-t5---t1" class="section level3">
<h3>T1 versus T5 (T5 - T1)</h3>
<p>There is a confound between visits T5 and the other visit times. The T5 visit used the HGU133A 2.0 chip while the other visits used the HGU133 Plus 2.0 chip. Boxplots of normalized scores and MDS plots show a clear separation between the samples for the two chips (see Quality Control section).</p>
<div id="htmlwidget-14032f3f90232640cb54" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-14032f3f90232640cb54">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210"],["B cell surface signature (S2)","respiratory electron transport chain (mitochondrion) (M219)","respiratory electron transport chain (mitochondrion) (M231)","interferon alpha response (I) (M158.0)","enriched in antigen presentation (II) (M95.0)","regulation of antigen presentation and immune response (M5.0)","enriched in monocytes (II) (M11.0)","TBA (M174)","TBA (M26.1)","TBA (M26.0)","TBA (M72.0)","TBA (M72.1)","TBA (M41.1)","Naive B cell surface signature (S8)","Memory B cell surface signature (S9)","chaperonin mediated protein folding (I) (M204.0)","TBA (source: B cells) (M152.0)","translation initiation factor 3 complex (M245)","intracellular transport (M147)","TBA (M166)","cell cycle, ATP binding (M144)","enriched in T cells (I) (M7.0)","TBA (M41.3)","interferon alpha response (II) (M158.1)","leukocyte migration (M88.0)","TBA (M194)","extracellular matrix, collagen (M210)","TBA (M170)","proteasome (M226)","mitosis (TF motif CCAATNNSNNNGCG) (M169)","TBA (M41.2)","Wnt signaling pathway (M206)","transcription elongation, RNA polymerase II (M234)","respiratory electron transport chain (mitochondrion) (M238)","TBA (M98.0)","nuclear pore complex (mitosis) (M106.1)","TBA (M211)","respiratory electron transport chain (mitochondrion) (M216)","extracellular matrix (I) (M2.0)","enriched in calcium signaling proteins (M239)","TBA (M72.2)","translation initiation (M227)","extracellular matrix (III) (M2.2)","chaperonin mediated protein folding (II) (M204.1)","TBA (M41.0)","cell development (M175)","T cell activation (I) (M7.1)","spliceosome (M250)","purine nucleotide biosynthesis (M212)","T cell activation and signaling (M5.1)","enriched in extracellular matrix &amp; associated proteins (M202)","TBA (M141)","MHC-TLR7-TLR8 cluster (M146)","phosphatidylinositol signaling system (M101)","integrin cell surface interactions (I) (M1.0)","TBA (M26.2)","integrin cell surface interactions (II) (M1.1)","TBA (M197)","muscle contraction, SRF targets (M195)","enriched in antigen presentation (III) (M95.1)","TBA (source: B cells) (M217)","TBA (source: memory B cells) (M152.2)","TBA (M70.0)","ATF targets network (M41.4)","enriched in plasma membrane proteins (I) (M135.0)","TBA (M116)","enriched in NK cells (I) (M7.2)","signal transduction, plasma membrane (M82)","blood coagulation (M11.1)","enriched for TF motif PAX3 (M179)","amino acid metabolishm and transport (M154.0)","TLR and inflammatory signaling (M16)","T cell differentiation (Th2) (M19)","T cell activation (II) (M7.3)","nuclear pore complex (M106.0)","enriched for cell migration (M122)","TBA (M98.1)","TBA (M125)","xenobiotic metabolism (M164)","golgi membrane (II) (M237)","leukocyte activation and migration (M45)","TBA (M153)","TBA (M108)","TBA (M120)","cell adhesion (M51)","mitochondrial cluster (M235)","T cell signaling and costimulation (M44)","TBA (M185)","inositol phosphate metabolism (M129)","TBA (source: naive B cells) (M152.1)","extracellular region cluster (GO) (M189)","TBA (M184.0)","transmembrane and ion transporters (I) (M142)","regulation of transcription, transcription factors (M213)","TBA (M184.1)","Monocyte surface signature (S4)","TBA (M229)","TBA (M90)","TBA (M149)","enriched for TF motif TNCATNTCCYR (M232)","nuclear pore, transport; mRNA splicing, processing (M143)","BCR signaling (M54)","small GTPase mediated signal transduction (M215)","cell junction (M162.1)","TBA (M218)","TBA (M128)","TBA (M186)","plasma membrane, cell junction (M162.0)","TBA (M244)","metabolism in mitochondria, peroxisome (M187)","TBA (M137)","transmembrane transport (SLC cluster) (M154.1)","TBA (M176)","mismatch repair (I) (M22.0)","enriched in monocytes (IV) (M118.0)","enriched in plasma membrane proteins (II) (M135.1)","cell cell adhesion (M133.1)","TBA (M233)","TBA (M198)","enriched in monocytes (I) (M4.15)","TBA (M66)","transmembrane and ion transporters (II) (M224)","cell adhesion (GO) (M117)","TBA (M99)","Hox cluster II (M17.1)","TBA (M207)","TBA (M180)","TBA (M220)","TBA (M236)","enriched in hepatocyte nuclear factors (II) (M88.2)","type I interferon response (M127)","TBA (M70.1)","T cell surface signature (S0)","cell cycle, mitotic phase (M230)","targets of FOSL1/2 (M0)","mismatch repair (II) (M22.1)","platelet activation (III) (M42)","mitotic cell division (M6)","TBA (M190)","T cell activation (IV) (M52)","TBA (M214)","Hox cluster VI (M107)","cell division (M37.3)","nucleotide metabolism (M181)","Hox cluster III (M17.2)","enriched for TF motif TTCNRGNNNNTTC (M172)","chemokine cluster (I) (M27.0)","platelet activation (I) (M32.0)","enriched in hepatocyte nuclear factors (I) (M88.1)","TBA (M243)","receptors, cell migration (M109)","inflammatory response (M33)","Hox cluster IV (M17.3)","innate antiviral response (M150)","cell adhesion, membrane (M133.0)","collagen, TGFB family et al (M77)","platelet activation (II) (M32.1)","IL2, IL7, TCR network (M65)","T cell differentiation (M14)","PLK1 signaling events (M4.2)","metabolism of steroids (M225)","axon guidance (M110)","Hox cluster I (M17.0)","growth factor induced, enriched in nuclear receptor subfamily 4 (M94)","TBA (M205)","DC surface signature (S5)","cell junction (GO) (M4.13)","integrin mediated leukocyte migration (M39)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","myeloid cell cytokines, metallopeptidases and laminins (M78)","enriched in T cells (II) (M223)","mitotic cell cycle (M4.7)","enriched in NK cells (III) (M157)","lysosomal/endosomal proteins (M139)","TBA (M148)","KLF12 targets network (M32.3)","enriched for ubiquitination (M138)","integrins and cell adhesion (M84)","TBA (M48)","CD28 costimulation (M12)","suppression of MAPK signaling (M56)","complement activation (II) (M112.1)","DNA repair (M76)","TBA (M208)","TBA (M161)","Membrane, ER proteins (M134)","B cell development/activation (M58)","myeloid cell enriched receptors and transporters (M4.3)","T cell activation (III) (M7.4)","TBA (M183)","cell division (stimulated CD4+ T cells) (M46)","Plasma cell surface signature (S3)","Hox cluster V (M96)","E2F transcription factor network (M8)","CORO1A-DEF6 network (I) (M32.2)","extracellular matrix (II) (M2.1)","enriched in monocytes (III) (M73)","antigen processing and presentation (M200)","T &amp; B cell development, activation (M62.0)","CCR1, 7 and cell signaling (M59)","inflammasome receptors and signaling (M53)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","enriched in B cell differentiation (M123)","enriched for unknown TF motif CTCNANGTGNY (M62.1)","cell cycle (I) (M4.1)","TBA (M203)","enriched in B cells (I) (M47.0)","enriched in DNA interacting proteins (M182)","adhesion and migration, chemotaxis (M91)","Activated (LPS) dendritic cell surface signature (S11)"],[137,17,11,16,20,73,173,21,18,26,21,16,13,38,31,12,16,9,16,16,15,51,10,12,45,13,28,11,10,14,12,14,13,12,14,11,10,11,30,7,12,10,14,9,15,9,43,9,11,21,11,23,16,13,29,10,12,11,8,12,9,10,17,9,12,12,39,12,19,9,23,43,16,26,16,10,6,10,8,13,11,12,8,10,35,13,9,14,12,8,15,14,9,14,11,65,7,11,7,10,11,12,11,11,11,11,9,16,7,7,15,16,10,25,44,11,10,12,10,10,16,11,21,9,11,10,11,10,10,10,11,9,22,8,11,13,10,25,10,13,12,9,10,9,10,8,25,22,11,9,14,9,8,11,14,33,21,11,11,31,6,10,10,12,11,65,10,10,15,11,9,20,12,10,11,16,8,9,12,8,11,10,21,7,14,13,11,23,13,9,24,16,7,12,19,44,11,6,30,10,10,10,8,6,127,10,37,12,12,27],["Down","Up","Up","Down","Up","Up","Up","Up","Down","Down","Up","Up","Down","Down","Down","Up","Down","Up","Up","Down","Up","Up","Down","Down","Down","Up","Down","Down","Up","Up","Down","Down","Up","Up","Down","Up","Up","Up","Down","Down","Up","Up","Down","Up","Down","Down","Up","Up","Up","Up","Down","Down","Up","Up","Down","Down","Down","Down","Down","Up","Down","Down","Down","Down","Down","Down","Up","Down","Up","Up","Down","Up","Up","Up","Up","Down","Down","Down","Down","Up","Up","Up","Down","Up","Down","Up","Up","Up","Up","Down","Down","Down","Down","Up","Down","Up","Up","Down","Down","Up","Up","Up","Up","Down","Up","Up","Down","Down","Down","Down","Up","Down","Down","Up","Up","Down","Down","Down","Up","Up","Up","Down","Up","Down","Down","Down","Up","Down","Up","Down","Up","Down","Up","Up","Down","Up","Up","Down","Up","Up","Up","Down","Up","Up","Down","Down","Down","Up","Down","Up","Up","Up","Down","Up","Down","Down","Up","Up","Up","Down","Down","Down","Down","Down","Up","Down","Up","Up","Up","Down","Up","Down","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Down","Up","Down","Up","Up","Up","Down","Down","Down","Down","Down","Up","Down","Up","Up","Down","Up","Up","Up","Down","Down","Down","Up","Up","Up","Up","Down"],[2.40356704435057e-09,1.51799023065376e-07,7.71648245869337e-07,1.32856657645535e-06,1.35447877946093e-06,1.43440881339789e-06,1.6080788798292e-06,3.20871916663307e-06,4.70884696295066e-06,6.19911143957757e-06,6.49324501678842e-06,6.90250041180434e-06,7.9620189971319e-06,8.45923010173472e-06,1.01366564040058e-05,1.04679027675004e-05,1.07156494790787e-05,1.30472720940677e-05,1.62925898915088e-05,1.73201968026851e-05,1.99243983546245e-05,2.07800617249286e-05,2.18607384528942e-05,2.23522794527977e-05,2.63187777870715e-05,2.70184987179027e-05,2.70333414333236e-05,2.78170926109667e-05,5.08865929722502e-05,5.27588221342762e-05,6.5327120864051e-05,6.64206103179342e-05,7.81412309616518e-05,8.18869022115493e-05,8.20248604423041e-05,0.000102853343530955,0.000104761741396433,0.000105632240034674,0.000113024635610752,0.000140790572247149,0.000160842088770876,0.000166665808676871,0.000190172421529224,0.000196010373762836,0.000215132191996472,0.000246776258089094,0.000278269039743373,0.000291374603980331,0.000325038833217858,0.000336747565604521,0.000341304728357174,0.00035382078704187,0.000372645580829384,0.000399887320240715,0.000418659070357292,0.000428326655558434,0.000435732552165912,0.00051330049023582,0.000523019618618848,0.000541567531905768,0.000601823350550037,0.000657711213733312,0.000703119907679373,0.000709218226259292,0.000739052643814637,0.000759131035247531,0.000864628151053624,0.000966538691858118,0.00100620328447406,0.00103640253144921,0.00108828372485674,0.00118962877453636,0.00119076757348634,0.00120791888787945,0.00130019321981391,0.00132827664801182,0.00139119752945348,0.00158527680530804,0.00159599942272326,0.00161362877136662,0.00162519646097597,0.00167366040048489,0.00172722479198261,0.00183609631306958,0.00209787754965184,0.00213706735925587,0.00221819795569414,0.00231446377921375,0.00245130464374224,0.00254430394923549,0.0025446817356089,0.00264232552717906,0.00277539113037456,0.00282825986735622,0.00284840050179489,0.00285381612444929,0.00289124405319542,0.00314257258545312,0.00337674014807473,0.00343740844455905,0.0034872400705217,0.00353152218064722,0.00369838034872323,0.00375323450278772,0.00383957646446773,0.00419909262502312,0.00434634919930424,0.00476270834720663,0.00486283899638691,0.0051225148151271,0.00540371580564722,0.00542470129707649,0.00545497238481702,0.00587025837789818,0.00603734749553329,0.00614312412129085,0.00698473910186368,0.007350152777465,0.00737379850661672,0.00770976191935981,0.00792397801296175,0.0082165479608309,0.0082174161097161,0.0093254274488662,0.009330216740553,0.0100194040354262,0.0105732571791827,0.0117173522651496,0.0121666579580559,0.0139698092008934,0.0145191551878272,0.0145267425422082,0.0147601585543464,0.0152499565481034,0.0156972562612194,0.0158748140136877,0.0165346101514044,0.0167226077754068,0.0168019450335856,0.0168079942932368,0.0169489151213142,0.0178032853585802,0.0179655469574555,0.0181916430640401,0.018270527206042,0.0190082381777159,0.0202836736554617,0.0205047862440166,0.0208985278657112,0.0245271060993756,0.0246265759037907,0.0246367517979931,0.0250385119906522,0.0261592802857622,0.0279826491764939,0.0282219429150587,0.0286139330463956,0.0297757576795045,0.0307464788943991,0.0327993927852992,0.0329585704910134,0.0330965675248776,0.033320288139888,0.0336051006653825,0.0342336297932115,0.034316924318335,0.0365662725801923,0.0374001550964241,0.037553232099067,0.0385564286503003,0.0407405793705371,0.0429373281522538,0.0442941554309953,0.0446205725757112,0.0456003663943978,0.0459049313034254,0.046301022994027,0.046976642509972,0.0470957494761304,0.0483803719928795,0.0486821615188802,0.0514437456610489,0.0535534153503647,0.0541839658294106,0.0574429445141631,0.0588072875807951,0.059179623387812,0.0598698531191929,0.0607872705389186,0.0619077496648196,0.0669357487843004,0.0670926658059111,0.0679673129241465,0.0681681245290463,0.0694877115819003,0.073494783871703,0.0737297277672316,0.0746908663242094,0.0766953105886688,0.0784674019954013,0.0808703563679168,0.0808720598879061,0.0860877879233825,0.0922551497371287,0.100818328674743,0.101466599874392,0.101521903558209,0.105350173129562,0.106126424929823,0.11454902406543],[8.17212795079193e-07,2.58058339211139e-05,7.81066884488467e-05,7.81066884488467e-05,7.81066884488467e-05,7.81066884488467e-05,7.81066884488467e-05,0.000136370564581906,0.000177889774155914,0.000195570845001123,0.000195570845001123,0.000195570845001123,0.000205438445327843,0.000205438445327843,0.000214312989581574,0.000214312989581574,0.000214312989581574,0.000246448472887945,0.000291551608584894,0.000294443345645647,0.000316657292247968,0.000316657292247968,0.000316657292247968,0.000316657292247968,0.000337778981704596,0.000337778981704596,0.000337778981704596,0.000337778981704596,0.000596601434847071,0.000597933317521796,0.000705718984628051,0.000705718984628051,0.000796812930010954,0.000796812930010954,0.000796812930010954,0.000945130568731294,0.000945130568731294,0.000945130568731294,0.00098534297711938,0.00119671986410077,0.0013338124434658,0.00134919940357467,0.00150368891441712,0.0015146256154401,0.00162544322841779,0.00182399842935417,0.00201301007473929,0.00206390344486068,0.00225537149579739,0.0022753648557145,0.0022753648557145,0.00231344360758146,0.00239055655626397,0.00251780905336747,0.00258807425311781,0.00259910645151597,0.00259910645151597,0.00300900287379619,0.00301401136153234,0.00306888268079935,0.00335442523257398,0.00360680343015042,0.00376772182700249,0.00376772182700249,0.00386581382918425,0.00391067503006304,0.00438766524415272,0.00483269345929059,0.00495810314088667,0.00503395515275332,0.00521149952748299,0.00554604075322404,0.00554604075322404,0.00554989759295965,0.0058942092631564,0.00594229026742131,0.00614295013005432,0.00682181230533121,0.00682181230533121,0.00682181230533121,0.00682181230533121,0.00693956751420565,0.00707537866595285,0.00743181841004352,0.00839151019860737,0.00844887095519762,0.0086688195969656,0.00894224641968951,0.0093645345940715,0.00950760208908819,0.00950760208908819,0.00976511607870522,0.0101072654407579,0.0101072654407579,0.0101072654407579,0.0101072654407579,0.0101342575060458,0.0109028028474904,0.0115968853570243,0.0116871887115008,0.011739223999776,0.0117717406021574,0.0122082458113194,0.0122701897206521,0.0124329142658955,0.0134687876651685,0.013810829231434,0.0149937114634283,0.015168488612583,0.0158332276103928,0.0164131912463521,0.0164131912463521,0.0164131912463521,0.0175077881446086,0.0178495491172289,0.0180057086313697,0.0202975324327663,0.0210679957331906,0.0210679957331906,0.0218443254381861,0.0222657233422066,0.0227148087585648,0.0227148087585648,0.0253781895343042,0.0253781895343042,0.0270364870797216,0.0283063578025365,0.0311242169543036,0.0320671605096046,0.036536424063875,0.0374173671541727,0.0374173671541727,0.0377327361539683,0.0386939195996654,0.0395338305838119,0.0396870350342193,0.0408194147121466,0.0408194147121466,0.0408194147121466,0.0408194147121466,0.0408697244060059,0.0426275846613891,0.0427152864722718,0.0428412362072708,0.0428412362072708,0.0442657601398864,0.0469146193391631,0.0471055900200381,0.04768791593518,0.0551085237586687,0.0551085237586687,0.0551085237586687,0.0556411377570048,0.057754255176358,0.061381294967793,0.0615093627635895,0.0619664792087548,0.0640744152596932,0.065747187572929,0.0694619318423357,0.0694619318423357,0.0694619318423357,0.0695024415187848,0.0696691111355492,0.070287676314662,0.070287676314662,0.074446303456679,0.0755508811460519,0.0755508811460519,0.0771128573006006,0.0810046607367404,0.0848761137893389,0.0870520973788348,0.0871896245732288,0.0885949975662585,0.0886799809270718,0.0889398181806169,0.0894556135300801,0.0894556135300801,0.0913851470976614,0.0914471542343605,0.0961037006854759,0.0994981487383825,0.100122545554346,0.105570816944948,0.107497192351991,0.107599315250567,0.108275266279391,0.109352761816044,0.110782288873888,0.118809929031301,0.118809929031301,0.119469908968432,0.119469908968432,0.121158061219724,0.127249276349537,0.127249276349537,0.128257043182986,0.131037214071092,0.133394583392182,0.136121288920238,0.136121288920238,0.14418644282734,0.153758582895215,0.166750952704304,0.166750952704304,0.166750952704304,0.172207013769477,0.17264585873751,0.185460324677363]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="heatmap-of-significant-gene-sets" class="section level1">
<h1>Heatmap of significant gene-sets</h1>
<p>FDR cutoff &lt;= 10^{-5}</p>
<img src="figure/GSEA_T1.Rmd/heatmap-1.png" width="672" style="display: block; margin: auto;" />
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
geneset
</th>
<th style="text-align:center;">
NGenes
</th>
<th style="text-align:center;">
Direction
</th>
<th style="text-align:center;">
PValue
</th>
<th style="text-align:center;">
FDR
</th>
<th style="text-align:center;">
time
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
173
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in T cells (I) (M7.0)
</td>
<td style="text-align:center;">
51
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells (II) (M165)
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (I) (M7.2)
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
Monocyte surface signature (S4)
</td>
<td style="text-align:center;">
65
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
T cell activation (I) (M7.1)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M70.0)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
innate antiviral response (M150)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M125)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
antiviral IFN signature (M75)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid cell enriched receptors and transporters (M4.3)
</td>
<td style="text-align:center;">
23
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
type I interferon response (M127)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M66)
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (IV) (M118.0)
</td>
<td style="text-align:center;">
44
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
T cell activation (III) (M7.4)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells/monocytes (M64)
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
formyl peptide receptor mediated neutrophil response (M11.2)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (II) (M61.0)
</td>
<td style="text-align:center;">
23
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and transcription (M4.0)
</td>
<td style="text-align:center;">
286
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
activated dendritic cells (M67)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and transcription (M4.0)
</td>
<td style="text-align:center;">
286
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T3
</td>
</tr>
<tr>
<td style="text-align:center;">
T cell differentiation (M14)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
cell junction (GO) (M4.13)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
T cell differentiation via ITK and PKC (M18)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in B cells (I) (M47.0)
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T3
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
173
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T3
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (receptor activation) (M61.2)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
T2
</td>
</tr>
</tbody>
</table>
<div id="summary" class="section level2">
<h2>Summary</h2>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
visit
</th>
<th style="text-align:center;">
significant genesets
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
T2
</td>
<td style="text-align:center;">
130
</td>
</tr>
<tr>
<td style="text-align:center;">
T3
</td>
<td style="text-align:center;">
145
</td>
</tr>
<tr>
<td style="text-align:center;">
T4
</td>
<td style="text-align:center;">
46
</td>
</tr>
<tr>
<td style="text-align:center;">
T5
</td>
<td style="text-align:center;">
210
</td>
</tr>
</tbody>
</table>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 3.5.3 (2019-03-11)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.5 LTS

Matrix products: default
BLAS/LAPACK: /app/easybuild/software/OpenBLAS/0.2.18-GCC-5.4.0-2.26-LAPACK-3.6.1/lib/libopenblas_prescottp-r0.2.18.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] heatmap3_1.1.6       magrittr_1.5         DT_0.5              
 [4] limma_3.38.3         RNASeqUtilities_1.1  here_0.1            
 [7] data.table_1.12.2    forcats_0.4.0        stringr_1.4.0       
[10] dplyr_0.8.0.1        purrr_0.3.2          readr_1.3.1         
[13] tidyr_0.8.3          tibble_2.1.1         ggplot2_3.1.1       
[16] tidyverse_1.2.1      GSEABase_1.44.0      graph_1.60.0        
[19] annotate_1.60.1      XML_3.98-1.19        AnnotationDbi_1.44.0
[22] IRanges_2.16.0       S4Vectors_0.20.1     Biobase_2.42.0      
[25] BiocGenerics_0.28.0  knitr_1.22          

loaded via a namespace (and not attached):
 [1] nlme_3.1-137       bitops_1.0-6       fs_1.2.7          
 [4] lubridate_1.7.4    bit64_0.9-7        webshot_0.5.1     
 [7] httr_1.4.0         rprojroot_1.3-2    tools_3.5.3       
[10] backports_1.1.3    R6_2.4.0           DBI_1.0.0         
[13] lazyeval_0.2.1     colorspace_1.4-1   withr_2.1.2       
[16] tidyselect_0.2.5   gridExtra_2.3      bit_1.1-14        
[19] compiler_3.5.3     git2r_0.25.2       cli_1.1.0         
[22] rvest_0.3.2        xml2_1.2.0         scales_1.0.0      
[25] digest_0.6.18      rmarkdown_1.12     pkgconfig_2.0.2   
[28] htmltools_0.3.6    htmlwidgets_1.3    rlang_0.3.4       
[31] readxl_1.3.0       rstudioapi_0.10    RSQLite_2.1.1     
[34] shiny_1.3.1        generics_0.0.2     jsonlite_1.6      
[37] crosstalk_1.0.0    RCurl_1.95-4.12    kableExtra_1.1.0  
[40] Rcpp_1.0.1         munsell_0.5.0      stringi_1.4.3     
[43] yaml_2.2.0         plyr_1.8.4         grid_3.5.3        
[46] blob_1.1.1         promises_1.0.1     crayon_1.3.4      
[49] lattice_0.20-38    haven_2.1.0        hms_0.4.2         
[52] pillar_1.3.1       fastcluster_1.1.25 reshape2_1.4.3    
[55] glue_1.3.1         evaluate_0.13      modelr_0.1.4      
[58] httpuv_1.5.1       cellranger_1.1.0   gtable_0.2.0      
[61] assertthat_0.2.0   xfun_0.6           mime_0.6          
[64] xtable_1.8-3       broom_0.5.2        later_0.8.0       
[67] viridisLite_0.3.0  memoise_1.1.0      workflowr_1.3.0   </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
