<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Carl Murie" />

<meta name="date" content="2019-04-30" />

<title>GSEA analysis - baseline (T0) vs all other time periods</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<script src="site_libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.5/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">VaheyProject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">GSEA analysis - baseline (T0) vs all other time periods</h1>
<h3 class="subtitle">interaction effect between visit and disease status</h3>
<h4 class="author">Carl Murie</h4>
<h4 class="date">April 30, 2019</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-04-30
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>VaheyProject/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.3.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20181008code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20181008)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20181008code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20181008)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong1ed83ed"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 1ed83ed </a>
</p>
</div>
<div id="strongRepositoryversionstrong1ed83ed" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    analysis/figure/
    Ignored:    data/.Rhistory

Untracked files:
    Untracked:  VaheyReport.tar.gz
    Untracked:  analysis/DEG_Disease.Rmd
    Untracked:  analysis/DEG_T0.Rmd
    Untracked:  analysis/DEG_T0_Interaction.Rmd
    Untracked:  analysis/DEG_T1.Rmd
    Untracked:  analysis/DEG_T1_Interaction.Rmd
    Untracked:  analysis/GSEA_Disease.Rmd
    Untracked:  analysis/GSEA_T0.Rmd
    Untracked:  analysis/GSEA_T0_Interaction.Rmd
    Untracked:  analysis/GSEA_T1.Rmd
    Untracked:  analysis/GSEA_T1_Interaction.Rmd
    Untracked:  analysis/analysis_T4_disease.Rmd
    Untracked:  analysis/analysis_T5_disease.Rmd
    Untracked:  analysis/pamr.Rmd
    Untracked:  code/processGEOFiles.R
    Untracked:  data/BTM_for_GSEA_20131008.gmt
    Untracked:  data/GPL570.soft
    Untracked:  data/GPL571.soft
    Untracked:  data/GSE18323-GPL570_series_matrix.txt.gz
    Untracked:  data/GSE18323-GPL571_series_matrix.txt.gz
    Untracked:  data/gpl_eset.Rds
    Untracked:  data/h.all.v5.0.symbols.gmt
    Untracked:  docs/figure/DEG_T0.Rmd/
    Untracked:  docs/figure/DEG_T0_Interaction.Rmd/
    Untracked:  docs/figure/DEG_T1.Rmd/
    Untracked:  docs/figure/DEG_T1_Interaction.Rmd/
    Untracked:  docs/figure/GSEA_T0.Rmd/
    Untracked:  docs/figure/GSEA_T0_Interaction.Rmd/
    Untracked:  docs/figure/GSEA_T1.Rmd/
    Untracked:  docs/figure/GSEA_T1_Interaction.Rmd/
    Untracked:  docs/figure/pamr.Rmd/
    Untracked:  output/dmso_M3_both_disease.csv
    Untracked:  output/dmso_M3_old_disease.csv
    Untracked:  output/dmso_M3_young_disease.csv
    Untracked:  output/pam_res.Rda

Unstaged changes:
    Modified:   analysis/analysis_PCA.Rmd
    Modified:   analysis/experimental_design.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/processVahey.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with <code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="gsea-analysis" class="section level1">
<h1>GSEA Analysis</h1>
<p>The Blood Transcript Modules (BTM) from Shuzhao et al. Molecular signatures of antibody responses derived from a systems biological study of 5 human vaccines, 2014 were used for the GSEA analysis.</p>
<div id="baseline-t0-vs-visits-t1-to-t5-time-vs-disease-interaction-effect" class="section level2">
<h2>baseline (T0) vs visits T1 to T5: time vs disease interaction effect</h2>
<ul>
<li><strong>visit</strong>
<ul>
<li>T0 - study entry (baseline)</li>
<li>T1 - day of third vaccination</li>
<li>T2 - 24h after T1</li>
<li>T3 - 72h after T1</li>
<li>T4 - 2 weeks after T1</li>
<li>T5 - 5 days after challenge</li>
</ul></li>
<li><strong>disease</strong> - response to malaria challenge
<ul>
<li>Protected - vaccinated and protected from challenge<br />
</li>
<li>NonProtected - vaccinated and not protected from challenge</li>
</ul></li>
</ul>
<p>A FDR cutoff of &lt;= 0.2 was used.</p>
<p>The formula used in the camera function is: ~visit*disease</p>
</div>
<div id="sample-size" class="section level2 tabset tabset-fade tabset-pills">
<h2>Sample size</h2>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
disease
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
total
</th>
<th style="text-align:center;">
Protected
</th>
<th style="text-align:center;">
NonProtected
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
T0
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
T1
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
T2
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
T3
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
T4
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
T5
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
26
</td>
</tr>
</tbody>
</table>
</div>
<div id="comparisons" class="section level2 tabset tabset-fade tabset-pills">
<h2>Comparisons</h2>
<div id="t0-versus-t1-t1---t0" class="section level3">
<h3>T0 versus T1 (T1 - T0)</h3>
<div id="htmlwidget-9c23e70c978f4c72e429" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9c23e70c978f4c72e429">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67"],["enriched in monocytes (II) (M11.0)","enriched in activated dendritic cells (II) (M165)","enriched in NK cells (I) (M7.2)","enriched in NK cells (II) (M61.0)","TLR and inflammatory signaling (M16)","phosphatidylinositol signaling system (M101)","TBA (M236)","enriched in NK cells (KIR cluster) (M61.1)","TBA (M214)","TBA (M205)","TBA (M72.0)","TBA (M72.2)","cytoskeletal remodeling (M32.8)","inositol phosphate metabolism (M129)","TBA (M174)","cell cycle, ATP binding (M144)","platelet activation (II) (M32.1)","TBA (M72.1)","enriched in NK cells (receptor activation) (M61.2)","enriched in T cells (I) (M7.0)","regulation of transcription, transcription factors (M213)","golgi membrane (II) (M237)","TBA (M148)","immune activation - generic cluster (M37.0)","TBA (M177.1)","NK cell surface signature (S1)","enriched in monocytes (IV) (M118.0)","TBA (M194)","myeloid cell enriched receptors and transporters (M4.3)","TBA (M180)","intracellular transport (M147)","nuclear pore complex (mitosis) (M106.1)","TBA (M137)","TBA (M161)","Monocyte surface signature (S4)","TBA (M32.5)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","TBA (M153)","plasma cells, immunoglobulins (M156.1)","MAPK, RAS signaling (M100)","TBA (M26.0)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","TBA (M128)","regulation of localization (GO) (M63)","endoplasmic reticulum (M37.2)","proinflammatory cytokines and chemokines (M29)","enriched in activated dendritic cells (I) (M119)","TBA (M243)","enriched in monocytes (I) (M4.15)","enriched in monocytes (III) (M73)","TBA (M192)","TBA (M136)","mitotic cell cycle - DNA replication (M4.4)","proteasome (M226)","TBA (M80)","interferon alpha response (II) (M158.1)","mitosis (TF motif CCAATNNSNNNGCG) (M169)","TBA (M32.6)","TBA (M66)","type I interferon response (M127)","TBA (M125)","antiviral IFN signature (M75)","enriched in activated dendritic cells/monocytes (M64)","TBA (source: B cells) (M217)","antigen processing and presentation (M200)","CCR1, 7 and cell signaling (M59)","Hox cluster III (M17.2)"],[173,32,39,23,43,13,10,13,12,11,21,12,10,12,21,15,21,16,14,51,14,13,11,266,8,41,44,13,23,11,16,11,15,14,65,14,10,12,11,9,26,15,11,12,15,9,10,9,10,11,12,16,26,10,16,12,14,13,16,11,10,20,15,9,6,10,10],["Down","Down","Up","Up","Down","Down","Down","Up","Down","Down","Down","Down","Up","Down","Down","Down","Up","Down","Up","Up","Down","Down","Down","Down","Down","Up","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Down","Down","Up","Up","Up","Down","Down","Up","Down","Down","Down","Up","Down","Down","Up","Up","Up","Down","Down","Up","Down","Up","Down","Down","Down","Down","Down","Up","Down","Down","Up"],[2.31592574208034e-06,1.44654227008303e-05,1.8237664995333e-05,2.0993847630218e-05,2.54130938521984e-05,8.85686192663894e-05,0.000108147967489485,0.000122594028289195,0.00016097738670398,0.000238305375743058,0.000436253327681568,0.000476508595249433,0.000503193410328289,0.000568708525748621,0.00105718111316731,0.00122605303731127,0.00135709238934641,0.00136757988670642,0.00138472687691696,0.00143991698468098,0.00147904908987979,0.00159087492819273,0.001808742845771,0.00184203899412455,0.00197883254686929,0.00280464765586835,0.00297788149546576,0.00315301023962165,0.00334596846702477,0.00340650729957752,0.00390200526803534,0.00403451463844322,0.00453322191425473,0.00457543257199493,0.00520220841791614,0.00785290133820105,0.00828890259514246,0.00830604314977437,0.00878121228603106,0.010709665968168,0.0112529224722124,0.01145510127771,0.0122741715049719,0.0126355280916499,0.014242516751955,0.0159015348922386,0.0202010844452618,0.0206103897945836,0.0210388247235545,0.023830283667575,0.0248430080805049,0.0253751989015803,0.0255452516608126,0.027210250468106,0.0279978708261315,0.0297282858229302,0.0307977578800972,0.0308050291188622,0.0310022942704709,0.0316370468468916,0.033771392291047,0.0342228005517284,0.0345021626406222,0.0350286822432,0.0365785102849483,0.0376197467799219,0.0391888567536489],[0.000787414752307315,0.00172809038194949,0.00172809038194949,0.00172809038194949,0.00172809038194949,0.0050188884250954,0.00521024620229079,0.00521024620229079,0.00608136794215036,0.00810238277526397,0.0131604430393553,0.0131604430393553,0.0131604430393553,0.0138114927681808,0.0239465090742442,0.0239465090742442,0.0239465090742442,0.0239465090742442,0.0239465090742442,0.0239465090742442,0.0239465090742442,0.0245862488902513,0.0260955524167645,0.0260955524167645,0.0269121226374224,0.0366761616536631,0.0374992484614207,0.0382865529096914,0.0386070827285453,0.0386070827285453,0.0427961868107102,0.0428667180334592,0.0457543257199493,0.0457543257199493,0.0505357389168996,0.0741662904163432,0.0743172281821918,0.0743172281821918,0.07655415839104,0.0910321607294283,0.0927317722481288,0.0927317722481288,0.0970515886439639,0.097638171617295,0.107610126570326,0.117533083986112,0.145983681755276,0.145983681755276,0.145983681755276,0.16204592893951,0.163875199333515,0.163875199333515,0.163875199333515,0.17132379924363,0.173077746925176,0.178657289016273,0.178657289016273,0.178657289016273,0.178657289016273,0.179276598799053,0.186089874417,0.186089874417,0.186089874417,0.186089874417,0.191333746105883,0.193798695532931,0.198868825317024]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t0-versus-t2-t2---t0" class="section level3">
<h3>T0 versus T2 (T2 - T0)</h3>
<div id="htmlwidget-a0e6efde7f4990d8641e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-a0e6efde7f4990d8641e">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50"],["TLR and inflammatory signaling (M16)","enriched in neutrophils (I) (M37.1)","enriched in activated dendritic cells (II) (M165)","enriched in monocytes (II) (M11.0)","enriched in NK cells (I) (M7.2)","enriched in T cells (I) (M7.0)","TBA (M125)","immune activation - generic cluster (M37.0)","enriched in NK cells (II) (M61.0)","plasma cells, immunoglobulins (M156.1)","endoplasmic reticulum (M37.2)","platelet activation (II) (M32.1)","interferon alpha response (I) (M158.0)","TBA (source: B cells) (M152.0)","recruitment of neutrophils (M132)","proinflammatory cytokines and chemokines (M29)","Monocyte surface signature (S4)","enriched in activated dendritic cells/monocytes (M64)","TBA (M70.0)","plasma cells &amp; B cells, immunoglobulins (M156.0)","proteasome (M226)","enriched in myeloid cells and monocytes (M81)","TBA (source: naive B cells) (M152.1)","TBA (M174)","chemokines and inflammatory molecules in myeloid cells (M86.0)","enriched in monocytes (IV) (M118.0)","TLR8-BAFF network (M25)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","transmembrane transport (I) (M87)","translation initiation factor 3 complex (M245)","T cell activation (I) (M7.1)","cytoskeletal remodeling (M32.8)","TBA (M161)","CORO1A-DEF6 network (I) (M32.2)","enriched in neutrophils (II) (M163)","proinflammatory dendritic cell, myeloid cell response (M86.1)","cell junction (GO) (M4.13)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","formyl peptide receptor mediated neutrophil response (M11.2)","enriched in NK cells (KIR cluster) (M61.1)","cell junction (M162.1)","TBA (source: memory B cells) (M152.2)","inflammasome receptors and signaling (M53)","TBA (M70.1)","T cell surface signature (S0)","antiviral IFN signature (M75)","platelet activation (I) (M32.0)","TBA (M148)","enriched in monocytes (I) (M4.15)","enriched in NK cells (receptor activation) (M61.2)"],[43,39,32,173,39,51,10,266,23,11,15,21,16,16,9,9,65,15,17,24,10,34,8,21,18,44,8,10,24,9,43,10,14,19,12,12,10,15,9,13,11,10,10,9,22,20,22,11,10,14],["Down","Down","Down","Down","Up","Up","Down","Down","Up","Up","Down","Up","Down","Down","Down","Down","Down","Down","Down","Up","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Up","Up","Down","Up","Down","Down","Down","Down","Down","Up","Down","Down","Down","Down","Up","Down","Up","Down","Down","Up"],[2.09533081072212e-07,5.3317336973085e-07,4.2411317951777e-06,6.71006919919476e-06,8.44399396739212e-06,9.59885079259019e-05,0.000107792920384857,0.0001220738457875,0.000152104443743604,0.000287260510786966,0.000481782825566084,0.000548152328341949,0.000750888814251347,0.000938102145280026,0.00196047112014121,0.00254698669962549,0.00274343505943086,0.00309962115989791,0.00328934766054193,0.00328949519682033,0.00351360573047647,0.00387061471509565,0.0044811010051359,0.00586463634694621,0.00673235109398228,0.00744981078723806,0.0077472237016917,0.00815981561147964,0.00979001344785582,0.0111234661473568,0.0115755880637392,0.0116861593550726,0.0126101585841741,0.0150762582092394,0.0160422852621049,0.016628317403748,0.0186392962374641,0.0194710368230654,0.0196936913304811,0.0203360358785489,0.0206600616375651,0.0211051393108477,0.0213645142167469,0.0248702015317981,0.026370566297261,0.0266483893080084,0.0278454415913915,0.0284439896697327,0.028446489702934,0.0291010416208156],[7.12412475645522e-05,9.06394728542445e-05,0.000480661603453473,0.000570355881931554,0.000574191589782664,0.00518813844596876,0.00518813844596876,0.00518813844596876,0.00574616787475839,0.00976685736675683,0.0148914691538608,0.0155309826363552,0.0196386305265737,0.0227824806710864,0.0444373453898675,0.0541234673670416,0.0548687011886171,0.0559214183459456,0.0559214183459456,0.0559214183459456,0.0568869499220001,0.0598185910514783,0.0662423626846176,0.0830823482484046,0.0915599748781591,0.0974206026023439,0.0975576317990807,0.0990834752822527,0.114779468009344,0.124165443147646,0.124165443147646,0.124165443147646,0.129922846018764,0.150762582092394,0.155839342546162,0.157045219924286,0.16892871706265,0.16892871706265,0.16892871706265,0.16892871706265,0.16892871706265,0.16892871706265,0.16892871706265,0.19217883001844,0.196966355754845,0.196966355754845,0.197383806101991,0.197383806101991,0.197383806101991,0.197887083021546]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t0-versus-t3-t3---t0" class="section level3">
<h3>T0 versus T3 (T3 - T0)</h3>
<div id="htmlwidget-c47734cf8f874295dd41" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-c47734cf8f874295dd41">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86"],["enriched in monocytes (II) (M11.0)","enriched in activated dendritic cells (II) (M165)","TLR and inflammatory signaling (M16)","TBA (M72.0)","platelet activation (II) (M32.1)","TBA (M174)","enriched in NK cells (I) (M7.2)","TBA (M185)","regulation of transcription, transcription factors (M213)","enriched in T cells (I) (M7.0)","cell cycle, ATP binding (M144)","enriched in NK cells (II) (M61.0)","Monocyte surface signature (S4)","platelet activation (I) (M32.0)","TBA (M72.2)","TBA (M72.1)","cytoskeletal remodeling (M32.8)","TBA (M148)","TBA (M32.5)","intracellular transport (M147)","plasma cells, immunoglobulins (M156.1)","TBA (M136)","inositol phosphate metabolism (M129)","phosphatidylinositol signaling system (M101)","TBA (M192)","TBA (M205)","interferon alpha response (II) (M158.1)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","TBA (M161)","enriched in monocytes (IV) (M118.0)","TBA (M125)","CORO1A-DEF6 network (I) (M32.2)","TBA (M153)","regulation of localization (GO) (M63)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","TBA (M32.6)","MAPK, RAS signaling (M100)","TBA (M214)","plasma cells &amp; B cells, immunoglobulins (M156.0)","enriched in antigen presentation (II) (M95.0)","C-MYC transcriptional network (M4.12)","TBA (M183)","TBA (M180)","TBA (M32.7)","TBA (M197)","TBA (M137)","enriched in B cells (I) (M47.0)","enriched in monocytes (III) (M73)","enriched for TF motif PAX3 (M179)","enriched in monocytes (I) (M4.15)","myeloid cell enriched receptors and transporters (M4.3)","TBA (M66)","TBA (M79)","translation initiation (M227)","KLF12 targets network (M32.3)","golgi membrane (II) (M237)","T cell surface signature (S0)","proinflammatory cytokines and chemokines (M29)","enriched in activated dendritic cells/monocytes (M64)","enriched in cell cycle (M167)","type I interferon response (M127)","TBA (M26.0)","enriched in NK cells (KIR cluster) (M61.1)","CORO1A-DEF6 network (II) (M32.4)","TBA (M194)","TBA (M128)","TBA (M243)","antiviral IFN signature (M75)","cell junction (GO) (M4.13)","TBA (M70.0)","mitotic cell cycle - DNA replication (M4.4)","innate antiviral response (M150)","T cell activation (I) (M7.1)","TBA (M105)","TBA (M218)","enriched in B cells (II) (M47.1)","TLR8-BAFF network (M25)","nuclear pore complex (mitosis) (M106.1)","enriched in dendritic cells (M168)","enriched in antigen presentation (III) (M95.1)","proteasome (M226)","chemokines and inflammatory molecules in myeloid cells (M86.0)","TBA (M184.0)","T cell activation (II) (M7.3)","TBA (M41.0)","proinflammatory dendritic cell, myeloid cell response (M86.1)"],[173,32,43,21,21,21,39,14,14,51,15,23,65,22,12,16,10,11,14,16,11,16,12,13,12,11,12,10,14,44,10,19,12,12,15,13,9,12,24,20,11,9,11,11,11,15,37,11,9,10,23,16,8,10,16,13,22,9,15,13,11,26,13,14,13,11,9,20,10,17,26,11,43,12,11,28,8,11,13,12,10,18,14,26,15,12],["Down","Down","Down","Down","Up","Down","Up","Down","Down","Up","Down","Up","Down","Up","Down","Down","Up","Down","Up","Down","Up","Up","Down","Down","Up","Down","Up","Down","Down","Down","Down","Up","Down","Up","Down","Up","Up","Down","Up","Down","Up","Up","Down","Up","Up","Down","Up","Down","Down","Down","Down","Down","Down","Down","Up","Down","Up","Down","Down","Up","Down","Up","Up","Up","Down","Down","Up","Down","Down","Down","Up","Down","Up","Down","Down","Up","Down","Down","Down","Down","Down","Down","Down","Up","Up","Down"],[1.29508563196916e-07,9.50726839205771e-07,8.84026800661199e-06,2.01539434486969e-05,2.79400048729205e-05,5.97015264339227e-05,0.000104316851763595,0.00017007588116682,0.000203954034045062,0.000237377980319589,0.000458300687505857,0.000587599025463587,0.000659245034908535,0.000664537901973371,0.000671991807728156,0.000730527620080525,0.000735487696852346,0.00107419645766216,0.0011110841965291,0.00117400894958658,0.00150228065418052,0.00152853591794721,0.00172568129098034,0.00190316409505462,0.00238485017900841,0.00248462794671837,0.00285103432659446,0.00294892768000561,0.00313101141566984,0.00319321719834506,0.00333795533069232,0.00354366288363662,0.0036344017437224,0.00364083997240241,0.00384490269114946,0.00451508370804995,0.00472117711525088,0.00618369338021989,0.00650271965119905,0.00682287211283239,0.00685430286184082,0.00691411774852791,0.0074368491329432,0.0076306673874554,0.00765960852187923,0.00791415669868168,0.00860711568143315,0.00904872992422184,0.00918489536370605,0.00937239483671792,0.00937291003187576,0.00978871137793755,0.0102787907827122,0.0114788421587474,0.0118057045484372,0.0122360992754242,0.012994525551019,0.0148189455229797,0.0155408048260714,0.0158349275441473,0.0166318317694495,0.0175103995272305,0.0175273111766459,0.0204403895216805,0.0205587342182614,0.0206799650410269,0.0208674265975616,0.0213610197754937,0.0215304021255516,0.0242972445292447,0.0246764281476284,0.0273515943853815,0.0288274783114986,0.0289657220420385,0.0290649566477751,0.0301942772838176,0.0365061819556195,0.0384458102257298,0.038958657321795,0.0400732736354011,0.0404344852728519,0.0430013902769294,0.0434238321991583,0.04497957741385,0.0468220047116613,0.0470893764628233],[4.40329114869516e-05,0.000161623562664981,0.00100189704074936,0.00171308519313924,0.00189992033135859,0.00338308649792228,0.00506681851423175,0.00722822494958987,0.00770493017503566,0.00807085133086604,0.0141656576138174,0.0147097539370469,0.0147097539370469,0.0147097539370469,0.0147097539370469,0.0147097539370469,0.0147097539370469,0.0198825593063103,0.0198825593063103,0.0199581521429718,0.0236228278228204,0.0236228278228204,0.0255100712579702,0.0269614913466071,0.0324339624345144,0.0324912885340094,0.0358084075429252,0.0358084075429252,0.0361897949145774,0.0361897949145774,0.0364083997240241,0.0364083997240241,0.0364083997240241,0.0364083997240241,0.0373504832854519,0.042642457242694,0.0433837897077108,0.0553277828756516,0.055971429392845,0.055971429392845,0.055971429392845,0.055971429392845,0.0578725977208653,0.0578725977208653,0.0578725977208653,0.0584959408163428,0.0622642410997292,0.0624860668791718,0.0624860668791718,0.0624860668791718,0.0624860668791718,0.0640031128557455,0.0659394125683422,0.0722741913698911,0.0729807190267025,0.0742906027436471,0.0775112050411658,0.0868696806519498,0.0895571803536316,0.0897312560835016,0.0927020131411939,0.094591838096184,0.094591838096184,0.105894403629417,0.105894403629417,0.105894403629417,0.105894403629417,0.106091836560689,0.106091836560689,0.118015187713474,0.118168810847798,0.129160306819857,0.131761136803247,0.131761136803247,0.131761136803247,0.135079661532868,0.161196128115722,0.16758430098395,0.167670170752029,0.169724999910736,0.169724999910736,0.17788075840619,0.17788075840619,0.182060194294155,0.186167302294883,0.186167302294883]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t0-versus-t4-t4---t0" class="section level3">
<h3>T0 versus T4 (T4 - T0)</h3>
<div id="htmlwidget-14032f3f90232640cb54" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-14032f3f90232640cb54">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106"],["enriched in monocytes (II) (M11.0)","enriched in activated dendritic cells (II) (M165)","TLR and inflammatory signaling (M16)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","cell cycle and transcription (M4.0)","type I interferon response (M127)","innate antiviral response (M150)","antiviral IFN signature (M75)","activated dendritic cells (M67)","Monocyte surface signature (S4)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","cell junction (GO) (M4.13)","RIG-1 like receptor signaling (M68)","myeloid cell enriched receptors and transporters (M4.3)","enriched in T cells (I) (M7.0)","MHC-TLR7-TLR8 cluster (M146)","TBA (M66)","enriched in monocytes (IV) (M118.0)","enriched in antigen presentation (II) (M95.0)","regulation of antigen presentation and immune response (M5.0)","formyl peptide receptor mediated neutrophil response (M11.2)","regulation of localization (GO) (M63)","TBA (M48)","enriched in activated dendritic cells/monocytes (M64)","enriched in antigen presentation (III) (M95.1)","enriched in monocytes (I) (M4.15)","enriched in NK cells (I) (M7.2)","TBA (M148)","TBA (M136)","translation initiation factor 3 complex (M245)","BCR signaling (M54)","RA, WNT, CSF receptors network (monocyte) (M23)","endoplasmic reticulum (M37.2)","Hox cluster III (M17.2)","enriched in activated dendritic cells (I) (M119)","innate activation by cytosolic DNA sensing (M13)","enriched in monocytes (III) (M73)","lysosomal/endosomal proteins (M139)","lysosome (M209)","proteasome (M226)","cell division in stimulated CD4 T cells (M4.6)","T cell differentiation (Th2) (M19)","blood coagulation (M11.1)","cell cycle (I) (M4.1)","platelet activation (I) (M32.0)","enriched in NK cells (II) (M61.0)","inflammasome receptors and signaling (M53)","complement and other receptors in DCs (M40)","G protein coupled receptors cluster (M155)","regulation of signal transduction (M3)","recruitment of neutrophils (M132)","TBA (M26.0)","Wnt signaling pathway (M206)","TBA (M26.2)","TBA (M26.1)","TBA (M192)","T cell differentiation via ITK and PKC (M18)","TBA (M214)","T cell activation (I) (M7.1)","antigen processing and presentation (M200)","complement activation (I) (M112.0)","TBA (M205)","T cell surface signature (S0)","T cell activation (III) (M7.4)","T cell differentiation (M14)","IL2, IL7, TCR network (M65)","enriched in neutrophils (II) (M163)","TBA (M180)","enriched in myeloid cells and monocytes (M81)","immune activation - generic cluster (M37.0)","Hox cluster II (M17.1)","KLF12 targets network (M32.3)","enriched in NK cells (receptor activation) (M61.2)","CCR1, 7 and cell signaling (M59)","TBA (M198)","Hox cluster IV (M17.3)","suppression of MAPK signaling (M56)","TBA (M161)","TBA (M55)","platelet activation (III) (M42)","B cell surface signature (S2)","enriched in antigen presentation (I) (M71)","TBA (M185)","TBA (M153)","inositol phosphate metabolism (M129)","xenobiotic metabolism (M164)","B cell development/activation (M58)","TBA (M41.2)","enriched in extracellular matrix &amp; associated proteins (M202)","transcription regulation in cell development (M49)","T cell activation (II) (M7.3)","TBA (M177.1)","TLR8-BAFF network (M25)","TBA (M41.0)","inflammatory response (M33)","cell activation (IL15, IL23, TNF) (M24)","chemokines and inflammatory molecules in myeloid cells (M86.0)","TBA (M233)","TBA (M121)","CORO1A-DEF6 network (II) (M32.4)","transmembrane transport (I) (M87)","E2F transcription factor network (M8)","transmembrane and ion transporters (II) (M224)","intracellular transport (M147)","proinflammatory dendritic cell, myeloid cell response (M86.1)","TBA (M149)"],[173,32,43,10,286,11,11,20,11,65,15,10,9,23,51,16,16,44,20,73,9,12,12,15,12,10,39,11,16,9,12,11,15,10,10,11,11,10,7,10,13,16,19,127,22,23,10,12,9,40,9,26,14,10,18,12,11,12,43,6,16,11,22,13,11,11,12,11,34,266,11,16,14,10,10,8,11,14,11,10,137,16,14,12,12,8,11,12,11,45,26,8,8,15,9,14,18,12,11,14,24,12,11,16,12,7],["Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Down","Down","Up","Down","Down","Down","Up","Down","Down","Down","Down","Down","Down","Down","Up","Down","Down","Down","Up","Down","Down","Down","Down","Down","Up","Up","Up","Up","Down","Up","Down","Up","Down","Down","Down","Up","Up","Up","Up","Down","Down","Down","Down","Up","Down","Up","Down","Down","Up","Down","Down","Down","Down","Up","Down","Down","Down","Down","Up","Down","Up","Up","Down","Up","Down","Down","Up","Down","Down","Down","Up","Down","Down","Down","Down","Up","Down","Down","Up"],[1.18205899175897e-18,1.47697687203834e-14,1.51758639995649e-14,5.32075343527018e-12,2.34658216813919e-11,4.61280289852461e-11,5.89613150604279e-11,5.17246212035073e-10,1.23300980722465e-08,2.64525170623073e-08,1.03306186896947e-07,2.78762147654128e-07,3.77169841709757e-07,1.87636812680445e-06,2.20830878353193e-06,2.84967915970626e-06,7.73529548293414e-06,1.69855565830348e-05,2.30416998462014e-05,3.79712980999139e-05,3.95847886473237e-05,8.23376999372831e-05,9.87879798138872e-05,0.000135810760812097,0.000199678135409531,0.000259130159086903,0.000265451073198926,0.000334605905150913,0.000337438558295145,0.000543870893035919,0.000549393231155963,0.000584685136057402,0.000687059807402205,0.000725073512497055,0.00148864010428755,0.00151657260147241,0.00155780805759672,0.0015995297026407,0.00162549463931316,0.00177564043877953,0.00199553384622393,0.00252587571821988,0.00334376462638275,0.00348453316347049,0.00359767789504219,0.00369287739281156,0.00408291693432901,0.00415439871142092,0.00420386949123069,0.0042505591939254,0.00427017629166617,0.00509374745272349,0.00510429032321777,0.00518781193605277,0.0052192341784125,0.00526133517515519,0.00551762010168503,0.00564494819847186,0.0060720389126425,0.00689410247227883,0.00831780781176375,0.00864213527737942,0.00948321408045502,0.010046042198675,0.0101799811001442,0.010551951818781,0.0109671999939873,0.0118837338346023,0.0126780609194099,0.014002734918743,0.0149909926785398,0.0151412426876172,0.0151413638494797,0.0157557228999469,0.0161376251735145,0.0162363908183788,0.0164856986393575,0.0171590561803436,0.0182460366785383,0.018520491342467,0.019334626909561,0.0196007373855631,0.0209826117464876,0.0244357806612771,0.0279640804657465,0.0295545033709727,0.029699199457374,0.0305587916510951,0.0323869675583945,0.0358101394795589,0.0361186031754692,0.0381677593909094,0.0383365754200703,0.0395212631664547,0.0403663083549209,0.0418358933068347,0.0423723086264434,0.0435018004088627,0.0447586203621661,0.0454344599520882,0.0469196995013483,0.0491608623463846,0.0519037335976734,0.056643993414036,0.0607704816823901,0.0620599671497553],[4.01900057198051e-16,1.71993125328403e-12,1.71993125328403e-12,4.52264041997965e-10,1.59567587433465e-09,2.61392164249728e-09,2.86383530293507e-09,2.19829640114906e-08,4.65803704951536e-07,8.99385580118449e-07,3.19310032226926e-06,7.89826085020031e-06,9.86444201394748e-06,4.55689402223939e-05,5.00549990933905e-05,6.0555682143758e-05,0.000154705909658683,0.000320838291012879,0.000412325155142551,0.000640896578099527,0.000640896578099527,0.00127249172630347,0.00146034404942268,0.00192398577817137,0.00271562264156962,0.00334271721806055,0.00334271721806055,0.00395617620070169,0.00395617620070169,0.00602560318042024,0.00602560318042024,0.0062122795706099,0.00707879801565908,0.00725073512497055,0.0141709789068327,0.0141709789068327,0.0141709789068327,0.0141709789068327,0.0141709789068327,0.015092943729626,0.0165483294564911,0.0204475653379705,0.0264390691388403,0.0269259380813629,0.0271824552069855,0.0272951807294768,0.0284678419444411,0.0284678419444411,0.0284678419444411,0.0284678419444411,0.0284678419444411,0.0319438207062994,0.0319438207062994,0.0319438207062994,0.0319438207062994,0.0319438207062994,0.0329121199047879,0.0330910756462143,0.0349914106830246,0.0390665806762467,0.0463615517376996,0.0473923547469194,0.0511792505929318,0.0532491319084468,0.0532491319084468,0.0543585396725083,0.055654447730682,0.0594186691730113,0.0624716045304256,0.0680132838910376,0.0705214206688097,0.0705214206688097,0.0705214206688097,0.0723911592700263,0.0726364852401156,0.0726364852401156,0.0727939939919681,0.074795885914318,0.0785272464645954,0.0787120882054848,0.0811576932006264,0.0812713501352615,0.0859528673952504,0.0989067312480262,0.111856321862986,0.116065836959852,0.116065836959852,0.118068058651958,0.123725494043305,0.134948627249006,0.134948627249006,0.140155221965848,0.140155221965848,0.142949249751006,0.144468893059717,0.14816878879504,0.148521494154544,0.150924613663401,0.153716473971075,0.1544771638371,0.157947503271866,0.163869541154615,0.171332712846689,0.185182286161272,0.196780607352501,0.199060271989781]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t0-versus-t5-t5---t0" class="section level3">
<h3>T0 versus T5 (T5 - T0)</h3>
<p>There is a confound between visits T5 and the other visit times. The T5 visit used the HGU133A 2.0 chip while the other visits used the HGU133 Plus 2.0 chip. Boxplots of normalized scores and MDS plots show a clear separation between the samples for the two chips (see Quality Control section).</p>
<div id="htmlwidget-8dc1c60bd7d5801d7665" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-8dc1c60bd7d5801d7665">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80"],["cell cycle (I) (M4.1)","cell cycle and transcription (M4.0)","enriched in T cells (I) (M7.0)","TLR and inflammatory signaling (M16)","enriched in NK cells (I) (M7.2)","mitotic cell division (M6)","enriched in monocytes (II) (M11.0)","enriched in activated dendritic cells (II) (M165)","cell division (stimulated CD4+ T cells) (M46)","cell cycle (II) (M4.10)","enriched in NK cells (II) (M61.0)","T cell activation (I) (M7.1)","E2F transcription factor network (M8)","cell division in stimulated CD4 T cells (M4.6)","Ran mediated mitosis (M15)","PLK1 signaling events (M4.2)","cytoskeletal remodeling (M32.8)","endoplasmic reticulum (M37.2)","transcription regulation in cell development (M49)","heme biosynthesis (I) (M171)","enriched in NK cells (receptor activation) (M61.2)","T cell differentiation (Th2) (M19)","platelet activation (II) (M32.1)","T cell activation and signaling (M5.1)","cell cycle (III) (M103)","C-MYC transcriptional network (M4.12)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","Monocyte surface signature (S4)","T cell differentiation via ITK and PKC (M18)","transmembrane transport (I) (M87)","IL2, IL7, TCR network (M65)","mitotic cell cycle in stimulated CD4 T cells (M4.5)","T cell activation (III) (M7.4)","mitotic cell cycle in stimulated CD4 T cells (M4.9)","putative targets of PAX3 (M89.0)","TBA (M184.0)","translation initiation factor 3 complex (M245)","activated dendritic cells (M67)","T cell surface signature (S0)","leukocyte activation and migration (M45)","antiviral IFN signature (M75)","TBA (M233)","T cell activation (II) (M7.3)","enriched in NK cells (KIR cluster) (M61.1)","interferon alpha response (I) (M158.0)","DNA repair (M76)","mitotic cell cycle - DNA replication (M4.4)","extracellular matrix (II) (M2.1)","T cell differentiation (M14)","immune activation - generic cluster (M37.0)","enriched in neutrophils (I) (M37.1)","type I interferon response (M127)","TBA (M26.1)","leukocyte differentiation (M160)","cell cell adhesion (M133.1)","TBA (M211)","TBA (M120)","small GTPase mediated signal transduction (M215)","mitotic cell cycle (M4.7)","ATF targets network (M41.4)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","TBA (M70.0)","MAPK, RAS signaling (M100)","myeloid cell enriched receptors and transporters (M4.3)","recruitment of neutrophils (M132)","proinflammatory dendritic cell, myeloid cell response (M86.1)","NK cell surface signature (S1)","signal transduction, plasma membrane (M82)","proinflammatory cytokines and chemokines (M29)","innate antiviral response (M150)","putative targets of PAX3 (M89.1)","TBA (source: naive B cells) (M152.1)","Plasma cell surface signature (S3)","growth factor induced, enriched in nuclear receptor subfamily 4 (M94)","TBA (M141)","TBA (M161)","TBA (M32.5)","leukocyte migration (M88.0)","TBA (M244)","myeloid cell cytokines, metallopeptidases and laminins (M78)"],[127,286,51,43,39,25,173,32,24,14,23,43,12,13,13,31,10,15,45,10,14,16,21,21,51,11,10,65,11,24,11,23,13,13,15,14,9,11,22,11,20,12,26,13,16,21,26,44,11,266,39,11,18,15,10,10,10,11,20,9,15,17,9,23,9,12,41,12,9,11,10,8,16,12,23,14,14,45,7,11],["Down","Down","Up","Down","Up","Down","Down","Down","Down","Down","Up","Up","Down","Down","Down","Down","Up","Down","Down","Down","Up","Up","Up","Up","Down","Down","Down","Down","Up","Down","Up","Down","Up","Down","Up","Down","Up","Down","Up","Up","Down","Down","Up","Up","Down","Down","Down","Down","Up","Down","Down","Down","Down","Up","Down","Up","Up","Up","Down","Down","Down","Down","Up","Down","Down","Down","Up","Down","Down","Down","Up","Up","Down","Up","Down","Down","Up","Down","Down","Down"],[4.58921745799143e-11,8.33252275821255e-09,2.40414748041111e-07,2.28797991499793e-06,2.37495828912013e-06,1.35121681577348e-05,1.44827076210224e-05,2.11948609339836e-05,3.75813686335825e-05,8.82571700768707e-05,0.000143680495807769,0.000163750755860435,0.000188624098696316,0.000289808462067941,0.000360596944761337,0.000565153364606508,0.00102204464402567,0.0011725992791094,0.00131245660957063,0.00145710798410156,0.00170014144708068,0.00316048410993594,0.00331387135659851,0.00371150972308037,0.00418110509957846,0.00442948534706315,0.00466056742268493,0.00577150555929956,0.00578671508096611,0.00600846760544906,0.00662901422131087,0.00666220180177293,0.00715794175561043,0.00772418216644515,0.00940845110365462,0.00994630710118035,0.0101255931449208,0.01069321526776,0.0106960763270799,0.0109481651204297,0.0112637454733099,0.0116641053936195,0.0118818748902444,0.0122527356384235,0.0124422238703467,0.012505757404347,0.012550061541747,0.013187295367025,0.0132655489561879,0.0133428658852111,0.014200982584064,0.0149864214144659,0.0171370046899094,0.0175267048094789,0.0176308565595769,0.0179340922050128,0.0209433756654706,0.0215018204691061,0.0226655543885553,0.0240683207108874,0.0241907180035639,0.0255900231505557,0.0265162191025558,0.0276346188768696,0.0283339014725638,0.0307471251558435,0.0310430728822177,0.0318152402412818,0.0345553169842496,0.0351176716525174,0.0357087579582146,0.0388368673615956,0.0390080505378483,0.0390998669036972,0.0413740806161175,0.0439000938772656,0.0442996998498335,0.0465256710133862,0.0465838398209973,0.0468280821105799],[1.56033393571709e-08,1.41652886889613e-06,2.72470047779926e-05,0.000161497163660169,0.000161497163660169,0.000703445798735375,0.000703445798735375,0.000900781589694303,0.00141974059282423,0.0030007437826136,0.0044410335067856,0.00463960474937898,0.00493324565821133,0.00703820550736428,0.00817353074792363,0.0120095089978883,0.0204408928805135,0.0221490974942888,0.0234860656449481,0.0247708357297265,0.0275260996194015,0.0488438453353736,0.0489876635323258,0.052579721076972,0.0568630293542671,0.0579240391539028,0.0586886268041806,0.0678442457768441,0.0678442457768441,0.0680959661950893,0.0707858941438374,0.0707858941438374,0.0737484908153802,0.0772418216644515,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0907314880194356,0.0946732172270934,0.0979881400176614,0.108885559816149,0.108885559816149,0.108885559816149,0.108885559816149,0.124925398706316,0.12604515447407,0.130615059188285,0.134833510183799,0.134833510183799,0.140332385019176,0.14310340468046,0.14680891278337,0.148208100010334,0.157532011641105,0.157532011641105,0.159076201206409,0.170272576444128,0.170571548026513,0.170999685997084,0.179648037125095,0.179648037125095,0.179648037125095,0.187562498793066,0.195609064271992,0.195609064271992,0.199019348969965,0.199019348969965,0.199019348969965]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="heatmap-of-significant-gene-sets" class="section level1">
<h1>Heatmap of significant gene-sets</h1>
<p>FDR cutoff &lt;= 0.01</p>
<img src="figure/GSEA_T0_Interaction.Rmd/heatmap-1.png" width="672" style="display: block; margin: auto;" />
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
geneset
</th>
<th style="text-align:center;">
NGenes
</th>
<th style="text-align:center;">
Direction
</th>
<th style="text-align:center;">
PValue
</th>
<th style="text-align:center;">
FDR
</th>
<th style="text-align:center;">
time
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
173
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells (II) (M165)
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and transcription (M4.0)
</td>
<td style="text-align:center;">
286
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
type I interferon response (M127)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
innate antiviral response (M150)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
antiviral IFN signature (M75)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
activated dendritic cells (M67)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000005
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
Monocyte surface signature (S4)
</td>
<td style="text-align:center;">
65
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000009
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.00e-07
</td>
<td style="text-align:center;">
0.0000032
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
cell junction (GO) (M4.13)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
3.00e-07
</td>
<td style="text-align:center;">
0.0000079
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
RIG-1 like receptor signaling (M68)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
4.00e-07
</td>
<td style="text-align:center;">
0.0000099
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
173
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.00e-07
</td>
<td style="text-align:center;">
0.0000440
</td>
<td style="text-align:center;">
T3
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid cell enriched receptors and transporters (M4.3)
</td>
<td style="text-align:center;">
23
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.90e-06
</td>
<td style="text-align:center;">
0.0000456
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in T cells (I) (M7.0)
</td>
<td style="text-align:center;">
51
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
2.20e-06
</td>
<td style="text-align:center;">
0.0000501
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
MHC-TLR7-TLR8 cluster (M146)
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
2.80e-06
</td>
<td style="text-align:center;">
0.0000606
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
2.00e-07
</td>
<td style="text-align:center;">
0.0000712
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in neutrophils (I) (M37.1)
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
5.00e-07
</td>
<td style="text-align:center;">
0.0000906
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M66)
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
7.70e-06
</td>
<td style="text-align:center;">
0.0001547
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells (II) (M165)
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.00e-06
</td>
<td style="text-align:center;">
0.0001616
</td>
<td style="text-align:center;">
T3
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (IV) (M118.0)
</td>
<td style="text-align:center;">
44
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.70e-05
</td>
<td style="text-align:center;">
0.0003208
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in antigen presentation (II) (M95.0)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
2.30e-05
</td>
<td style="text-align:center;">
0.0004123
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells (II) (M165)
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
4.20e-06
</td>
<td style="text-align:center;">
0.0004807
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
173
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
6.70e-06
</td>
<td style="text-align:center;">
0.0005704
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (I) (M7.2)
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
8.40e-06
</td>
<td style="text-align:center;">
0.0005742
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
regulation of antigen presentation and immune response (M5.0)
</td>
<td style="text-align:center;">
73
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
3.80e-05
</td>
<td style="text-align:center;">
0.0006409
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
formyl peptide receptor mediated neutrophil response (M11.2)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
3.96e-05
</td>
<td style="text-align:center;">
0.0006409
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
173
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
2.30e-06
</td>
<td style="text-align:center;">
0.0007874
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
8.80e-06
</td>
<td style="text-align:center;">
0.0010019
</td>
<td style="text-align:center;">
T3
</td>
</tr>
</tbody>
</table>
<div id="summary" class="section level2">
<h2>Summary</h2>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
visit
</th>
<th style="text-align:center;">
significant genesets
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
T1
</td>
<td style="text-align:center;">
67
</td>
</tr>
<tr>
<td style="text-align:center;">
T2
</td>
<td style="text-align:center;">
50
</td>
</tr>
<tr>
<td style="text-align:center;">
T3
</td>
<td style="text-align:center;">
86
</td>
</tr>
<tr>
<td style="text-align:center;">
T4
</td>
<td style="text-align:center;">
106
</td>
</tr>
<tr>
<td style="text-align:center;">
T5
</td>
<td style="text-align:center;">
80
</td>
</tr>
</tbody>
</table>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 3.5.3 (2019-03-11)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.5 LTS

Matrix products: default
BLAS/LAPACK: /app/easybuild/software/OpenBLAS/0.2.18-GCC-5.4.0-2.26-LAPACK-3.6.1/lib/libopenblas_prescottp-r0.2.18.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] heatmap3_1.1.6       magrittr_1.5         DT_0.5              
 [4] limma_3.38.3         RNASeqUtilities_1.1  here_0.1            
 [7] data.table_1.12.2    forcats_0.4.0        stringr_1.4.0       
[10] dplyr_0.8.0.1        purrr_0.3.2          readr_1.3.1         
[13] tidyr_0.8.3          tibble_2.1.1         ggplot2_3.1.1       
[16] tidyverse_1.2.1      GSEABase_1.44.0      graph_1.60.0        
[19] annotate_1.60.1      XML_3.98-1.19        AnnotationDbi_1.44.0
[22] IRanges_2.16.0       S4Vectors_0.20.1     Biobase_2.42.0      
[25] BiocGenerics_0.28.0  knitr_1.22          

loaded via a namespace (and not attached):
 [1] nlme_3.1-137       bitops_1.0-6       fs_1.2.7          
 [4] lubridate_1.7.4    bit64_0.9-7        webshot_0.5.1     
 [7] httr_1.4.0         rprojroot_1.3-2    tools_3.5.3       
[10] backports_1.1.3    R6_2.4.0           DBI_1.0.0         
[13] lazyeval_0.2.1     colorspace_1.4-1   withr_2.1.2       
[16] tidyselect_0.2.5   gridExtra_2.3      bit_1.1-14        
[19] compiler_3.5.3     git2r_0.25.2       cli_1.1.0         
[22] rvest_0.3.2        xml2_1.2.0         scales_1.0.0      
[25] digest_0.6.18      rmarkdown_1.12     pkgconfig_2.0.2   
[28] htmltools_0.3.6    htmlwidgets_1.3    rlang_0.3.4       
[31] readxl_1.3.0       rstudioapi_0.10    RSQLite_2.1.1     
[34] shiny_1.3.1        generics_0.0.2     jsonlite_1.6      
[37] crosstalk_1.0.0    RCurl_1.95-4.12    kableExtra_1.1.0  
[40] Rcpp_1.0.1         munsell_0.5.0      stringi_1.4.3     
[43] yaml_2.2.0         plyr_1.8.4         grid_3.5.3        
[46] blob_1.1.1         promises_1.0.1     crayon_1.3.4      
[49] lattice_0.20-38    haven_2.1.0        hms_0.4.2         
[52] pillar_1.3.1       fastcluster_1.1.25 reshape2_1.4.3    
[55] glue_1.3.1         evaluate_0.13      modelr_0.1.4      
[58] httpuv_1.5.1       cellranger_1.1.0   gtable_0.2.0      
[61] assertthat_0.2.0   xfun_0.6           mime_0.6          
[64] xtable_1.8-3       broom_0.5.2        later_0.8.0       
[67] viridisLite_0.3.0  memoise_1.1.0      workflowr_1.3.0   </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
