<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Carl Murie and Raphael Gottardo" />

<meta name="date" content="2019-04-29" />

<title>Protected vs Non-Protected</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<script src="site_libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.6/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">VaheyProject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Protected vs Non-Protected</h1>
<h4 class="author">Carl Murie and Raphael Gottardo</h4>
<h4 class="date">April 29, 2019</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-05-28
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>VaheyProject/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.3.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20181008code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20181008)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20181008code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20181008)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong1ed83ed"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 1ed83ed </a>
</p>
</div>
<div id="strongRepositoryversionstrong1ed83ed" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    data/.Rhistory

Untracked files:
    Untracked:  VaheyReport.tar.gz
    Untracked:  analysis/DEG_Disease.Rmd
    Untracked:  analysis/DEG_T0.Rmd
    Untracked:  analysis/DEG_T0_Interaction.Rmd
    Untracked:  analysis/DEG_T1.Rmd
    Untracked:  analysis/DEG_T1_Interaction.Rmd
    Untracked:  analysis/GSEA_Disease.Rmd
    Untracked:  analysis/GSEA_T0.Rmd
    Untracked:  analysis/GSEA_T0_Interaction.Rmd
    Untracked:  analysis/GSEA_T1.Rmd
    Untracked:  analysis/GSEA_T1_Interaction.Rmd
    Untracked:  analysis/analysis_T4_disease.Rmd
    Untracked:  analysis/analysis_T5_disease.Rmd
    Untracked:  analysis/ciber_MLratio.Rmd
    Untracked:  analysis/pamr.Rmd
    Untracked:  code/processGEOFiles.R
    Untracked:  data/BTM_for_GSEA_20131008.gmt
    Untracked:  data/GPL570.soft
    Untracked:  data/GPL571.soft
    Untracked:  data/GSE18323-GPL570_series_matrix.txt.gz
    Untracked:  data/GSE18323-GPL571_series_matrix.txt.gz
    Untracked:  data/gpl_eset.Rds
    Untracked:  data/h.all.v5.0.symbols.gmt
    Untracked:  data/vahey_ciber_cells.csv
    Untracked:  data/vahey_exprs.tsv
    Untracked:  docs/figure/DEG_Disease.Rmd/
    Untracked:  docs/figure/DEG_T0.Rmd/
    Untracked:  docs/figure/DEG_T0_Interaction.Rmd/
    Untracked:  docs/figure/DEG_T1.Rmd/
    Untracked:  docs/figure/DEG_T1_Interaction.Rmd/
    Untracked:  docs/figure/GSEA_Disease.Rmd/
    Untracked:  docs/figure/GSEA_T0.Rmd/
    Untracked:  docs/figure/GSEA_T0_Interaction.Rmd/
    Untracked:  docs/figure/GSEA_T1.Rmd/
    Untracked:  docs/figure/GSEA_T1_Interaction.Rmd/
    Untracked:  docs/figure/ciber_MLratio.Rmd/
    Untracked:  docs/figure/pamr.Rmd/
    Untracked:  output/Vahey_GSEA_disease.csv
    Untracked:  output/dmso_M3_both_disease.csv
    Untracked:  output/dmso_M3_old_disease.csv
    Untracked:  output/dmso_M3_young_disease.csv
    Untracked:  output/pam_res.Rda

Unstaged changes:
    Modified:   analysis/analysis_PCA.Rmd
    Modified:   analysis/experimental_design.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/processVahey.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with <code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="gsea-analysis" class="section level1">
<h1>GSEA Analysis</h1>
<p>The Blood Transcript Modules (BTM) from Shuzhao et al. Molecular signatures of antibody responses derived from a systems biological study of 5 human vaccines, 2014 were used for the GSEA analysis.</p>
<div id="disease-effect-nonprotected---protected-for-t0-to-t5" class="section level2">
<h2>Disease effect (NonProtected - Protected) for T0 to T5</h2>
<ul>
<li><strong>visit</strong>
<ul>
<li>T0 - study entry (baseline)</li>
<li>T1 - day of third vaccination</li>
<li>T2 - 24h after T1</li>
<li>T3 - 72h after T1</li>
<li>T4 - 2 weeks after T1</li>
<li>T5 - 5 days after challenge</li>
</ul></li>
<li><strong>disease</strong> - response to malaria challenge
<ul>
<li>Protected - vaccinated and protected from challenge<br />
</li>
<li>NonProtected - vaccinated and not protected from challenge</li>
</ul></li>
</ul>
<p>A FDR cutoff of &lt;= 0.2 was used.</p>
<p>linear model: ~disease<br />
coefficient: diseaseNonProtected</p>
</div>
<div id="sample-size" class="section level2 tabset tabset-fade tabset-pills">
<h2>Sample size</h2>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
disease
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
total
</th>
<th style="text-align:center;">
Protected
</th>
<th style="text-align:center;">
NonProtected
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
T0
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
T1
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
T2
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
T3
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
T4
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
T5
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
26
</td>
</tr>
</tbody>
</table>
</div>
<div id="comparisons" class="section level2 tabset tabset-fade tabset-pills">
<h2>Comparisons</h2>
<div id="t0" class="section level3">
<h3>T0</h3>
<div id="htmlwidget-9c23e70c978f4c72e429" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9c23e70c978f4c72e429">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117"],["enriched in monocytes (II) (M11.0)","TLR and inflammatory signaling (M16)","enriched in activated dendritic cells (II) (M165)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","enriched in T cells (I) (M7.0)","enriched in monocytes (IV) (M118.0)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","TBA (M148)","TBA (M214)","regulation of transcription, transcription factors (M213)","TBA (M174)","antiviral IFN signature (M75)","innate antiviral response (M150)","Monocyte surface signature (S4)","enriched in antigen presentation (II) (M95.0)","cell cycle and transcription (M4.0)","TBA (M161)","inositol phosphate metabolism (M129)","enriched in monocytes (I) (M4.15)","myeloid cell enriched receptors and transporters (M4.3)","phosphatidylinositol signaling system (M101)","type I interferon response (M127)","intracellular transport (M147)","TBA (M205)","TBA (M66)","TBA (M72.0)","TBA (M26.0)","proinflammatory cytokines and chemokines (M29)","RIG-1 like receptor signaling (M68)","TBA (M185)","enriched in activated dendritic cells/monocytes (M64)","cytoskeletal remodeling (M32.8)","cell junction (GO) (M4.13)","activated dendritic cells (M67)","T cell activation (I) (M7.1)","golgi membrane (II) (M237)","proteasome (M226)","enriched in monocytes (III) (M73)","T cell differentiation (Th2) (M19)","TBA (M137)","TBA (M180)","IL2, IL7, TCR network (M65)","Wnt signaling pathway (M206)","lysosome (M209)","TBA (M153)","proinflammatory dendritic cell, myeloid cell response (M86.1)","enriched in antigen presentation (III) (M95.1)","TBA (M48)","nuclear pore complex (mitosis) (M106.1)","TBA (M26.2)","TBA (M236)","enriched for ubiquitination (M138)","plasma cells, immunoglobulins (M156.1)","endoplasmic reticulum (M37.2)","complement and other receptors in DCs (M40)","formyl peptide receptor mediated neutrophil response (M11.2)","enriched in neutrophils (II) (M163)","platelet activation &amp; blood coagulation (M199)","MHC-TLR7-TLR8 cluster (M146)","TBA (M194)","T cell surface signature (S0)","antigen processing and presentation (M200)","innate activation by cytosolic DNA sensing (M13)","platelet activation (II) (M32.1)","TBA (M79)","blood coagulation (M11.1)","enriched in antigen presentation (I) (M71)","T cell differentiation via ITK and PKC (M18)","chromosome Y linked (M240)","BCR signaling (M54)","TBA (M128)","extracellular matrix (I) (M2.0)","enriched in activated dendritic cells (I) (M119)","TLR8-BAFF network (M25)","T cell differentiation (M14)","signaling in T cells (II) (M35.1)","TBA (M177.1)","Resting dendritic cell surface signature (S10)","T cell activation (II) (M7.3)","plasma cells &amp; B cells, immunoglobulins (M156.0)","platelet activation - actin binding (M196)","TBA (M218)","B cell surface signature (S2)","TBA (M72.2)","transcription elongation, RNA polymerase II (M234)","inflammasome receptors and signaling (M53)","DC surface signature (S5)","suppression of MAPK signaling (M56)","lysosomal/endosomal proteins (M139)","TBA (M32.5)","T cell activation and signaling (M5.1)","enriched in extracellular matrix &amp; associated proteins (M202)","RA, WNT, CSF receptors network (monocyte) (M23)","cell cycle, ATP binding (M144)","B cell development/activation (M58)","Naive B cell surface signature (S8)","chemokines and inflammatory molecules in myeloid cells (M86.0)","TBA (M26.1)","enriched in B cells (I) (M47.0)","double positive thymocytes (M126)","TBA (M72.1)","TBA (M233)","cytoskeleton/actin (SRF transcription targets) (M145.1)","T cell activation (III) (M7.4)","amino acid metabolishm and transport (M154.0)","recruitment of neutrophils (M132)","regulation of antigen presentation and immune response (M5.0)","enriched in dendritic cells (M168)","extracellular region cluster (GO) (M189)","immune activation - generic cluster (M37.0)","MAPK, RAS signaling (M100)","enriched in neutrophils (I) (M37.1)","cytoskeleton/actin (SRF transcription targets) (M145.0)","cytoskeletal remodeling (enriched for SRF targets) (M34)","G protein coupled receptors cluster (M155)","enriched for cell migration (M122)","CCR1, 7 and cell signaling (M59)"],[173,43,32,10,51,44,15,11,12,14,21,20,11,65,20,286,14,12,10,23,13,11,16,11,16,21,26,9,9,14,15,10,10,11,43,13,10,11,16,15,11,11,14,7,12,12,12,12,11,10,10,8,11,15,12,9,12,10,16,13,22,6,11,21,8,19,16,11,9,12,11,30,10,8,11,10,8,61,26,24,15,11,137,12,13,10,65,11,10,14,21,11,11,15,11,38,18,18,37,10,16,12,13,13,23,9,73,13,15,266,9,39,14,10,9,10,10],["Up","Up","Up","Up","Down","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Up","Up","Up","Down","Up","Up","Down","Up","Up","Up","Down","Up","Up","Down","Down","Up","Up","Up","Up","Up","Up","Down","Up","Up","Down","Up","Up","Up","Up","Down","Up","Up","Down","Up","Up","Down","Up","Up","Up","Down","Up","Up","Up","Down","Up","Up","Down","Up","Up","Up","Down","Down","Down","Up","Down","Up","Down","Up","Up","Up","Up","Down","Down","Down","Up","Up","Up","Down","Up","Down","Down","Down","Up","Down","Down","Down","Down","Up","Up","Up","Down","Up","Down","Up","Down","Down","Up","Down","Up"],[9.86029004778698e-11,1.27968512060143e-09,2.68162831801266e-09,4.63387197660422e-06,2.03397229528451e-05,2.19047642406614e-05,2.31293233828559e-05,5.73190877863612e-05,7.25008067825045e-05,7.71929144392195e-05,0.000124392380853005,0.000137857477304314,0.000176590255745342,0.000187128316306207,0.000192263054091728,0.000198845354288031,0.000237267951560986,0.000306785677195632,0.000351745281478747,0.000356971256520511,0.000382200930733,0.000382974065182746,0.000527728133222904,0.000685228208354672,0.000774180372461224,0.00101314595840664,0.0010291456529641,0.00127227647839718,0.00151632745905186,0.00168494303482513,0.00169892033637069,0.00182585863642936,0.00203927921591214,0.00226722739840339,0.00246530257756584,0.00274691048071523,0.00285789481381763,0.00335803167714414,0.00343179622790836,0.00392630239945744,0.00414593201372303,0.00456624085568112,0.00504443534752478,0.00526195271447105,0.00535578650366076,0.00566487213591881,0.00602547154526045,0.00711102032686391,0.00719482703418044,0.0073104185531685,0.00735171689049993,0.00789894361043217,0.00791293767068708,0.00814889975583766,0.00815632596136975,0.00820098141777591,0.00851868175090644,0.00870877416552867,0.00931921083521358,0.0094101499819539,0.00956069715804014,0.0100274163783494,0.0102674998927255,0.0109785530047013,0.0111411182380382,0.0114881358853234,0.0115386355885886,0.0140903882863586,0.0148483890326889,0.0149404760667341,0.0151988386332092,0.0166647679646745,0.0166721730733142,0.016740619408205,0.0170486377219189,0.018051364104729,0.0193573097539846,0.0197088638452628,0.0197444564753373,0.0209479086675718,0.0215654008215423,0.0238857275153961,0.0246694520123842,0.0251842635191765,0.0266316791207908,0.0274397404207515,0.0285035923693116,0.0297170004909187,0.0310566406565613,0.0328060038250629,0.0343122287773648,0.0353877659097143,0.0361207726126358,0.0371102591865247,0.0396895162653374,0.0408537225400718,0.0425600729783239,0.0437557630942152,0.044743774728881,0.0464760359010699,0.0481303948423395,0.0482652097117985,0.0515536734332906,0.0517397743264667,0.0526358009015027,0.0536819811119867,0.0562291217517134,0.0587695199678863,0.0588685046463332,0.05944749477395,0.0596732606243654,0.0601318222236897,0.0614310834303366,0.06191687016193,0.0634994940057226,0.0679956274356582,0.0685050598290118],[3.35249861624757e-08,2.17546470502243e-07,3.03917876041435e-07,0.000393879118011359,0.00112342427859586,0.00112342427859586,0.00112342427859586,0.00243606123092035,0.00262455909093346,0.00262455909093346,0.00384485540818378,0.00390596185695556,0.00422546377862066,0.00422546377862066,0.00422546377862066,0.00422546377862066,0.00474535903121973,0.00579484056925083,0.0059186900982788,0.0059186900982788,0.0059186900982788,0.0059186900982788,0.00780119849112119,0.00970739961835786,0.0105288530654726,0.0129596119262146,0.0129596119262146,0.0154490715233944,0.017777632278539,0.0186333198182591,0.0186333198182591,0.019399748012062,0.0210107555578826,0.0226722739840339,0.0239486536106396,0.0259430434289772,0.0262617361269728,0.0299182235253549,0.0299182235253549,0.0333735703953882,0.0343808996259959,0.0369648069269424,0.0398862329804285,0.0404659424721035,0.0404659424721035,0.0418707940480956,0.0435885175614586,0.0490114459366662,0.0490114459366662,0.0490114459366662,0.0490114459366662,0.0497916728936394,0.0497916728936394,0.0497916728936394,0.0497916728936394,0.0497916728936394,0.0508131893913717,0.051051434763444,0.0532891317005516,0.0532891317005516,0.0532891317005516,0.0549890575586901,0.0554119041829632,0.0582766184758922,0.0582766184758922,0.0585542701510465,0.0585542701510465,0.070451941431793,0.0725680266098512,0.0725680266098512,0.0727831709195931,0.0769163594431039,0.0769163594431039,0.0769163594431039,0.0772871576726989,0.0807561025737874,0.0849761417925909,0.0849761417925909,0.0849761417925909,0.0890286118371802,0.0905214355472147,0.0990383823809105,0.101055586556755,0.101936304720476,0.106526716483163,0.108482694686692,0.111393349489264,0.114815683714913,0.118643346328437,0.123933792228015,0.128199536091253,0.130780874014161,0.132054437508561,0.134228597057642,0.142046689791734,0.144690267329421,0.14917963724361,0.151805708694216,0.153665488967874,0.158018522063638,0.160884032372662,0.160884032372662,0.169149262221141,0.169149262221141,0.170439736252485,0.172187486585618,0.17867197565965,0.182543031750486,0.182543031750486,0.182543031750486,0.182543031750486,0.182543031750486,0.184664349605756,0.184664349605756,0.187737634451702,0.199074532836444,0.199074532836444]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t1" class="section level3">
<h3>T1</h3>
<div id="htmlwidget-a0e6efde7f4990d8641e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-a0e6efde7f4990d8641e">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17"],["enriched in NK cells (KIR cluster) (M61.1)","NK cell surface signature (S1)","enriched in NK cells (II) (M61.0)","enriched in NK cells (I) (M7.2)","enriched in NK cells (receptor activation) (M61.2)","enriched in B cells (II) (M47.1)","innate antiviral response (M150)","TBA (M125)","enriched in B cells (I) (M47.0)","enriched in B cells (VI) (M69)","antiviral IFN signature (M75)","regulation of localization (GO) (M63)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","TBA (M236)","erythrocyte differentiation (M173)","extracellular matrix (II) (M2.1)","TBA (M188)"],[13,41,23,39,14,28,11,10,37,17,20,12,10,10,14,44,6],["Up","Up","Up","Up","Up","Down","Up","Down","Down","Down","Up","Up","Up","Down","Up","Down","Down"],[8.45549078229716e-13,3.40744231057967e-10,5.28365559828754e-09,3.39572303387217e-06,3.61661330692393e-05,0.000288467773328608,0.000437551659588975,0.00044713803099343,0.000507783539917886,0.00109380573130642,0.00116700549180246,0.00267866876438353,0.00294032222692723,0.0060198401620876,0.00635499028862369,0.00636912812710488,0.00727239313209772],[2.87486686598103e-10,5.79265192798544e-08,5.98814301139254e-07,0.000288636457879134,0.00245929704870827,0.0163465071552878,0.0190033663172208,0.0190033663172208,0.0191829337302313,0.0360710788375307,0.0360710788375307,0.0758956149908668,0.076900735165789,0.135343972700979,0.135343972700979,0.135343972700979,0.145447862641955]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t2" class="section level3">
<h3>T2</h3>
<div id="htmlwidget-c47734cf8f874295dd41" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-c47734cf8f874295dd41">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44"],["enriched in NK cells (KIR cluster) (M61.1)","TBA (M125)","enriched in NK cells (II) (M61.0)","NK cell surface signature (S1)","TBA (M70.0)","enriched in NK cells (I) (M7.2)","TBA (M184.1)","interferon alpha response (I) (M158.0)","enriched in neutrophils (I) (M37.1)","TBA (M70.1)","platelet activation &amp; blood coagulation (M199)","platelet activation - actin binding (M196)","cell junction (M162.1)","innate antiviral response (M150)","enriched in NK cells (receptor activation) (M61.2)","TBA (M186)","TBA (M131)","TBA (source: B cells) (M152.0)","TBA (M184.0)","extracellular matrix (I) (M2.0)","B cell surface signature (S2)","plasma membrane, cell junction (M162.0)","leukocyte migration (M88.0)","enriched in membrane proteins (M124)","TBA (M205)","type I interferon response (M127)","enriched in myeloid cells and monocytes (M81)","myeloid cell cytokines, metallopeptidases and laminins (M78)","TBA (M193)","TBA (source: naive B cells) (M152.1)","TBA (M203)","cell adhesion (M51)","activated dendritic cells (M67)","cytoskeleton/actin (SRF transcription targets) (M145.1)","immune activation - generic cluster (M37.0)","cell movement, Adhesion &amp; Platelet activation (M30)","TBA (source: memory B cells) (M152.2)","TBA (M114.0)","TBA (M188)","chromosome Y linked (M240)","signaling in T cells (II) (M35.1)","TBA (M141)","extracellular matrix (II) (M2.1)","cell adhesion (lymphocyte homing) (M21)"],[13,10,23,41,17,39,11,16,39,9,10,15,11,11,14,9,11,16,14,30,137,16,45,16,11,11,34,11,9,8,10,35,11,13,266,17,10,30,6,9,10,23,44,10],["Up","Down","Up","Up","Down","Up","Down","Down","Down","Down","Down","Down","Down","Up","Up","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Up","Down","Down","Down","Down","Up","Down","Up","Down","Down","Down","Down","Down","Down","Up","Up","Down","Down","Down"],[2.84558194577288e-08,8.29829338798655e-08,1.37304609569055e-06,2.36662321736737e-06,5.45135726888121e-05,7.72527803567256e-05,9.96195835661514e-05,0.000213148662116175,0.000237324198274078,0.000594836303310417,0.000731145244347059,0.000982243293128733,0.0010643954126154,0.00126360646719321,0.00219651185647838,0.0024508736322821,0.00284954328505242,0.00332158019565568,0.00336371895605845,0.00403571052658532,0.00419113397434297,0.0052959638738013,0.00661524769295235,0.00667732377930887,0.00865510117713458,0.00898446927966023,0.0102296045367455,0.0120074925963991,0.0120982599801754,0.0122742325600213,0.012959754225802,0.0129657253246274,0.0133779003693653,0.0144107329822313,0.0146163653199729,0.0146682364440894,0.0149150342117002,0.0151274377632982,0.0160553438679003,0.0190379935356596,0.0199170276231747,0.0216769399725386,0.0236375859680664,0.025170253550483],[9.67497861562779e-06,1.41070987595771e-05,0.000155611890844929,0.000201162973476227,0.00370692294283922,0.00437765755354778,0.00483866548749878,0.0089655808236874,0.0089655808236874,0.0202244343125542,0.0225990348252727,0.0278302266386474,0.0278380338684028,0.0306875856318352,0.0497876020801766,0.0520810646859945,0.0569908657010484,0.060192865529467,0.060192865529467,0.0678564548226957,0.0678564548226957,0.0818467144132929,0.0945954202068757,0.0945954202068757,0.117489213657095,0.117489213657095,0.128817242314573,0.135350758934774,0.135350758934774,0.135350758934774,0.135350758934774,0.135350758934774,0.135350758934774,0.135350758934774,0.135350758934774,0.135350758934774,0.135350758934774,0.135350758934774,0.139969664489387,0.161822945053106,0.165165594923888,0.175479990253884,0.186901842538199,0.194497413799187]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t3" class="section level3">
<h3>T3</h3>
<div id="htmlwidget-14032f3f90232640cb54" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-14032f3f90232640cb54">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52"],["enriched in NK cells (KIR cluster) (M61.1)","NK cell surface signature (S1)","enriched in NK cells (II) (M61.0)","TBA (M125)","enriched in NK cells (I) (M7.2)","C-MYC transcriptional network (M4.12)","regulation of localization (GO) (M63)","platelet activation - actin binding (M196)","signaling in T cells (II) (M35.1)","TBA (M70.0)","TBA (M184.0)","TBA (M136)","platelet activation (I) (M32.0)","cell cycle (I) (M4.1)","chromosome Y linked (M240)","cell cycle and transcription (M4.0)","Rho GTPase cycle (M4.14)","platelet activation (II) (M32.1)","antiviral IFN signature (M75)","cell movement, Adhesion &amp; Platelet activation (M30)","innate antiviral response (M150)","TBA (M244)","leukocyte differentiation (M160)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","translation initiation (M227)","TBA (M211)","cytoskeletal remodeling (enriched for SRF targets) (M34)","platelet activation &amp; blood coagulation (M199)","interferon alpha response (II) (M158.1)","TBA (M32.7)","G protein mediated calcium signaling (M159)","CORO1A-DEF6 network (I) (M32.2)","enriched in cell cycle (M167)","cell cycle, ATP binding (M144)","TBA (M186)","activated dendritic cells (M67)","RIG-1 like receptor signaling (M68)","TBA (M72.1)","cell cycle (II) (M4.10)","extracellular matrix (I) (M2.0)","TBA (M104)","signaling in T cells (I) (M35.0)","MHC-TLR7-TLR8 cluster (M146)","PLK1 signaling events (M4.2)","BCR signaling (M54)","TBA (M48)","TBA (M72.2)","type I interferon response (M127)","TBA (M72.0)","cell adhesion (M51)","TBA (M70.1)","TBA (M192)"],[13,41,23,10,39,11,12,15,10,17,14,16,22,127,9,286,8,21,20,17,11,7,15,10,10,10,10,10,12,11,9,19,13,15,9,11,9,16,14,30,9,14,16,31,12,12,12,11,21,35,9,12],["Up","Up","Up","Down","Up","Up","Up","Down","Up","Down","Down","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Down","Up","Up","Down","Down","Down","Down","Up","Up","Down","Up","Up","Down","Down","Up","Up","Down","Up","Down","Down","Up","Up","Up","Up","Up","Down","Up","Down","Down","Down","Up"],[2.26054089481973e-06,4.48668219659917e-06,1.9499874482578e-05,7.5401047859403e-05,0.000192462190903558,0.00126045140456076,0.0012840826230026,0.00145036044184225,0.00153707467591874,0.00215696334159092,0.00215750345626534,0.00233461986355892,0.00303366957663628,0.00321578082412709,0.00330365447311048,0.00348615820362319,0.00371809109267705,0.0037567238088912,0.00387179287834203,0.00435083377269518,0.00530696776750719,0.00555802375708145,0.00662550270511934,0.00704952860565925,0.00824248112396531,0.00992823674234137,0.00994533761038785,0.0110966502641236,0.0117135635793922,0.0119011496514441,0.0130508052723214,0.0134164658269878,0.0145579493660573,0.0156555716766523,0.0163761504334293,0.0176346904691494,0.0194275204266552,0.0205786806973375,0.0211216932959831,0.0213729588995429,0.0224539219501027,0.0234311123738718,0.0236950327191365,0.0252421853625353,0.0253234601619317,0.0254524477330074,0.025492413900991,0.0263324265000631,0.0271135060521772,0.0280284137367108,0.0294000971854799,0.0298254316724945],[0.000762735973421858,0.000762735973421858,0.00220998577469218,0.00640908906804926,0.0130874289814419,0.0580672655347079,0.0580672655347079,0.0580672655347079,0.0580672655347079,0.066147562800836,0.066147562800836,0.066147562800836,0.0692847146650679,0.0692847146650679,0.0692847146650679,0.0692847146650679,0.0692847146650679,0.0692847146650679,0.0692847146650679,0.0739641741358181,0.0858967307912588,0.0858967307912588,0.0979422139017641,0.099868321913506,0.112097743285928,0.125237584723403,0.125237584723403,0.134745038921501,0.1348796960497,0.1348796960497,0.142549949411746,0.142549949411746,0.149990993468469,0.156555716766523,0.159082604210456,0.166549854430855,0.178523160677372,0.181670150646114,0.181670150646114,0.181670150646114,0.184413206943339,0.184413206943339,0.184413206943339,0.184413206943339,0.184413206943339,0.184413206943339,0.184413206943339,0.186521354375447,0.188134531790617,0.190593213409634,0.195012437858618,0.195012437858618]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t4" class="section level3">
<h3>T4</h3>
<div id="htmlwidget-8dc1c60bd7d5801d7665" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-8dc1c60bd7d5801d7665">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83"],["TBA (M136)","regulation of localization (GO) (M63)","type I interferon response (M127)","platelet activation (I) (M32.0)","TBA (M192)","platelet activation (II) (M32.1)","innate antiviral response (M150)","enriched in monocytes (II) (M11.0)","cell cycle and transcription (M4.0)","antiviral IFN signature (M75)","activated dendritic cells (M67)","enriched in activated dendritic cells (II) (M165)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","KLF12 targets network (M32.3)","TBA (M32.5)","TBA (M55)","cytoskeletal remodeling (M32.8)","CORO1A-DEF6 network (II) (M32.4)","TLR and inflammatory signaling (M16)","TBA (M32.6)","enriched in NK cells (KIR cluster) (M61.1)","Monocyte surface signature (S4)","CORO1A-DEF6 network (I) (M32.2)","RIG-1 like receptor signaling (M68)","cell junction (GO) (M4.13)","NK cell surface signature (S1)","cell division in stimulated CD4 T cells (M4.6)","enriched in NK cells (II) (M61.0)","cell cycle (I) (M4.1)","regulation of antigen presentation and immune response (M5.0)","enriched in NK cells (I) (M7.2)","MHC-TLR7-TLR8 cluster (M146)","MAPK, RAS signaling (M100)","TBA (M32.7)","translation initiation factor 3 complex (M245)","enriched in NK cells (receptor activation) (M61.2)","E2F transcription factor network (M8)","RA, WNT, CSF receptors network (monocyte) (M23)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","Hox cluster III (M17.2)","regulation of transcription, transcription factors (M213)","enriched in myeloid cells and monocytes (M81)","formyl peptide receptor mediated neutrophil response (M11.2)","platelet activation &amp; blood coagulation (M199)","small GTPase mediated signal transduction (M215)","TBA (M41.2)","regulation of signal transduction (M3)","myeloid cell enriched receptors and transporters (M4.3)","TBA (M194)","Hox cluster IV (M17.3)","chaperonin mediated protein folding (II) (M204.1)","TBA (M174)","mitotic cell division (M6)","TBA (M66)","TBA (M48)","TBA (M203)","enriched in cell cycle (M167)","platelet activation - actin binding (M196)","mitotic cell cycle in stimulated CD4 T cells (M4.5)","PLK1 signaling events (M4.2)","extracellular matrix (II) (M2.1)","TBA (M41.0)","myeloid, dendritic cell activation via NFkB (II) (M43.1)","cell cycle (II) (M4.10)","heme biosynthesis (II) (M222)","BCR signaling (M54)","Hox cluster I (M17.0)","cell division (stimulated CD4+ T cells) (M46)","T cell activation (IV) (M52)","CD1 and other DC receptors (M50)","enriched for TF motif PAX3 (M179)","enriched in antigen presentation (III) (M95.1)","erythrocyte differentiation (M173)","Hox cluster V (M96)","TBA (M198)","platelet activation (III) (M42)","nuclear pore complex (mitosis) (M106.1)","enriched in activated dendritic cells/monocytes (M64)","TBA (M128)","lysosomal/endosomal proteins (M139)","G protein coupled receptors cluster (M155)","C-MYC transcriptional network (M4.12)","enriched in plasma membrane proteins (II) (M135.1)"],[16,12,11,22,12,21,11,173,286,20,11,32,10,16,14,11,10,14,43,13,13,65,19,9,10,41,13,23,127,73,39,16,9,11,9,14,12,11,15,10,14,34,9,10,11,12,40,23,13,8,9,21,25,16,12,10,13,15,23,31,44,15,13,14,10,12,10,24,13,10,9,12,14,7,10,10,11,15,11,10,9,11,11],["Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Down","Down","Down","Down","Up","Down","Up","Down","Down","Up","Down","Down","Down","Up","Up","Down","Down","Down","Up","Up","Down","Down","Down","Down","Up","Down","Down","Up","Up","Up","Up","Down","Down","Down","Up","Down","Down","Down","Down","Down","Up","Down","Down","Up","Down","Up","Down","Down","Up","Up","Down","Up","Up","Down","Down","Up","Down","Up","Down","Down","Down","Down"],[6.90049800298809e-10,1.5479580392161e-09,4.42869895012702e-09,6.32557632248359e-09,1.06784249503093e-08,3.0477197278921e-08,7.11279391593476e-08,8.0464773063764e-08,1.47298906097691e-07,2.73443607230711e-07,6.07657211902999e-07,1.03276739778002e-06,1.56434575928277e-06,2.09198604043065e-06,3.07001332789456e-06,6.01876180771363e-06,6.95021650513654e-06,1.50901325601342e-05,3.68391464214093e-05,4.26463335823734e-05,4.34010005155864e-05,8.39486178236439e-05,8.89316378868956e-05,0.000115233601672988,0.000118791316825518,0.00011987397364687,0.000123342291438387,0.00013642656116488,0.00016900622565778,0.000201082272135639,0.000230433417787403,0.000453691849497447,0.000472437350376872,0.000734645401457685,0.00100328339325601,0.00104253505021116,0.00164204002129369,0.00254810510082889,0.00263267454465987,0.00269005400332059,0.00303273299692104,0.00324951878885914,0.00367152467743574,0.00441382781137913,0.00441958008437789,0.00495744757599246,0.00579318744673616,0.00618304341971104,0.00618530982970946,0.00626519376161966,0.0086456187625557,0.0102795854580035,0.0105133045395857,0.0117783788763221,0.0119109705879645,0.0123631385581504,0.0129078778904235,0.0131648302346612,0.0142122214053525,0.0149281282038459,0.014986217998507,0.0165668866504867,0.0174546125684648,0.0178636642962919,0.0209890867740822,0.0219003139491162,0.0265531181988553,0.0275916363356327,0.028802564457541,0.0297932085792907,0.0307440705910567,0.0308893671878328,0.035749404756779,0.0363200571412473,0.0378891621754552,0.038482345997033,0.0394597360286791,0.0406464248321484,0.041134349027553,0.044292884452211,0.0468326141788077,0.0471528001590908,0.0481073685782847],[2.34616932101595e-07,2.63152866666738e-07,5.01919214347728e-07,5.37673987411105e-07,7.26132896621033e-07,1.72704117913886e-06,3.41975285520997e-06,3.41975285520997e-06,5.56462534146834e-06,9.29708264584417e-06,1.87821320042745e-05,2.92617429371006e-05,4.09136583197031e-05,5.08053752676014e-05,6.958696876561e-05,0.000127898688413915,0.000139004330102731,0.00028503583724698,0.000659226830698903,0.000702682865490446,0.000702682865490446,0.00129738773000177,0.00131464160354541,0.00155319922552043,0.00155319922552043,0.00155319922552043,0.00155319922552043,0.0016566082427164,0.00198145230081535,0.00227893241753724,0.00252733425960378,0.00482047590091038,0.00486753633721626,0.00734645401457685,0.00974618153448691,0.00984616436310541,0.015089016411888,0.0227988351126796,0.022865459028225,0.022865459028225,0.0251494931451989,0.0263056282907645,0.0290306602401896,0.0333923828597441,0.0333923828597441,0.036642003822553,0.0419081645083042,0.0426033175790137,0.0426033175790137,0.0426033175790137,0.057637458417038,0.0672126741484846,0.0674438404426255,0.0736314545437805,0.0736314545437805,0.0750619126744848,0.0769943593463861,0.0771731427549103,0.081900936912201,0.0835297396638098,0.0835297396638098,0.0908506687284757,0.0941994964012386,0.0949007165740507,0.109789069279814,0.112819799131811,0.134747166979266,0.137958181678163,0.141925679935709,0.144709870242269,0.145866456164766,0.145866456164766,0.166504076949381,0.166875938216542,0.171764201862063,0.172157863670937,0.17423779545131,0.17703390720719,0.17703390720719,0.188244758921897,0.195511610415742,0.195511610415742,0.197066329115865]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t5" class="section level3">
<h3>T5</h3>
<div id="htmlwidget-136581537dae5111ff88" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-136581537dae5111ff88">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117"],["cell cycle (I) (M4.1)","mitotic cell division (M6)","cell division (stimulated CD4+ T cells) (M46)","TBA (M233)","TBA (M205)","E2F transcription factor network (M8)","TBA (M26.1)","cell cycle (II) (M4.10)","enriched in NK cells (KIR cluster) (M61.1)","NK cell surface signature (S1)","mitotic cell cycle - DNA replication (M4.4)","enriched in NK cells (I) (M7.2)","C-MYC transcriptional network (M4.12)","TBA (M26.0)","leukocyte migration (M88.0)","PLK1 signaling events (M4.2)","TBA (M72.0)","enriched in NK cells (II) (M61.0)","TBA (M148)","TBA (M137)","TBA (M214)","mitotic cell cycle in stimulated CD4 T cells (M4.5)","TBA (M184.0)","TBA (M244)","TBA (M184.1)","TBA (M153)","cell cycle (III) (M103)","putative targets of PAX3 (M89.0)","cell division in stimulated CD4 T cells (M4.6)","TBA (M26.2)","TBA (M174)","Ran mediated mitosis (M15)","extracellular matrix (II) (M2.1)","intracellular transport (M147)","TBA (M180)","phosphatidylinositol signaling system (M101)","TBA (M185)","TBA (M72.1)","enriched in NK cells (receptor activation) (M61.2)","TBA (M131)","amino acid metabolishm and transport (M154.0)","leukocyte differentiation (M160)","golgi membrane (II) (M237)","TBA (M141)","MHC-TLR7-TLR8 cluster (M146)","TBA (M70.0)","muscle contraction, SRF targets (M195)","TBA (M72.2)","extracellular matrix (I) (M2.0)","enriched for ubiquitination (M138)","interferon alpha response (I) (M158.0)","TBA (M125)","extracellular matrix (III) (M2.2)","inositol phosphate metabolism (M129)","signaling in T cells (I) (M35.0)","signaling in T cells (II) (M35.1)","TBA (source: naive B cells) (M152.1)","enriched in antigen presentation (II) (M95.0)","innate antiviral response (M150)","cell cell adhesion (M133.1)","putative targets of PAX3 (M89.1)","cell cycle and transcription (M4.0)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","cell adhesion (M51)","regulation of transcription, transcription factors (M213)","TBA (M128)","signal transduction, plasma membrane (M82)","chaperonin mediated protein folding (I) (M204.0)","complement and other receptors in DCs (M40)","plasma cells, immunoglobulins (M156.1)","CD1 and other DC receptors (M50)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","TBA (M203)","extracellular matrix, collagen (M210)","TBA (M194)","myeloid cell cytokines, metallopeptidases and laminins (M78)","TBA (M108)","integrin cell surface interactions (II) (M1.1)","cytoskeleton/actin (SRF transcription targets) (M145.1)","transmembrane and ion transporters (I) (M142)","enriched in antigen presentation (I) (M71)","chromosome Y linked (M240)","TBA (M242)","extracellular matrix, complement (M140)","DNA repair (M76)","extracellular region cluster (GO) (M189)","integrin cell surface interactions (I) (M1.0)","mitotic cell cycle in stimulated CD4 T cells (M4.9)","TBA (M177.0)","mitotic cell cycle (M4.7)","innate activation by cytosolic DNA sensing (M13)","TBA (M190)","E2F1 targets (Q4) (M10.1)","RA, WNT, CSF receptors network (monocyte) (M23)","enriched in plasma membrane proteins (I) (M135.0)","regulation of antigen presentation and immune response (M5.0)","transcription regulation in cell development (M49)","TBA (M90)","cytoskeletal remodeling (enriched for SRF targets) (M34)","mitotic cell cycle in stimulated CD4 T cells (M4.11)","heme biosynthesis (I) (M171)","TBA (M70.1)","enriched for TF motif PAX3 (M179)","type I interferon response (M127)","enriched in membrane proteins (M124)","transmembrane and ion transporters (II) (M224)","blood coagulation (M11.1)","BCR signaling (M54)","TBA (M48)","TBA (M66)","antiviral IFN signature (M75)","suppression of MAPK signaling (M56)","TBA (M220)","enriched in monocytes (IV) (M118.0)","E2F1 targets (Q3) (M10.0)","cell division - E2F transcription network (M4.8)","TBA (M120)"],[127,25,24,12,11,12,18,14,13,41,26,39,11,26,45,31,21,23,11,15,12,23,14,7,11,12,51,15,13,10,21,13,44,16,11,13,14,16,14,11,23,15,13,23,16,17,8,12,30,8,16,10,14,12,14,10,8,20,11,10,10,286,15,35,14,11,12,12,12,11,10,10,10,28,13,11,8,12,13,9,16,9,10,13,21,15,29,13,21,20,11,10,19,11,12,73,45,11,10,10,10,9,9,11,16,11,19,12,12,16,20,11,10,44,29,18,10],["Down","Down","Down","Down","Up","Down","Down","Down","Up","Up","Down","Up","Down","Down","Down","Down","Up","Up","Up","Up","Up","Down","Down","Down","Down","Up","Down","Up","Down","Down","Up","Down","Down","Up","Up","Up","Up","Up","Up","Down","Down","Up","Up","Down","Up","Down","Down","Up","Down","Up","Down","Down","Down","Up","Up","Up","Up","Up","Up","Down","Up","Down","Up","Down","Up","Up","Down","Up","Up","Down","Up","Up","Up","Down","Up","Down","Down","Down","Down","Down","Up","Up","Up","Down","Down","Down","Down","Down","Up","Down","Up","Up","Down","Up","Down","Up","Down","Down","Down","Down","Down","Down","Up","Up","Down","Down","Up","Up","Up","Up","Up","Up","Down","Up","Down","Down","Up"],[3.54815577809484e-11,5.3537464799867e-07,1.32769304134204e-05,2.1287908398588e-05,2.55820122668686e-05,2.91918982874905e-05,6.08942375511237e-05,8.64764742873771e-05,9.52833367892979e-05,0.000100365464722415,0.000149840133967361,0.000187390383610079,0.000195727217747709,0.000216436061758318,0.000236119238319294,0.000246941039132543,0.000255144452352974,0.000269175853494263,0.000308902507572445,0.00032278999290849,0.000446088453223249,0.000577172746578594,0.000624300155892229,0.000698484949734882,0.000724496561975206,0.000783759894972188,0.000825991664307462,0.000941601778385835,0.00106254528776386,0.00140710938116014,0.00158755175102901,0.00169149535995352,0.00175561710600319,0.00208010344460018,0.00243454278045418,0.00278615469035283,0.00292295017296396,0.00295948711807082,0.00326384961830945,0.00331408679815192,0.00372752468939925,0.00379269428496055,0.00384311734479566,0.00417964040020096,0.00430303765935029,0.00454698123555809,0.00504080766936852,0.00524515872557032,0.00524713642421461,0.00604771423455845,0.00666127145312482,0.00682525755019163,0.00785931411027811,0.00812657582657588,0.00857987933498507,0.00941276484836087,0.010444945989945,0.0106361739128626,0.0122879006664751,0.013377344098095,0.0135273978200808,0.0136002613729107,0.013991633727298,0.014461127469123,0.0145754430921175,0.0156460334347344,0.0167759582074775,0.0179023100535975,0.0179203183944659,0.0202596893061804,0.0203809419575946,0.0204855357201193,0.021949295688189,0.0219968660077296,0.0222702078607388,0.0224341341390233,0.0228891161800485,0.0238975635841805,0.0257762377346395,0.0271100885052393,0.0284607636638649,0.0289576677760084,0.0292123648489471,0.0304522858492047,0.0314878374250979,0.032230014596856,0.0349022024467514,0.0351172495101396,0.0363028460642347,0.0363526810557281,0.0364096264330577,0.0369837353469955,0.038030274900204,0.0382203679828347,0.0384892738994652,0.0385725694772023,0.0387849550483168,0.0390456977693237,0.040812282152328,0.0409298593084395,0.0409478424852055,0.0423063196980591,0.0423116406234731,0.0442163544845826,0.0460127072785633,0.0501912972023099,0.0526332505674763,0.0531498771576185,0.0539460029754632,0.0554529233469007,0.0556212103697138,0.0574662361946569,0.0589621337025662,0.0618132923809202,0.0660252953399553,0.0664442862804415,0.066948691436053],[1.20637296455224e-08,9.10136901597739e-05,0.00150471878018764,0.00165420756962446,0.00165420756962446,0.00165420756962446,0.00295772010962601,0.00341242580056212,0.00341242580056212,0.00341242580056212,0.00463142232262754,0.00508443278822496,0.00508443278822496,0.00508443278822496,0.00508443278822496,0.00508443278822496,0.00508443278822496,0.00508443278822496,0.00548742987944434,0.00548742987944434,0.00722238448075736,0.00891994244712373,0.00922878491318947,0.0098531532428628,0.0098531532428628,0.0102491678573286,0.0104013765135014,0.0114337358803994,0.0124574275117142,0.0159472396531483,0.0174118579145118,0.0179721381995061,0.0180881762436692,0.0208010344460018,0.0236498441529835,0.0263136831866656,0.026479621582739,0.026479621582739,0.0281697377842913,0.0281697377842913,0.0303874394704773,0.0303874394704773,0.0303874394704773,0.0322972212742801,0.0325118400928688,0.0336081221758642,0.0364087017190401,0.0364087017190401,0.0364087017190401,0.0411244567949974,0.0444084763541654,0.0446266839820222,0.0504182414621615,0.0511673292784407,0.0530392540708168,0.0571489294364767,0.0623031866066896,0.0623499850064357,0.0708116309593481,0.0745820784966073,0.0745820784966073,0.0745820784966073,0.0755104042425607,0.0762407792510763,0.0762407792510763,0.0806007783001468,0.0851317282170498,0.0883030181756292,0.0883030181756292,0.0967372520116744,0.0967372520116744,0.0967372520116744,0.100363231674578,0.100363231674578,0.100363231674578,0.100363231674578,0.101068824691123,0.104168866905402,0.110935706706043,0.115217876147267,0.119464933897704,0.119665109019783,0.119665109019783,0.123259252246781,0.125951349700392,0.127420987941059,0.135464665730307,0.135464665730307,0.135464665730307,0.135464665730307,0.135464665730307,0.135464665730307,0.135464665730307,0.135464665730307,0.135464665730307,0.135464665730307,0.135464665730307,0.135464665730307,0.137844222227425,0.137844222227425,0.137844222227425,0.139669493320202,0.139669493320202,0.144553466584212,0.148993528330586,0.160990953290428,0.167245842924691,0.167323687348058,0.168271935886766,0.170371275006331,0.170371275006331,0.174451074162351,0.177408189901527,0.18435543341678,0.194551752891094,0.194551752891094,0.194551752891094]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="heatmap-of-significant-gene-sets" class="section level1">
<h1>Heatmap of significant gene-sets</h1>
<p>FDR cutoff &lt;= 0.01</p>
<img src="figure/GSEA_Disease.Rmd/heatmap-1.png" width="672" style="display: block; margin: auto;" />
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
geneset
</th>
<th style="text-align:center;">
NGenes
</th>
<th style="text-align:center;">
Direction
</th>
<th style="text-align:center;">
PValue
</th>
<th style="text-align:center;">
FDR
</th>
<th style="text-align:center;">
time
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
enriched in NK cells (KIR cluster) (M61.1)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle (I) (M4.1)
</td>
<td style="text-align:center;">
127
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T5
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
173
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000000
</td>
<td style="text-align:center;">
T0
</td>
</tr>
<tr>
<td style="text-align:center;">
NK cell surface signature (S1)
</td>
<td style="text-align:center;">
41
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000001
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000002
</td>
<td style="text-align:center;">
T0
</td>
</tr>
<tr>
<td style="text-align:center;">
regulation of localization (GO) (M63)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000003
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells (II) (M165)
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000003
</td>
<td style="text-align:center;">
T0
</td>
</tr>
<tr>
<td style="text-align:center;">
type I interferon response (M127)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000005
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
platelet activation (I) (M32.0)
</td>
<td style="text-align:center;">
22
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000005
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (II) (M61.0)
</td>
<td style="text-align:center;">
23
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000006
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
platelet activation (II) (M32.1)
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000017
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
innate antiviral response (M150)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.00e-07
</td>
<td style="text-align:center;">
0.0000034
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
173
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.00e-07
</td>
<td style="text-align:center;">
0.0000034
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and transcription (M4.0)
</td>
<td style="text-align:center;">
286
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.00e-07
</td>
<td style="text-align:center;">
0.0000056
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
antiviral IFN signature (M75)
</td>
<td style="text-align:center;">
20
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
3.00e-07
</td>
<td style="text-align:center;">
0.0000093
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (KIR cluster) (M61.1)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
0.0000097
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
activated dendritic cells (M67)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
6.00e-07
</td>
<td style="text-align:center;">
0.0000188
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in activated dendritic cells (II) (M165)
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.00e-06
</td>
<td style="text-align:center;">
0.0000293
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.60e-06
</td>
<td style="text-align:center;">
0.0000409
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
KLF12 targets network (M32.3)
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
2.10e-06
</td>
<td style="text-align:center;">
0.0000508
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
mitotic cell division (M6)
</td>
<td style="text-align:center;">
25
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
5.00e-07
</td>
<td style="text-align:center;">
0.0000910
</td>
<td style="text-align:center;">
T5
</td>
</tr>
<tr>
<td style="text-align:center;">
cytoskeletal remodeling (M32.8)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
7.00e-06
</td>
<td style="text-align:center;">
0.0001390
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (II) (M61.0)
</td>
<td style="text-align:center;">
23
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
1.40e-06
</td>
<td style="text-align:center;">
0.0001556
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
NK cell surface signature (S1)
</td>
<td style="text-align:center;">
41
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
2.40e-06
</td>
<td style="text-align:center;">
0.0002012
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
CORO1A-DEF6 network (II) (M32.4)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.51e-05
</td>
<td style="text-align:center;">
0.0002850
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (I) (M7.2)
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
3.40e-06
</td>
<td style="text-align:center;">
0.0002886
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
4.60e-06
</td>
<td style="text-align:center;">
0.0003939
</td>
<td style="text-align:center;">
T0
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
3.68e-05
</td>
<td style="text-align:center;">
0.0006592
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (KIR cluster) (M61.1)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
4.34e-05
</td>
<td style="text-align:center;">
0.0007027
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (KIR cluster) (M61.1)
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
2.30e-06
</td>
<td style="text-align:center;">
0.0007627
</td>
<td style="text-align:center;">
T3
</td>
</tr>
</tbody>
</table>
<div id="summary" class="section level2">
<h2>Summary</h2>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
visit
</th>
<th style="text-align:center;">
significant genesets
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
T0
</td>
<td style="text-align:center;">
117
</td>
</tr>
<tr>
<td style="text-align:center;">
T1
</td>
<td style="text-align:center;">
17
</td>
</tr>
<tr>
<td style="text-align:center;">
T2
</td>
<td style="text-align:center;">
44
</td>
</tr>
<tr>
<td style="text-align:center;">
T3
</td>
<td style="text-align:center;">
52
</td>
</tr>
<tr>
<td style="text-align:center;">
T4
</td>
<td style="text-align:center;">
83
</td>
</tr>
<tr>
<td style="text-align:center;">
T5
</td>
<td style="text-align:center;">
117
</td>
</tr>
</tbody>
</table>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 3.5.3 (2019-03-11)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.5 LTS

Matrix products: default
BLAS/LAPACK: /app/easybuild/software/OpenBLAS/0.2.18-GCC-5.4.0-2.26-LAPACK-3.6.1/lib/libopenblas_prescottp-r0.2.18.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] heatmap3_1.1.6       magrittr_1.5         DT_0.6              
 [4] limma_3.38.3         RNASeqUtilities_1.1  here_0.1            
 [7] data.table_1.12.2    forcats_0.4.0        stringr_1.4.0       
[10] dplyr_0.8.1          purrr_0.3.2          readr_1.3.1         
[13] tidyr_0.8.3          tibble_2.1.1         ggplot2_3.1.1       
[16] tidyverse_1.2.1      GSEABase_1.44.0      graph_1.60.0        
[19] annotate_1.60.1      XML_3.98-1.19        AnnotationDbi_1.44.0
[22] IRanges_2.16.0       S4Vectors_0.20.1     Biobase_2.42.0      
[25] BiocGenerics_0.28.0  knitr_1.22          

loaded via a namespace (and not attached):
 [1] nlme_3.1-137       bitops_1.0-6       fs_1.3.1          
 [4] lubridate_1.7.4    bit64_0.9-7        webshot_0.5.1     
 [7] httr_1.4.0         rprojroot_1.3-2    tools_3.5.3       
[10] backports_1.1.3    R6_2.4.0           DBI_1.0.0         
[13] lazyeval_0.2.1     colorspace_1.4-1   withr_2.1.2       
[16] tidyselect_0.2.5   gridExtra_2.3      bit_1.1-14        
[19] compiler_3.5.3     git2r_0.25.2       cli_1.1.0         
[22] rvest_0.3.2        xml2_1.2.0         scales_1.0.0      
[25] digest_0.6.18      rmarkdown_1.12     pkgconfig_2.0.2   
[28] htmltools_0.3.6    htmlwidgets_1.3    rlang_0.3.4       
[31] readxl_1.3.0       rstudioapi_0.10    RSQLite_2.1.1     
[34] shiny_1.3.2        generics_0.0.2     jsonlite_1.6      
[37] crosstalk_1.0.0    RCurl_1.95-4.12    kableExtra_1.1.0  
[40] Rcpp_1.0.1         munsell_0.5.0      stringi_1.4.3     
[43] yaml_2.2.0         plyr_1.8.4         grid_3.5.3        
[46] blob_1.1.1         promises_1.0.1     crayon_1.3.4      
[49] lattice_0.20-38    haven_2.1.0        hms_0.4.2         
[52] pillar_1.4.0       fastcluster_1.1.25 reshape2_1.4.3    
[55] glue_1.3.1         evaluate_0.13      modelr_0.1.4      
[58] httpuv_1.5.1       cellranger_1.1.0   gtable_0.2.0      
[61] assertthat_0.2.1   xfun_0.6           mime_0.6          
[64] xtable_1.8-4       broom_0.5.2        later_0.8.0       
[67] viridisLite_0.3.0  memoise_1.1.0      workflowr_1.3.0   </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
