<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Carl Murie" />

<meta name="date" content="2019-04-10" />

<title>GSEA analysis - baseline (T0) vs all other time periods</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<script src="site_libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.5/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<div class="container-fluid main-container">

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">VaheyProject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">GSEA analysis - baseline (T0) vs all other time periods</h1>
<h3 class="subtitle"><em>time effect</em></h3>
<h4 class="author"><em>Carl Murie</em></h4>
<h4 class="date"><em>April 10, 2019</em></h4>

</div>


<p><strong>Last updated:</strong> 2019-04-10</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<p><details> <summary> <strong style="color:red;">✖</strong> <strong>R Markdown file:</strong> uncommitted changes </summary> The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20181008)</code> </summary></p>
<p>The command <code>set.seed(20181008)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> 1ed83ed </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    data/.Rhistory

Untracked files:
    Untracked:  VaheyReport.tar.gz
    Untracked:  analysis/DEG_T0.Rmd
    Untracked:  analysis/DEG_T0_Interaction.Rmd
    Untracked:  analysis/DEG_T1.Rmd
    Untracked:  analysis/DEG_T1_Interaction.Rmd
    Untracked:  analysis/GSEA_T0.Rmd
    Untracked:  analysis/GSEA_T0_Interaction.Rmd
    Untracked:  analysis/GSEA_T1.Rmd
    Untracked:  analysis/GSEA_T1_Interaction.Rmd
    Untracked:  analysis/analysis_T4_disease.Rmd
    Untracked:  analysis/analysis_T5_disease.Rmd
    Untracked:  analysis/pamr.Rmd
    Untracked:  code/processGEOFiles.R
    Untracked:  data/BTM_for_GSEA_20131008.gmt
    Untracked:  data/GPL570.soft
    Untracked:  data/GPL571.soft
    Untracked:  data/GSE18323-GPL570_series_matrix.txt.gz
    Untracked:  data/GSE18323-GPL571_series_matrix.txt.gz
    Untracked:  data/gpl_eset.Rds
    Untracked:  data/h.all.v5.0.symbols.gmt
    Untracked:  docs/figure/DEG_T0.Rmd/
    Untracked:  docs/figure/DEG_T0_Interaction.Rmd/
    Untracked:  docs/figure/DEG_T1.Rmd/
    Untracked:  docs/figure/DEG_T1_Interaction.Rmd/
    Untracked:  docs/figure/GSEA_T0.Rmd/
    Untracked:  docs/figure/GSEA_T0_Interaction.Rmd/
    Untracked:  docs/figure/pamr.Rmd/
    Untracked:  output/dmso_M3_both_disease.csv
    Untracked:  output/dmso_M3_old_disease.csv
    Untracked:  output/dmso_M3_young_disease.csv
    Untracked:  output/pam_res.Rda

Unstaged changes:
    Modified:   analysis/analysis_PCA.Rmd
    Modified:   analysis/experimental_design.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/processVahey.Rmd

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes. </details>
</li>
</ul>
<hr />
<div id="gsea-analysis" class="section level1">
<h1>GSEA Analysis</h1>
<p>The Blood Transcript Modules (BTM) from Shuzhao et al. Molecular signatures of antibody responses derived from a systems biological study of 5 human vaccines, 2014 were used for the GSEA analysis.</p>
<div id="baseline-t0-vs-visits-t1-to-t5-time-effect" class="section level2">
<h2>baseline (T0) vs visits T1 to T5: time effect</h2>
<ul>
<li><strong>visit</strong>
<ul>
<li>T0 - study entry (baseline)</li>
<li>T1 - day of third vaccination</li>
<li>T2 - 24h after T1</li>
<li>T3 - 72h after T1</li>
<li>T4 - 2 weeks after T1</li>
<li>T5 - 5 days after challenge</li>
</ul></li>
</ul>
<p>A FDR cutoff of &lt;= 0.2 was used.</p>
<p>The formula used in the camera function is: ~visit</p>
<div id="sample-size" class="section level3">
<h3>Sample size</h3>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="2">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">
disease
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
total
</th>
<th style="text-align:center;">
NonProtected
</th>
<th style="text-align:center;">
Protected
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
T0
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
25
</td>
<td style="text-align:center;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
T1
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
25
</td>
<td style="text-align:center;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
T2
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
T3
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
24
</td>
<td style="text-align:center;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
T4
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
T5
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="comparisons" class="section level2 tabset tabset-fade tabset-pills">
<h2>Comparisons</h2>
<div id="t0-versus-t1-t1---t0" class="section level3">
<h3>T0 versus T1 (T1 - T0)</h3>
<div id="htmlwidget-9c23e70c978f4c72e429" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9c23e70c978f4c72e429">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86"],["TBA (M70.0)","TBA (M125)","TBA (M70.1)","TBA (M186)","TBA (M184.0)","TBA (M184.1)","intracellular transport (M147)","phosphatidylinositol signaling system (M101)","golgi membrane (II) (M237)","TBA (M214)","TBA (M153)","TBA (M148)","TBA (M233)","inositol phosphate metabolism (M129)","TBA (M72.0)","TBA (M244)","interferon alpha response (I) (M158.0)","TBA (M137)","nuclear pore, transport; mRNA splicing, processing (M143)","regulation of transcription, transcription factors (M213)","TBA (M174)","mitosis (TF motif CCAATNNSNNNGCG) (M169)","TBA (M72.1)","DNA repair (M76)","nuclear pore complex (mitosis) (M106.1)","nuclear pore complex (M106.0)","E2F1 targets (Q3) (M10.0)","cell division (M37.3)","TBA (M203)","TBA (M128)","TBA (M194)","enriched in activated dendritic cells (II) (M165)","TBA (M72.2)","cell cycle and transcription (M4.0)","TBA (M236)","TBA (M180)","mismatch repair (I) (M22.0)","translation initiation factor 3 complex (M245)","cell cycle, ATP binding (M144)","complement activation (II) (M112.1)","TBA (M229)","enriched in DNA interacting proteins (M182)","antiviral IFN signature (M75)","type I interferon response (M127)","E2F1 targets (Q4) (M10.1)","enriched in monocytes (II) (M11.0)","innate antiviral response (M150)","TLR and inflammatory signaling (M16)","TBA (M218)","enriched in hepatocyte nuclear factors (I) (M88.1)","interferon alpha response (II) (M158.1)","enriched in monocytes (IV) (M118.0)","cell division - E2F transcription network (M4.8)","TBA (M166)","TBA (M90)","regulation of localization (GO) (M63)","TBA (M26.0)","putative targets of PAX3 (M89.1)","cell cycle (I) (M4.1)","TBA (M243)","transmembrane and ion transporters (II) (M224)","PLK1 signaling events (M4.2)","TBA (M98.0)","lysosome (M209)","Plasma cell surface signature (S3)","mismatch repair (II) (M22.1)","growth factor induced, enriched in nuclear receptor subfamily 4 (M94)","TBA (M205)","enriched for TF motif PAX3 (M179)","extracellular matrix (I) (M2.0)","cell cell adhesion (M133.1)","putative targets of PAX3 (M89.0)","mitotic cell cycle - DNA replication (M4.4)","TBA (M185)","leukocyte migration (M88.0)","enriched for ubiquitination (M138)","cell division (stimulated CD4+ T cells) (M46)","activated dendritic cells (M67)","TBA (M161)","cell junction (M162.1)","mitochondrial cluster (M235)","Hox cluster VI (M107)","TBA (M26.2)","T cell differentiation via ITK and PKC (M18)","extracellular matrix, collagen (M210)","TBA (M41.2)"],[17,10,9,9,14,11,16,13,13,12,12,11,12,12,21,7,16,15,11,14,21,14,16,21,11,16,29,10,10,11,13,32,12,286,10,11,25,9,15,10,7,12,20,11,19,173,11,43,11,11,12,44,18,16,11,12,26,10,127,9,11,31,14,7,16,13,12,11,9,30,10,15,26,14,45,8,24,11,14,11,13,9,10,11,28,12],["Up","Up","Up","Up","Up","Up","Down","Down","Down","Down","Down","Down","Up","Down","Down","Up","Up","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Down","Up","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Up","Down","Down","Up","Up","Down","Up","Up","Down","Down","Up","Down","Up","Down","Up","Down","Up","Down","Down","Up","Up","Up","Down","Down","Up","Down","Down","Down","Down","Up","Down","Up","Up","Up","Up","Up"],[2.10892404371618e-16,1.10783496946644e-15,9.62770059283402e-10,8.92494095959457e-08,2.01032214363777e-07,4.36748990900812e-07,3.75305227629121e-06,1.5680688008039e-05,3.19122152711969e-05,4.08127003580314e-05,4.35009318130505e-05,7.58442619438639e-05,8.45970056816472e-05,0.000203817093091041,0.000278447319736795,0.000279162971640795,0.000298367438592682,0.00030197923363069,0.000348499981300098,0.000439897199194353,0.000564567215865326,0.000646878097341537,0.000725608218063697,0.00110837018860872,0.00118089275379922,0.00135577763076995,0.00144890944964545,0.00168225652941797,0.00168756285015964,0.0019870794119035,0.00228839533351058,0.00292577146446816,0.00303670328985784,0.00306536262951768,0.00322802987391976,0.00349363224741762,0.00398277453365947,0.00468931837706245,0.004986373672782,0.00503306653785167,0.00524107406147099,0.00536600713534123,0.00753670789672738,0.00761035429110365,0.0081922708696202,0.00835967352994922,0.00909314172492511,0.0109700984415721,0.0111939110461382,0.0119644506826282,0.012478227738276,0.0130031497410208,0.0131054817356355,0.01396554171505,0.0142522119648767,0.0155450489281724,0.0173802641605943,0.0191676616272506,0.0193318194235726,0.0216149616631681,0.0216170235689048,0.0225530086318891,0.0238202889516167,0.0245399486976088,0.0250582521384208,0.0253043587168752,0.0260041739240636,0.0280978739737218,0.0299892341995366,0.0304534347362766,0.0319728620205786,0.0321302338011959,0.0336867784243633,0.0343682580160879,0.0348803682727589,0.0350541526834713,0.0374823157794505,0.0417593692307036,0.0418651258878288,0.0424005878244048,0.0428449025493901,0.0462244651639817,0.0479518342120645,0.0484237407864805,0.0486967396002054,0.0491963777103395],[7.17034174863502e-14,1.88331944809295e-13,1.09113940052119e-07,7.58619981565538e-06,1.36701905767369e-05,2.47491094843794e-05,0.000182291110562716,0.000666429240341656,0.00120557257691188,0.00134457425603974,0.00134457425603974,0.00214892075507614,0.00221253707167385,0.00494984368935386,0.00570405219080193,0.00570405219080193,0.00570405219080193,0.00570405219080193,0.00623631545484386,0.007478252386304,0.00914061206639099,0.00999720695891466,0.0107263823539851,0.0157019110052903,0.0160601414516694,0.0177293997869917,0.0182455264029427,0.0197852196225613,0.0197852196225613,0.0225202333349063,0.0250985294643096,0.0306536262951768,0.0306536262951768,0.0306536262951768,0.0313580044895062,0.0329954156700553,0.0365984686876816,0.0419570591631904,0.0427810655717392,0.0427810655717392,0.0434391053813338,0.0434391053813338,0.0588072831585282,0.0588072831585282,0.0617888913083203,0.0617888913083203,0.0657801741803093,0.0776720358303468,0.0776720358303468,0.0813582646418718,0.0831881849218403,0.0840729017003035,0.0840729017003035,0.087931188576241,0.0881045830556012,0.094380654206761,0.10367175113337,0.111403705152791,0.111403705152791,0.120488328088977,0.120488328088977,0.12367778927165,0.128553940373805,0.130355787329357,0.130355787329357,0.130355787329357,0.131961479614651,0.140489369868609,0.147773038084673,0.147916683004772,0.151726104061203,0.151726104061203,0.156821209373424,0.156821209373424,0.156821209373424,0.156821209373424,0.165506329415756,0.179842800824601,0.179842800824601,0.179842800824601,0.179842800824601,0.191662416533583,0.194497307226924,0.194497307226924,0.194497307226924,0.194497307226924]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t0-versus-t2-t2---t0" class="section level3">
<h3>T0 versus T2 (T2 - T0)</h3>
<div id="htmlwidget-a0e6efde7f4990d8641e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-a0e6efde7f4990d8641e">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180"],["enriched in monocytes (II) (M11.0)","TLR and inflammatory signaling (M16)","enriched in T cells (I) (M7.0)","enriched in NK cells (I) (M7.2)","TBA (M125)","TBA (M70.0)","T cell activation (I) (M7.1)","Monocyte surface signature (S4)","enriched in NK cells (II) (M61.0)","T cell activation (II) (M7.3)","enriched in neutrophils (I) (M37.1)","TBA (M70.1)","myeloid cell enriched receptors and transporters (M4.3)","E2F1 targets (Q3) (M10.0)","TBA (M186)","NK cell surface signature (S1)","TBA (M66)","enriched in activated dendritic cells/monocytes (M64)","T cell differentiation (M14)","enriched in NK cells (receptor activation) (M61.2)","transmembrane transport (I) (M87)","formyl peptide receptor mediated neutrophil response (M11.2)","TBA (M184.1)","T cell surface signature (S0)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","enriched in activated dendritic cells (II) (M165)","TBA (M184.0)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","T cell differentiation (Th2) (M19)","enriched in T cells (II) (M223)","T cell activation (III) (M7.4)","enriched in monocytes (III) (M73)","T cell activation and signaling (M5.1)","mismatch repair (I) (M22.0)","enriched in monocytes (IV) (M118.0)","complement activation (I) (M112.0)","mitotic cell cycle - DNA replication (M4.4)","enriched in NK cells (III) (M157)","mitosis (TF motif CCAATNNSNNNGCG) (M169)","cell cycle (III) (M103)","E2F1 targets (Q4) (M10.1)","enriched in antigen presentation (III) (M95.1)","nuclear pore, transport; mRNA splicing, processing (M143)","TBA (M203)","interferon alpha response (I) (M158.0)","enriched in G-protein coupled receptors (M130)","DNA repair (M76)","enriched in NK cells (KIR cluster) (M61.1)","regulation of transcription, transcription factors (M213)","intracellular transport (M147)","chemokines and inflammatory molecules in myeloid cells (M86.0)","enriched for TF motif PAX3 (M179)","TBA (M174)","cell junction (GO) (M4.13)","cell cycle (I) (M4.1)","blood coagulation (M11.1)","innate antiviral response (M150)","TBA (M229)","suppression of MAPK signaling (M56)","nuclear pore complex (M106.0)","RIG-1 like receptor signaling (M68)","antiviral IFN signature (M75)","TBA (M128)","B cell development (M9)","recruitment of neutrophils (M132)","cell division - E2F transcription network (M4.8)","enriched in DNA interacting proteins (M182)","cell division (M37.3)","TBA (M211)","type I interferon response (M127)","TBA (M233)","IL2, IL7, TCR network (M65)","TBA (M137)","enriched in neutrophils (II) (M163)","enriched in monocytes (I) (M4.15)","TBA (M194)","T cell differentiation via ITK and PKC (M18)","antigen processing and presentation (M200)","inflammatory response (M33)","AP-1 transcription factor network (M20)","TBA (M72.1)","RA, WNT, CSF receptors network (monocyte) (M23)","interferon alpha response (II) (M158.1)","double positive thymocytes (M126)","mismatch repair (II) (M22.1)","Activated (LPS) dendritic cell surface signature (S11)","activated dendritic cells (M67)","inflammasome receptors and signaling (M53)","TBA (M72.0)","mitotic cell cycle in stimulated CD4 T cells (M4.5)","CD4 T cell surface signature Th2-stimulated (S7)","TBA (M153)","enriched in antigen presentation (I) (M71)","phosphatidylinositol signaling system (M101)","putative targets of PAX3 (M89.1)","cell cycle and transcription (M4.0)","TBA (M244)","TBA (M198)","cell cycle and growth arrest (M31)","complement and other receptors in DCs (M40)","TBA (M98.0)","regulation of signal transduction (M3)","lysosomal/endosomal proteins (M139)","TBA (M48)","nuclear pore complex (mitosis) (M106.1)","extracellular matrix (II) (M2.1)","extracellular matrix, collagen (M210)","mitochondrial cluster (M235)","enriched in antigen presentation (II) (M95.0)","regulation of antigen presentation and immune response (M5.0)","platelet activation (III) (M42)","TBA (M141)","DC surface signature (S5)","enriched in activated dendritic cells (I) (M119)","leukocyte migration (M88.0)","enriched in B cells (I) (M47.0)","cell cycle, ATP binding (M144)","enriched in B cells (II) (M47.1)","inositol phosphate metabolism (M129)","enriched in hepatocyte nuclear factors (I) (M88.1)","TBA (M166)","cell cycle, mitotic phase (M230)","nucleotide metabolism (M181)","innate activation by cytosolic DNA sensing (M13)","G protein coupled receptors cluster (M155)","cell activation (IL15, IL23, TNF) (M24)","CD28 costimulation (M12)","MHC-TLR7-TLR8 cluster (M146)","PLK1 signaling events (M4.2)","TLR8-BAFF network (M25)","cell division (stimulated CD4+ T cells) (M46)","TBA (M79)","T &amp; B cell development, activation (M62.0)","CORO1A-DEF6 network (I) (M32.2)","Resting dendritic cell surface signature (S10)","TBA (M72.2)","TBA (M55)","Plasma cell surface signature (S3)","cell cell adhesion (M133.1)","signal transduction, plasma membrane (M82)","putative targets of PAX3 (M89.0)","TBA (M41.2)","enriched in monocytes (surface) (M118.1)","TBA (M41.1)","cell adhesion (GO) (M117)","complement activation (II) (M112.1)","growth factor induced, enriched in nuclear receptor subfamily 4 (M94)","TBA (M218)","mitotic cell cycle in stimulated CD4 T cells (M4.9)","immune activation - generic cluster (M37.0)","lysosome (M209)","TBA (M32.7)","TBA (M26.0)","myeloid cell cytokines, metallopeptidases and laminins (M78)","endoplasmic reticulum (M37.2)","enriched in myeloid cells and monocytes (M81)","T cell activation (IV) (M52)","TBA (M148)","transmembrane and ion transporters (II) (M224)","integrin cell surface interactions (II) (M1.1)","extracellular matrix, complement (M140)","enriched in membrane proteins (M124)","lipid metabolism, endoplasmic reticulum (M92)","cytoskeletal remodeling (M32.8)","enriched in B cells (VI) (M69)","TBA (M26.2)","TBA (M90)","muscle contraction, SRF targets (M195)","leukocyte differentiation (M160)","TBA (M41.3)","enriched in B cells (V) (M47.4)","enriched for ubiquitination (M138)","cell cycle (II) (M4.10)","TBA (M149)","mitotic cell cycle in stimulated CD4 T cells (M4.11)","plasma cells &amp; B cells, immunoglobulins (M156.0)","signaling in T cells (II) (M35.1)","transmembrane transport (SLC cluster) (M154.1)","myeloid, dendritic cell activation via NFkB (II) (M43.1)","cytokines - recepters cluster (M115)"],[173,43,51,39,10,17,43,65,23,26,39,9,23,29,9,41,16,15,11,14,24,9,11,22,10,32,14,15,16,9,13,11,21,25,44,16,26,12,14,51,19,12,11,10,16,10,21,13,14,16,18,9,21,10,127,19,11,7,11,16,9,20,11,11,9,18,12,10,10,11,12,11,15,12,10,13,11,6,9,14,16,11,12,10,13,27,11,10,21,23,10,12,16,13,10,286,7,10,11,12,14,40,10,12,11,44,28,13,20,73,10,23,65,10,45,37,15,28,12,11,16,8,9,11,9,14,8,16,31,8,24,8,30,19,61,12,11,16,10,12,15,12,9,13,21,10,12,11,13,266,7,11,26,11,15,34,13,11,11,12,13,16,8,10,17,10,11,8,15,10,8,8,14,7,10,24,10,16,13,11],["Up","Up","Down","Down","Up","Up","Down","Up","Down","Down","Up","Up","Up","Down","Up","Down","Up","Up","Down","Down","Up","Up","Up","Down","Up","Up","Up","Up","Down","Down","Down","Up","Down","Down","Up","Up","Down","Down","Down","Down","Down","Up","Down","Down","Up","Down","Down","Down","Down","Down","Up","Down","Down","Up","Down","Up","Up","Down","Up","Down","Up","Up","Down","Up","Up","Down","Down","Down","Down","Up","Up","Down","Down","Up","Up","Down","Down","Up","Up","Up","Down","Up","Up","Down","Down","Up","Up","Up","Down","Down","Down","Down","Up","Down","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Up","Down","Up","Up","Up","Up","Up","Up","Up","Down","Down","Down","Down","Up","Up","Down","Down","Up","Up","Up","Down","Up","Down","Up","Down","Up","Down","Down","Up","Down","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Up","Down","Down","Up","Up","Down","Up","Up","Up","Up","Down","Down","Up","Up","Up","Up","Up","Down","Down","Up","Up","Up","Up","Up","Down","Down","Down","Up","Down","Down","Down","Up","Up","Up"],[6.00420066464822e-20,1.44455618612738e-13,2.94586649597693e-13,9.28190688202051e-13,1.39141487007547e-11,2.95491400380312e-11,6.43199372726179e-11,5.92716507819638e-09,3.60707008514562e-08,8.87647338642733e-08,1.08441078452539e-07,2.44674750915807e-07,4.64974674817348e-07,6.59109840994135e-07,7.33857222956643e-07,1.21167366298215e-06,1.58814443938665e-06,2.42962574428356e-06,2.60112161517637e-06,2.7652586856043e-06,2.83287650115326e-06,4.16994883639583e-06,4.74507412755298e-06,4.98884697731191e-06,5.43002483586861e-06,6.84751125224676e-06,9.46126998310747e-06,1.35636128644966e-05,1.50055666354458e-05,1.74876677375854e-05,2.30523614679415e-05,2.4840053047462e-05,4.28892537551787e-05,4.35621080946879e-05,5.19790428791339e-05,5.41138882446998e-05,5.87966409458936e-05,6.1742119515832e-05,7.16184184600944e-05,7.41993193597587e-05,7.52814661106112e-05,8.40059837847653e-05,9.02524631683705e-05,0.000120687373195988,0.000126307104577717,0.000132106167553559,0.000144723566213583,0.000161404773326706,0.000172304265082822,0.000206027071384639,0.000214417416822503,0.000247330675729248,0.000249729372950715,0.00027961886319017,0.000298370465234247,0.000351080523881668,0.000363608020213903,0.000373648288387538,0.000454075809557278,0.000456245019654623,0.000507763111402647,0.000544067168937305,0.000546024822600473,0.000649006279865813,0.000651765938998662,0.000656501622915176,0.000657982922987045,0.000678440386679425,0.000796119655538779,0.000856935518636757,0.000878619761348507,0.000951846164958527,0.00143319952355321,0.00148002070783599,0.00163093770011547,0.00195220963752616,0.00198180445603093,0.0021195896558789,0.00215081848656902,0.00222655071582392,0.00236463393176875,0.00237516896569653,0.00248488312688168,0.00249655293467285,0.00256273133696393,0.0027658503186479,0.00282671078683841,0.00288547287242982,0.00296607378177928,0.00322166259879586,0.00351853283267095,0.00361362153336402,0.00390457799389701,0.00407139031375482,0.00522424890246949,0.00559173942714974,0.00572984328691872,0.00604445547328667,0.00606541668405422,0.00609857771919064,0.00635416054515983,0.006359738238233,0.00684095273919796,0.00796366503135363,0.00829501362309803,0.00831428886509174,0.0086663618613678,0.00872827526229156,0.00899596294668407,0.00919461576556865,0.00959571834860594,0.00968304915020557,0.00998416513346534,0.010168144323501,0.0102061722654947,0.0104103737103315,0.0116683294400079,0.0123166066890969,0.0134770929675883,0.0140582837556698,0.0144625643978007,0.0147608840023091,0.0154324730791008,0.0158726121343465,0.0171063575428016,0.0172749455912804,0.0175006200981221,0.018452591877116,0.0189064342323559,0.0193341258521866,0.0216505820232879,0.0225699532090236,0.0263370483471216,0.0264426315055121,0.0280056459962759,0.0280821154367239,0.028856045034945,0.030507216931365,0.0322236045287662,0.0336217594888551,0.0347075414876312,0.0363044070901225,0.0384487196803275,0.0389628815511244,0.0393775139332525,0.0398340701783411,0.0426230990917363,0.0429140488049855,0.0457319835533344,0.0473785060093073,0.0484716244129019,0.0508216783088464,0.0530791059625468,0.054362734611727,0.0554064127215584,0.0554267943592556,0.0562917295473682,0.0569575910830032,0.0606180563606026,0.0617445887862891,0.0621129391228512,0.0688083554502751,0.0691021742674267,0.0702220703455673,0.0712542581759313,0.0758374236364955,0.0806268373024226,0.0818120378164678,0.0834427827890281,0.0871156914273145,0.0876327425301051,0.0878462444509469,0.0894131167049909,0.0989233975971312,0.0993874573329172,0.10253205668048,0.102952873641797,0.103301796496467,0.103772240056224,0.103939498960544],[2.0414282259804e-17,2.45574551641655e-11,3.33864869544052e-11,7.88962084971743e-11,9.46162111651323e-10,1.67445126882177e-09,3.12411123895573e-09,2.51904515823346e-07,1.36267092105501e-06,3.01800095138529e-06,3.3518151521694e-06,6.93245127594788e-06,1.21608761106076e-05,1.60069532812861e-05,1.66340970536839e-05,2.57480653383707e-05,3.1762888787733e-05,4.58656195424814e-05,4.58656195424814e-05,4.58656195424814e-05,4.58656195424814e-05,6.44446638352083e-05,7.01445740594788e-05,7.06753321785854e-05,7.38483377678131e-05,8.95443779139961e-05,0.000119141918305798,0.000164701013354602,0.000175927332967295,0.000198193567692635,0.000252832351583874,0.000263925563629284,0.000435621080946879,0.000435621080946879,0.000504939273683015,0.000511075611199943,0.000540293457340644,0.000552429490404812,0.000624285328722142,0.000624285328722142,0.000624285328722142,0.000680048440162386,0.000713624127377813,0.000932584247423541,0.000954320345698307,0.000976436890613264,0.00104693643643868,0.00114328381106417,0.0011955806148604,0.00140098408541554,0.00142944944548335,0.00160203748685364,0.00160203748685364,0.00176056321267885,0.00184447196690262,0.00213156032356727,0.00216888994513556,0.00219035203537522,0.00258538844470953,0.00258538844470953,0.0028301550471623,0.00294680062990731,0.00294680062990731,0.00333901781814321,0.00333901781814321,0.00333901781814321,0.00333901781814321,0.00339220193339712,0.00392290844758239,0.00416225823337853,0.00420747491349989,0.00449482911230416,0.00667517586312455,0.0068000951441113,0.00739358424052347,0.0087335694310381,0.00875082487078594,0.00923923696152342,0.00925668715738567,0.00946284054225166,0.00984826156508319,0.00984826156508319,0.0101050952117711,0.0101050952117711,0.0102509253478557,0.0109347570737243,0.0110469157186788,0.0111484179162061,0.011331068379831,0.0121707253732288,0.0131461666275618,0.0133546882754757,0.0142748012680106,0.014726305390177,0.0186973118614698,0.019804077137822,0.0200839867788904,0.0207351642452482,0.0207351642452482,0.0207351642452482,0.0211991274607767,0.0211991274607767,0.0225817857410418,0.0260350587563484,0.0266684737182188,0.0266684737182188,0.0274779036035105,0.0274779036035105,0.028060801852042,0.0284197214572122,0.0293922904371713,0.0293949706345526,0.030040850844055,0.0301747701762453,0.0301747701762453,0.0305131643233855,0.0339079658940401,0.0354885277482452,0.0385059799073952,0.0398318039743977,0.040638610704564,0.0411368898425007,0.0426588686739371,0.0435216784328856,0.0465292925164204,0.0466149325478994,0.0468520538059962,0.0490146971735895,0.0498309119302404,0.0505661753057188,0.0561923502894495,0.0581347279626365,0.0670932441184636,0.0670932441184636,0.0702052885918098,0.0702052885918098,0.0716135424224913,0.0751627083816239,0.0788203276243202,0.0816528444729337,0.0836919440127278,0.0869260451453638,0.0914165363028765,0.0919956925512659,0.092333480946937,0.0927642730180546,0.0985840387155805,0.0985863283357775,0.104354861799555,0.107391280287763,0.109141405962826,0.113680069901367,0.11795356880566,0.120021621870047,0.12080198770607,0.12080198770607,0.121905656344619,0.122566968153298,0.129623516745943,0.131170181998568,0.131170181998568,0.144139504606902,0.144139504606902,0.145582340960322,0.146826956241313,0.155329662869931,0.164150447202537,0.165571981295233,0.167873054131772,0.17364955298443,0.17364955298443,0.17364955298443,0.175725200460676,0.193095631389668,0.193095631389668,0.19633016470325,0.19633016470325,0.19633016470325,0.19633016470325,0.19633016470325]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t0-versus-t3-t3---t0" class="section level3">
<h3>T0 versus T3 (T3 - T0)</h3>
<div id="htmlwidget-c47734cf8f874295dd41" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-c47734cf8f874295dd41">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46"],["TBA (M125)","TBA (M70.0)","TBA (M70.1)","TBA (M186)","interferon alpha response (I) (M158.0)","TBA (M184.0)","intracellular transport (M147)","TBA (M184.1)","TBA (M148)","TBA (M137)","enriched in B cells (II) (M47.1)","translation initiation factor 3 complex (M245)","TBA (M233)","nuclear pore, transport; mRNA splicing, processing (M143)","TBA (M214)","interferon alpha response (II) (M158.1)","TBA (M174)","enriched in B cells (I) (M47.0)","TBA (M72.1)","inositol phosphate metabolism (M129)","TBA (M244)","nuclear pore complex (M106.0)","TBA (M72.0)","regulation of transcription, transcription factors (M213)","golgi membrane (II) (M237)","phosphatidylinositol signaling system (M101)","TBA (M153)","TBA (M203)","TBA (M243)","type I interferon response (M127)","cell division (M37.3)","respiratory electron transport chain (mitochondrion) (M219)","TBA (M72.2)","TBA (M229)","TBA (M128)","TBA (M166)","enriched in DNA interacting proteins (M182)","transmembrane transport (I) (M87)","Plasma cell surface signature (S3)","mitosis (TF motif CCAATNNSNNNGCG) (M169)","enriched in activated dendritic cells (II) (M165)","antiviral IFN signature (M75)","complement activation (II) (M112.1)","E2F1 targets (Q4) (M10.1)","TBA (M180)","nuclear pore complex (mitosis) (M106.1)"],[10,17,9,9,16,14,16,11,11,15,28,9,12,11,12,12,21,37,16,12,7,16,21,14,13,13,12,10,9,11,10,17,12,7,11,16,12,24,16,14,32,20,10,19,11,11],["Up","Up","Up","Up","Up","Up","Down","Up","Down","Down","Down","Up","Up","Down","Down","Up","Down","Down","Down","Down","Up","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Down","Down","Down","Up","Down","Up","Up","Down","Down","Down","Up","Down","Down","Down"],[4.82717655627392e-15,1.81145563518842e-14,2.0399668867306e-09,6.16514252099681e-07,1.42906139789214e-05,1.88610680794668e-05,2.76938030159852e-05,4.71834540182198e-05,5.95541730641657e-05,0.000436781461378845,0.000624636992059403,0.000632319935902259,0.000781994506931501,0.00079312424682914,0.000898617395714372,0.00106004458163353,0.00106036261317147,0.00107503367035023,0.00142523070716633,0.0015772487740564,0.00161085749184653,0.00163196168655015,0.0017377679731009,0.00214071940529208,0.00243885011400921,0.00274947562643336,0.00289244687349637,0.00487343712035552,0.00580214945489646,0.00713533091819982,0.00939886991501217,0.00999198560148036,0.0103484854670605,0.0109708970123479,0.0111147468069979,0.0159370194111829,0.0166876798475509,0.0189873330482442,0.0202276078049783,0.0203813087677731,0.0209258953584812,0.0225911538307219,0.0234328084521648,0.0259534319495073,0.0265390274995483,0.0267702857056738],[1.64124002913313e-12,3.07947457982031e-12,2.31196247162801e-07,5.24037114284729e-05,0.000971761750566658,0.00106879385783645,0.00134512757506214,0.00200529679577434,0.00224982431575737,0.0148505696868807,0.0179157315172307,0.0179157315172307,0.0192615888515648,0.0192615888515648,0.02030619155106,0.02030619155106,0.02030619155106,0.02030619155106,0.0252212260648659,0.0252212260648659,0.0252212260648659,0.0252212260648659,0.0256887439501872,0.0303268582416378,0.0331683615505253,0.0359546812687439,0.036423405073658,0.0591774507471742,0.0680252005056826,0.080867083739598,0.103084379713037,0.106164847015729,0.106620759357594,0.107971826125122,0.107971826125122,0.150516294438949,0.153346247247765,0.169886664115869,0.173241124526071,0.173241124526071,0.173531815167893,0.182880769105844,0.185282671482233,0.197867329128893,0.197867329128893,0.197867329128893]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t0-versus-t4-t4---t0" class="section level3">
<h3>T0 versus T4 (T4 - T0)</h3>
<div id="htmlwidget-14032f3f90232640cb54" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-14032f3f90232640cb54">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61"],["putative targets of PAX3 (M89.0)","putative targets of PAX3 (M89.1)","AP-1 transcription factor network (M20)","cell cycle and growth arrest (M31)","growth factor induced, enriched in nuclear receptor subfamily 4 (M94)","signaling in T cells (I) (M35.0)","enriched in monocytes (II) (M11.0)","chemokines and inflammatory molecules in myeloid cells (M86.0)","NK cell surface signature (S1)","TBA (M236)","cell cycle and transcription (M4.0)","TBA (M218)","TBA (M66)","TLR and inflammatory signaling (M16)","TBA (M233)","TBA (M194)","enriched in antigen presentation (II) (M95.0)","TBA (M177.1)","enriched in neutrophils (I) (M37.1)","leukocyte differentiation (M160)","TBA (M177.0)","myeloid cell enriched receptors and transporters (M4.3)","phosphatidylinositol signaling system (M101)","cell junction (GO) (M4.13)","inositol phosphate metabolism (M129)","lipid metabolism, endoplasmic reticulum (M92)","TBA (M72.0)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","golgi membrane (II) (M237)","interferon alpha response (II) (M158.1)","proteasome (M226)","BCR signaling (M54)","type I interferon response (M127)","mitosis (TF motif CCAATNNSNNNGCG) (M169)","innate antiviral response (M150)","enriched in NK cells (III) (M157)","nuclear pore complex (mitosis) (M106.1)","lysosome (M209)","activated dendritic cells (M67)","extracellular region cluster (GO) (M189)","enriched for TF motif TTCNRGNNNNTTC (M172)","enriched in NK cells (KIR cluster) (M61.1)","myeloid cell cytokines, metallopeptidases and laminins (M78)","CCR1, 7 and cell signaling (M59)","mismatch repair (I) (M22.0)","Monocyte surface signature (S4)","cell cycle (I) (M4.1)","MHC-TLR7-TLR8 cluster (M146)","TBA (M208)","enriched in NK cells (I) (M7.2)","complement and other receptors in DCs (M40)","interferon alpha response (I) (M158.0)","enriched in antigen presentation (III) (M95.1)","enriched in NK cells (II) (M61.0)","TBA (M26.0)","TBA (M153)","antiviral IFN signature (M75)","enriched for ubiquitination (M138)","nuclear pore, transport; mRNA splicing, processing (M143)","formyl peptide receptor mediated neutrophil response (M11.2)","TBA (M114.0)"],[15,10,14,11,12,14,173,18,41,10,286,11,16,43,12,13,20,8,39,15,21,23,13,10,12,8,21,10,13,12,10,12,11,14,11,12,11,7,11,15,8,13,11,10,25,65,127,16,7,39,12,16,12,23,26,12,20,8,11,9,30],["Up","Up","Up","Up","Up","Up","Down","Up","Down","Down","Down","Down","Down","Down","Up","Down","Down","Down","Up","Up","Down","Down","Down","Down","Down","Up","Down","Down","Down","Up","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Up","Down","Up","Down","Down","Down","Down","Down","Up","Down","Down","Up","Down","Down","Up","Down","Down","Down","Down","Down","Down"],[7.12471694502225e-17,4.38656609531707e-10,5.75775183209878e-10,1.22705947877985e-09,1.95042418860479e-08,3.23034259544559e-06,7.81201382074349e-06,8.24048819630845e-06,1.07017211340028e-05,3.65534245913616e-05,5.18354108105677e-05,0.000146102536469368,0.000162625597141331,0.000181812802578282,0.000430490333906087,0.000431886946673104,0.000511971138119566,0.000683658448796348,0.000952789248553879,0.00100449397167482,0.00121758907439427,0.00155904870485094,0.00157743719316302,0.00163609421111964,0.00163631346323332,0.00185466557923578,0.00401433555074475,0.00407407040254193,0.00601513283534416,0.00609422950831775,0.00662280649854809,0.00695043581690612,0.00724174666624378,0.00845974766461142,0.00919924750184769,0.010647696713806,0.011511460438627,0.0116656380383857,0.0124314153605272,0.0130932127684889,0.0137667547044598,0.0147726454228301,0.0168617367233508,0.0170088911774593,0.0170701565586795,0.0172576303490744,0.0185541199248395,0.0189344457298596,0.0196810370654791,0.0196820531528156,0.0199403847734543,0.0206100275961169,0.0225900751646892,0.0226802958516295,0.0228067706829674,0.0240599221238089,0.0258454161741853,0.0285987029233779,0.028859163668402,0.0302834161334107,0.0351724708963005],[2.42240376130756e-14,6.52545207637862e-08,6.52545207637862e-08,1.04300055696287e-07,1.32628844825126e-06,0.00018305274707525,0.000350220748343109,0.000350220748343109,0.000404287242840104,0.00124281643610629,0.00160218542505391,0.0041395718666321,0.00425328484831174,0.00441545377690113,0.00917759761680345,0.00917759761680345,0.0102394227623913,0.0129135484772643,0.0170499128688589,0.0170763975184719,0.0197133469187644,0.0222538630999731,0.0222538630999731,0.0222538630999731,0.0222538630999731,0.0242533191130833,0.0494708548880091,0.0494708548880091,0.0690679344276012,0.0690679344276012,0.072637232564721,0.0738483805546275,0.0746119353491783,0.0845974766461142,0.0893641185893775,0.100561580074834,0.104376761396083,0.104376761396083,0.108376441604596,0.111292308532156,0.11416333169552,0.119588081994339,0.127556398232289,0.127556398232289,0.127556398232289,0.127556398232289,0.132935898489696,0.132935898489696,0.132935898489696,0.132935898489696,0.132935898489696,0.134757872743842,0.140987309676526,0.140987309676526,0.140987309676526,0.14607809860884,0.154165640337246,0.166307044868758,0.166307044868758,0.171606024755994,0.196043280405609]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="t0-versus-t5-t5---t0" class="section level3">
<h3>T0 versus T5 (T5 - T0)</h3>
<p>There is a confound between visits T5 and the other visit times. The T5 visit used the HGU133 Plus 2.0 chip while the other visits used the HGU133A 2.0 chip. Boxplots of normalized scores and MDS plots show a clear separation between the samples for the two chips (see Quality Control section).</p>
<div id="htmlwidget-8dc1c60bd7d5801d7665" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-8dc1c60bd7d5801d7665">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199"],["B cell surface signature (S2)","respiratory electron transport chain (mitochondrion) (M219)","respiratory electron transport chain (mitochondrion) (M231)","enriched in antigen presentation (II) (M95.0)","interferon alpha response (I) (M158.0)","regulation of antigen presentation and immune response (M5.0)","chaperonin mediated protein folding (I) (M204.0)","TBA (M26.1)","TBA (source: B cells) (M152.0)","enriched in T cells (I) (M7.0)","TBA (M41.1)","Naive B cell surface signature (S8)","Memory B cell surface signature (S9)","TBA (M72.1)","TBA (M26.0)","TBA (M174)","TBA (M72.0)","translation initiation factor 3 complex (M245)","TBA (M170)","TBA (M41.3)","enriched in monocytes (II) (M11.0)","transcription elongation, RNA polymerase II (M234)","interferon alpha response (II) (M158.1)","extracellular matrix, collagen (M210)","respiratory electron transport chain (mitochondrion) (M238)","respiratory electron transport chain (mitochondrion) (M216)","leukocyte migration (M88.0)","translation initiation (M227)","Wnt signaling pathway (M206)","TBA (M166)","cell cycle, ATP binding (M144)","intracellular transport (M147)","TBA (M194)","chaperonin mediated protein folding (II) (M204.1)","TBA (M41.2)","extracellular matrix (III) (M2.2)","TBA (M98.0)","proteasome (M226)","enriched in calcium signaling proteins (M239)","nuclear pore complex (mitosis) (M106.1)","spliceosome (M250)","extracellular matrix (I) (M2.0)","mitosis (TF motif CCAATNNSNNNGCG) (M169)","TBA (M211)","TBA (M41.0)","T cell activation (I) (M7.1)","TBA (M72.2)","cell development (M175)","purine nucleotide biosynthesis (M212)","TBA (M141)","TBA (M197)","T cell activation and signaling (M5.1)","enriched in extracellular matrix &amp; associated proteins (M202)","integrin cell surface interactions (II) (M1.1)","integrin cell surface interactions (I) (M1.0)","enriched in antigen presentation (III) (M95.1)","TBA (M116)","ATF targets network (M41.4)","TBA (M26.2)","TBA (source: B cells) (M217)","muscle contraction, SRF targets (M195)","TBA (source: memory B cells) (M152.2)","MHC-TLR7-TLR8 cluster (M146)","enriched in plasma membrane proteins (I) (M135.0)","enriched in NK cells (I) (M7.2)","enriched for TF motif PAX3 (M179)","cell adhesion (M51)","amino acid metabolishm and transport (M154.0)","leukocyte activation and migration (M45)","TBA (M120)","T cell signaling and costimulation (M44)","T cell differentiation (Th2) (M19)","blood coagulation (M11.1)","signal transduction, plasma membrane (M82)","phosphatidylinositol signaling system (M101)","TBA (M98.1)","xenobiotic metabolism (M164)","T cell activation (II) (M7.3)","TBA (source: naive B cells) (M152.1)","enriched for cell migration (M122)","cell adhesion (GO) (M117)","BCR signaling (M54)","TBA (M108)","mitochondrial cluster (M235)","nuclear pore complex (M106.0)","small GTPase mediated signal transduction (M215)","TBA (M185)","TBA (M229)","Monocyte surface signature (S4)","TBA (M218)","TBA (M176)","TBA (M149)","extracellular region cluster (GO) (M189)","enriched for TF motif TNCATNTCCYR (M232)","transmembrane and ion transporters (I) (M142)","TBA (M153)","regulation of transcription, transcription factors (M213)","metabolism in mitochondria, peroxisome (M187)","TBA (M90)","TLR and inflammatory signaling (M16)","enriched in plasma membrane proteins (II) (M135.1)","PLK1 signaling events (M4.2)","inositol phosphate metabolism (M129)","plasma membrane, cell junction (M162.0)","cell junction (M162.1)","cell cell adhesion (M133.1)","TBA (M99)","nuclear pore, transport; mRNA splicing, processing (M143)","TBA (M244)","TBA (M128)","transmembrane transport (SLC cluster) (M154.1)","TBA (M198)","cell cycle, mitotic phase (M230)","T cell surface signature (S0)","platelet activation (III) (M42)","enriched in monocytes (I) (M4.15)","golgi membrane (II) (M237)","enriched in hepatocyte nuclear factors (II) (M88.2)","TBA (M207)","Hox cluster II (M17.1)","metabolism of steroids (M225)","IL2, IL7, TCR network (M65)","receptors, cell migration (M109)","mismatch repair (I) (M22.0)","TBA (M184.1)","mitotic cell division (M6)","transmembrane and ion transporters (II) (M224)","enriched for TF motif TTCNRGNNNNTTC (M172)","T cell activation (III) (M7.4)","enriched in monocytes (IV) (M118.0)","targets of FOSL1/2 (M0)","TBA (M220)","cell division (stimulated CD4+ T cells) (M46)","mitotic cell cycle (M4.7)","integrin mediated leukocyte migration (M39)","chemokine cluster (I) (M27.0)","TBA (M66)","TBA (M137)","TBA (M183)","T cell differentiation (M14)","DC surface signature (S5)","KLF12 targets network (M32.3)","TBA (M236)","cell adhesion, membrane (M133.0)","inflammatory response (M33)","TBA (M190)","suppression of MAPK signaling (M56)","collagen, TGFB family et al (M77)","T cell activation (IV) (M52)","platelet activation (II) (M32.1)","platelet activation (I) (M32.0)","axon guidance (M110)","Hox cluster III (M17.2)","nucleotide metabolism (M181)","enriched in hepatocyte nuclear factors (I) (M88.1)","Hox cluster IV (M17.3)","TBA (M180)","Hox cluster VI (M107)","E2F transcription factor network (M8)","cell cycle (I) (M4.1)","myeloid cell cytokines, metallopeptidases and laminins (M78)","B cell development/activation (M58)","T cell differentiation via ITK and PKC (M18)","T &amp; B cell development, activation (M62.0)","integrins and cell adhesion (M84)","mismatch repair (II) (M22.1)","enriched in NK cells (III) (M157)","TBA (M208)","TBA (M184.0)","TBA (M243)","Hox cluster I (M17.0)","TBA (M205)","Membrane, ER proteins (M134)","TBA (M233)","CD28 costimulation (M12)","inflammasome receptors and signaling (M53)","enriched in T cells (II) (M223)","extracellular matrix (II) (M2.1)","cell division (M37.3)","antigen processing and presentation (M200)","enriched in B cells (I) (M47.0)","viral sensing &amp; immunity; IRF2 targets network (I) (M111.0)","growth factor induced, enriched in nuclear receptor subfamily 4 (M94)","CORO1A-DEF6 network (I) (M32.2)","enriched for ubiquitination (M138)","TBA (M214)","TBA (M48)","cell junction (GO) (M4.13)","MAPK, RAS signaling (M100)","CORO1A-DEF6 network (II) (M32.4)","complement activation (II) (M112.1)","Activated (LPS) dendritic cell surface signature (S11)","TBA (M161)","lysosomal/endosomal proteins (M139)","TBA (M70.0)","TBA (M186)","enriched for unknown TF motif CTCNANGTGNY (M62.1)","enriched in monocytes (III) (M73)","enriched in B cell differentiation (M123)"],[137,17,11,20,16,73,12,18,16,51,13,38,31,16,26,21,21,9,11,10,173,13,12,28,12,11,45,10,14,16,15,16,13,9,12,14,14,10,7,11,9,30,14,10,15,43,12,9,11,23,11,21,11,12,29,12,12,9,10,9,8,10,16,12,39,9,35,23,11,10,9,16,19,12,13,6,8,26,8,10,21,12,8,13,16,11,14,7,65,11,10,7,15,10,9,12,14,7,11,43,11,31,12,16,11,10,9,11,7,11,16,10,8,22,10,10,13,10,10,11,6,11,14,25,11,25,11,8,13,44,11,10,24,20,10,25,16,15,9,11,65,16,10,14,9,10,11,33,13,21,22,10,10,9,11,8,11,9,12,127,11,11,11,30,9,13,12,7,14,9,10,11,13,12,8,10,9,44,10,6,37,15,12,19,8,12,12,10,9,14,10,27,14,10,17,9,6,11,8],["Down","Up","Up","Up","Down","Up","Up","Down","Down","Up","Down","Down","Down","Up","Down","Up","Up","Up","Down","Down","Up","Up","Down","Down","Up","Up","Down","Up","Down","Down","Up","Up","Up","Up","Down","Down","Down","Up","Down","Up","Up","Down","Up","Up","Down","Up","Up","Down","Up","Down","Down","Up","Down","Down","Down","Up","Down","Down","Down","Down","Down","Down","Up","Down","Up","Up","Down","Down","Up","Up","Up","Up","Up","Down","Up","Down","Down","Up","Down","Down","Up","Up","Down","Up","Up","Up","Up","Up","Up","Up","Down","Down","Down","Up","Down","Up","Up","Down","Down","Up","Down","Down","Up","Down","Down","Down","Down","Up","Down","Up","Down","Up","Up","Up","Up","Up","Up","Down","Down","Down","Down","Up","Up","Up","Down","Down","Down","Down","Up","Up","Down","Down","Down","Down","Up","Down","Up","Up","Down","Up","Down","Up","Up","Down","Up","Up","Up","Down","Up","Up","Up","Down","Down","Up","Down","Down","Up","Down","Down","Down","Down","Up","Up","Down","Up","Up","Up","Down","Down","Up","Down","Up","Down","Down","Up","Up","Up","Down","Up","Up","Up","Up","Down","Up","Up","Up","Up","Up","Up","Up","Down","Down","Up","Up","Down","Down","Down","Up","Down"],[1.46309164651456e-09,1.14419536232487e-08,1.56926608053543e-07,1.32948524296856e-06,2.69186588762018e-06,2.76736143291944e-06,3.64583076252381e-06,5.13899788338129e-06,6.3051005632424e-06,6.50621437601739e-06,6.52666948341901e-06,8.36127548126721e-06,8.44804837730297e-06,1.27428353999235e-05,1.2922490997992e-05,1.3626615811252e-05,1.44167520720222e-05,1.50481463789336e-05,1.58968282409189e-05,1.68296399095043e-05,1.71562856722119e-05,1.96355089427105e-05,2.70846501565658e-05,2.97624191317531e-05,3.18232481559447e-05,3.30922884521938e-05,4.10683099311311e-05,4.24951928872557e-05,4.95952799058802e-05,5.04772337426719e-05,5.75237737626528e-05,6.3470805730743e-05,6.41689780939449e-05,6.50546923975194e-05,7.78860095066221e-05,8.65712474091895e-05,9.6415329338858e-05,0.00010439091883039,0.000109635630919648,0.000113630626544957,0.00011727436583774,0.000130314912796137,0.000143344094573852,0.000145383811906679,0.000170542296931552,0.00019154619422944,0.00024886164957554,0.000269351388757543,0.000270290758194022,0.000293234937368335,0.000296014403506197,0.000332458960297846,0.000339490864737984,0.000389986079090856,0.000423271997561891,0.000452047213503582,0.000471445292047943,0.000493596133924021,0.000495973474610823,0.000519211106100081,0.000523722253057519,0.000533376924645468,0.000537075188096695,0.000633611587740224,0.000713128614163104,0.000965861556149935,0.00100889388673488,0.00102764393150851,0.00104410939036116,0.00119306151580962,0.00120947214207022,0.00121983243191068,0.0013307968408084,0.00147554900829326,0.00158688078901147,0.00160548024009586,0.00166653426636976,0.00187156782624375,0.00214188661992516,0.0022311821521086,0.00255842742875993,0.00256245784632249,0.00271631175969253,0.00346672547505405,0.00350283642763075,0.00376756716451166,0.0038105807523359,0.00421313916860876,0.00504839545574921,0.0058383445131015,0.00587545285124616,0.00594975211733361,0.00595433554205206,0.0060030836686035,0.00632525547096006,0.00678532415459519,0.00716574353167607,0.00765945650762429,0.00773128763858699,0.00795456321940296,0.00802418065013864,0.00820614009354149,0.00822166760767593,0.00838277770040119,0.008637969014823,0.0094323095491816,0.00982018556330109,0.0102129750472968,0.0108188775046884,0.0117536995918161,0.0121843479693055,0.0129340796671378,0.0130647295242812,0.0138664000663775,0.0148442907581198,0.0150142035392894,0.0150859983717382,0.0151145316351845,0.0155438969434346,0.0169187182753182,0.0176943365031067,0.0180710625260521,0.0181651811199029,0.0184390061742797,0.0188453774444549,0.0194271547875544,0.0205067796112586,0.0206991462678267,0.0212587615672373,0.0219907130962324,0.0220458562835642,0.0223481396752793,0.0228880902925823,0.0229304558614761,0.0231600896303984,0.0237692909622413,0.0245372127509766,0.0245498293462536,0.0248565422458575,0.0257160380948902,0.0263973282201912,0.0264286614829013,0.0273150994096459,0.0273285711094976,0.0278671317468463,0.0282629445356455,0.028945380575269,0.0305161391250373,0.0307894423675971,0.0317302274834283,0.0322060983930181,0.032448164958878,0.0324946555290712,0.0333035222734752,0.0336143815350051,0.0344111570659958,0.0349337661194789,0.0359897244781842,0.0382397934985883,0.0386565887992194,0.040503855539265,0.0408624910834086,0.042508350803166,0.0442733681617317,0.0454052215551178,0.0462710653151688,0.0463955955550043,0.0467430804477841,0.0472907280223483,0.0492575264183394,0.0506651987840332,0.0509339781992611,0.0513760983070988,0.0526635597834844,0.059245236622182,0.0614266079200062,0.0645914865055483,0.0650275574070386,0.0653324282713615,0.0686070933977936,0.0756313800119272,0.0765783743541778,0.0771059716301954,0.0782363430357194,0.0817186421658237,0.0841975965541274,0.0843499406101537,0.0847854511875849,0.085038249244157,0.0902327488713053,0.0919451647868128,0.093319538436577,0.0956552427355483,0.0959528755934893,0.102245972028781,0.102670311077706,0.103732916676406,0.107782912017874,0.113457706298098],[4.97451159814949e-07,1.94513211595227e-06,1.77850155794015e-05,0.000113006245652328,0.000156817147865435,0.000156817147865435,0.000177083208465442,0.000201733420396588,0.000201733420396588,0.000201733420396588,0.000201733420396588,0.000220948957560232,0.000220948957560232,0.000277768434692955,0.000277768434692955,0.000277768434692955,0.000277768434692955,0.000277768434692955,0.000277768434692955,0.000277768434692955,0.000277768434692955,0.000303457865478253,0.000400381784923147,0.000421634271033169,0.000432745310528689,0.000432745310528689,0.000516013056488105,0.000516013056488105,0.000572075315750282,0.000572075315750282,0.000630905905783933,0.000650546923975194,0.000650546923975194,0.000650546923975194,0.0007566069494929,0.000817617336642345,0.000885978702032749,0.000934024010587702,0.000955797808017445,0.000965860325632137,0.000972519131337358,0.00105493024644492,0.00112342036473343,0.00112342036473343,0.00128854179903839,0.0014157762182176,0.00180027576288689,0.00187548689359117,0.00187548689359117,0.00197342935670798,0.00197342935670798,0.00217377012502438,0.00217786592473424,0.00245546790538687,0.0026165905303826,0.00274457236770032,0.0028121298122158,0.00285815222657084,0.00285815222657084,0.00289850101512502,0.00289850101512502,0.00289850101512502,0.00289850101512502,0.00336606155986994,0.00373021121254547,0.00497565044077239,0.00511976002223671,0.00513821965754256,0.00514488685105501,0.00576031981735599,0.00576031981735599,0.00576031981735599,0.00619823186129938,0.00677954949756364,0.00718241160042886,0.00718241160042886,0.00735872273461971,0.00815811616567786,0.00921824621233615,0.00948252414646156,0.0106248252164591,0.0106248252164591,0.0111270602204272,0.014011345710523,0.014011345710523,0.0148919247792438,0.0148919247792438,0.0162780376968975,0.0192860051118509,0.0217132813545233,0.0217132813545233,0.0217132813545233,0.0217132813545233,0.0217132813545233,0.0226377564223834,0.024031356380858,0.0251170391831945,0.0265518969406018,0.0265518969406018,0.0270120932776944,0.0270120932776944,0.0271394853068914,0.0271394853068914,0.0274052347897731,0.0279705663337126,0.0302545777992617,0.031204327958153,0.0321519584822306,0.0337469573540738,0.0363296169201588,0.0373214262122872,0.0392641704180969,0.0393098056482797,0.0413559300225293,0.0435503453895147,0.0435503453895147,0.0435503453895147,0.0435503453895147,0.0444111341240989,0.0479363684467348,0.0497196232318701,0.0502126957785935,0.0502126957785935,0.0505585653165732,0.0512594266489174,0.0524224811727658,0.054900039904157,0.0549821072739146,0.0560308444407804,0.0572182529497085,0.0572182529497085,0.0575633900726892,0.0581817536783723,0.0581817536783723,0.058329114624707,0.0594232274056032,0.0604850867951176,0.0604850867951176,0.0608001752776369,0.0624532353733048,0.0632798936914538,0.0632798936914538,0.0645257928974249,0.0645257928974249,0.0653436192684671,0.0658178160419143,0.0669484992897377,0.0701046439358965,0.0702577879529062,0.0719218489624375,0.0722103456201582,0.0722103456201582,0.0722103456201582,0.0735272569674127,0.073734772399366,0.0749986756566574,0.075652741914795,0.0774462425479913,0.0817706276070442,0.0821452511983413,0.0855360924431683,0.0857607837553021,0.0886677256016959,0.0917862510670047,0.0935622747196366,0.0944580987347393,0.0944580987347393,0.0945990913824201,0.09514110963076,0.0985150528366787,0.100683445277609,0.100683445277609,0.100970366615107,0.102905806473475,0.115105031151668,0.118665038027285,0.124074041875065,0.124095115152307,0.124095115152307,0.129591176418055,0.142069995602515,0.143058501540772,0.143256996471401,0.144567155609481,0.150185612629081,0.152978861074145,0.152978861074145,0.152978861074145,0.152978861074145,0.161469129559178,0.163672021086473,0.165253349314772,0.168164833514363,0.168164833514363,0.178101560032755,0.178101560032755,0.179031429796843,0.1850817681115,0.193847337393735]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>geneset<\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,4,5]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 4, 4, 3, ',', '.');\nDTWidget.formatRound(this, row, data, 5, 4, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="heatmap-of-significant-gene-sets" class="section level1">
<h1>Heatmap of significant gene-sets</h1>
<p>FDR cutoff &lt;= 0.001</p>
<img src="figure/GSEA_T0.Rmd/heatmap-1.png" width="672" style="display: block; margin: auto;" />
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
geneset
</th>
<th style="text-align:center;">
NGenes
</th>
<th style="text-align:center;">
Direction
</th>
<th style="text-align:center;">
PValue
</th>
<th style="text-align:center;">
FDR
</th>
<th style="text-align:center;">
time
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
enriched in monocytes (II) (M11.0)
</td>
<td style="text-align:center;">
173
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
putative targets of PAX3 (M89.0)
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M70.0)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M125)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M125)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T3
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M70.0)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T3
</td>
</tr>
<tr>
<td style="text-align:center;">
TLR and inflammatory signaling (M16)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in T cells (I) (M7.0)
</td>
<td style="text-align:center;">
51
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (I) (M7.2)
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M125)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M70.0)
</td>
<td style="text-align:center;">
17
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
T cell activation (I) (M7.1)
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
0.00e+00
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
putative targets of PAX3 (M89.1)
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
1.00e-07
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
AP-1 transcription factor network (M20)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
1.00e-07
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
cell cycle and growth arrest (M31)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
1.00e-07
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M70.1)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
1.00e-07
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M70.1)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
2.00e-07
</td>
<td style="text-align:center;">
T3
</td>
</tr>
<tr>
<td style="text-align:center;">
Monocyte surface signature (S4)
</td>
<td style="text-align:center;">
65
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
3.00e-07
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
growth factor induced, enriched in nuclear receptor subfamily 4 (M94)
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
1.30e-06
</td>
<td style="text-align:center;">
T4
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in NK cells (II) (M61.0)
</td>
<td style="text-align:center;">
23
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
0.0e+00
</td>
<td style="text-align:center;">
1.40e-06
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
T cell activation (II) (M7.3)
</td>
<td style="text-align:center;">
26
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.0e-07
</td>
<td style="text-align:center;">
3.00e-06
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
enriched in neutrophils (I) (M37.1)
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
1.0e-07
</td>
<td style="text-align:center;">
3.40e-06
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M70.1)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
2.0e-07
</td>
<td style="text-align:center;">
6.90e-06
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M186)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
1.0e-07
</td>
<td style="text-align:center;">
7.60e-06
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
myeloid cell enriched receptors and transporters (M4.3)
</td>
<td style="text-align:center;">
23
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
5.0e-07
</td>
<td style="text-align:center;">
1.22e-05
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M184.0)
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
2.0e-07
</td>
<td style="text-align:center;">
1.37e-05
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
E2F1 targets (Q3) (M10.0)
</td>
<td style="text-align:center;">
29
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
7.0e-07
</td>
<td style="text-align:center;">
1.60e-05
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M186)
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
7.0e-07
</td>
<td style="text-align:center;">
1.66e-05
</td>
<td style="text-align:center;">
T2
</td>
</tr>
<tr>
<td style="text-align:center;">
TBA (M184.1)
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
Up
</td>
<td style="text-align:center;">
4.0e-07
</td>
<td style="text-align:center;">
2.47e-05
</td>
<td style="text-align:center;">
T1
</td>
</tr>
<tr>
<td style="text-align:center;">
NK cell surface signature (S1)
</td>
<td style="text-align:center;">
41
</td>
<td style="text-align:center;">
Down
</td>
<td style="text-align:center;">
1.2e-06
</td>
<td style="text-align:center;">
2.57e-05
</td>
<td style="text-align:center;">
T2
</td>
</tr>
</tbody>
</table>
<div id="summary" class="section level2">
<h2>Summary</h2>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="text-align:center;">
visit
</th>
<th style="text-align:center;">
significant genesets
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
T1
</td>
<td style="text-align:center;">
86
</td>
</tr>
<tr>
<td style="text-align:center;">
T2
</td>
<td style="text-align:center;">
180
</td>
</tr>
<tr>
<td style="text-align:center;">
T3
</td>
<td style="text-align:center;">
46
</td>
</tr>
<tr>
<td style="text-align:center;">
T4
</td>
<td style="text-align:center;">
61
</td>
</tr>
<tr>
<td style="text-align:center;">
T5
</td>
<td style="text-align:center;">
199
</td>
</tr>
</tbody>
</table>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.5 LTS

Matrix products: default
BLAS/LAPACK: /app/easybuild/software/OpenBLAS/0.2.18-GCC-5.4.0-2.26-LAPACK-3.6.1/lib/libopenblas_prescottp-r0.2.18.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] heatmap3_1.1.1       magrittr_1.5         DT_0.5              
 [4] limma_3.36.1         RNASeqUtilities_1.1  here_0.1            
 [7] data.table_1.12.0    forcats_0.3.0        stringr_1.4.0       
[10] dplyr_0.7.99.9000    purrr_0.2.5          readr_1.3.1         
[13] tidyr_0.8.0          tibble_2.0.1         ggplot2_3.1.0       
[16] tidyverse_1.2.1      GSEABase_1.42.0      graph_1.58.0        
[19] annotate_1.58.0      XML_3.98-1.11        AnnotationDbi_1.42.1
[22] IRanges_2.14.6       S4Vectors_0.18.1     Biobase_2.42.0      
[25] BiocGenerics_0.28.0  knitr_1.21          

loaded via a namespace (and not attached):
 [1] nlme_3.1-137       bitops_1.0-6       lubridate_1.7.4   
 [4] bit64_0.9-7        webshot_0.5.1      httr_1.4.0        
 [7] rprojroot_1.3-2    tools_3.5.0        backports_1.1.2   
[10] R6_2.4.0           DBI_1.0.0          lazyeval_0.2.1    
[13] colorspace_1.4-0   withr_2.1.2        tidyselect_0.2.5  
[16] gridExtra_2.3      bit_1.1-14         compiler_3.5.0    
[19] git2r_0.24.0       cli_1.0.1          rvest_0.3.2       
[22] xml2_1.2.0         scales_1.0.0       digest_0.6.18     
[25] rmarkdown_1.11     R.utils_2.7.0      pkgconfig_2.0.2   
[28] htmltools_0.3.6    htmlwidgets_1.3    rlang_0.3.1       
[31] readxl_1.1.0       rstudioapi_0.9.0   RSQLite_2.1.1     
[34] shiny_1.2.0        generics_0.0.2     jsonlite_1.6      
[37] crosstalk_1.0.0    R.oo_1.22.0        RCurl_1.95-4.10   
[40] kableExtra_1.0.1   Rcpp_1.0.0         munsell_0.5.0     
[43] R.methodsS3_1.7.1  stringi_1.3.1      whisker_0.3-2     
[46] yaml_2.2.0         plyr_1.8.4         grid_3.5.0        
[49] blob_1.1.1         promises_1.0.1     crayon_1.3.4      
[52] lattice_0.20-35    haven_2.0.0        hms_0.4.2         
[55] pillar_1.3.1       fastcluster_1.1.24 reshape2_1.4.3    
[58] glue_1.3.0         evaluate_0.13      modelr_0.1.1      
[61] httpuv_1.4.5.1     cellranger_1.1.0   gtable_0.2.0      
[64] assertthat_0.2.0   xfun_0.5           mime_0.6          
[67] xtable_1.8-3       broom_0.5.1        later_0.8.0       
[70] viridisLite_0.3.0  memoise_1.1.0      workflowr_1.1.1   </code></pre>
</div>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
