<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Carl Murie" />

<meta name="date" content="2018-11-26" />

<title>Quality Control</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">VaheyProject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Quality Control</h1>
<h4 class="author"><em>Carl Murie</em></h4>
<h4 class="date"><em>November 26, 2018</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-11-26</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20181008)</code> </summary></p>
<p>The command <code>set.seed(20181008)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> c3cdf7e </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    data/.Rhistory

Untracked files:
    Untracked:  code/processGEOFiles.R
    Untracked:  data/GPL570.soft
    Untracked:  data/GPL571.soft
    Untracked:  data/GSE18323-GPL570_series_matrix.txt.gz
    Untracked:  data/GSE18323-GPL571_series_matrix.txt.gz
    Untracked:  data/gpl_eset.Rds
    Untracked:  data/h.all.v5.0.symbols.gmt

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes. </details>
</li>
</ul>
<details> <summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
f9dfd7e
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-26
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
2cfa7e7
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-26
</td>
<td style="text-align:left;">
publish project
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
c0f65f8
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-20
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
64c4259
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-20
</td>
<td style="text-align:left;">
initial commit
</td>
</tr>
</tbody>
</table>
</ul>
<p></details></p>
<hr />
<div id="experimental-design" class="section level1">
<h1>Experimental Design</h1>
<p>Microarray gene expression assay applied to PBMC samples of subjects exposed to RTS,S/AS01 or RTS,S/AS02 vaccine.</p>
<p><strong>Microarrays</strong><br />
HGU133A 2.0 - 13515 genes (used for all T5 samples)<br />
HGU133 Plus 2.0 - 23521 genes (used for all non-T5 samples)</p>
<p>The genes in the HGU133A are a subset of the genes in the HG133 Plus chip. The intersection of the genes from the 2 chips were used in this analysis which resulted in 13515 genes.</p>
<p><strong>Factors</strong>:</p>
<ul>
<li><strong>visit</strong>
<ul>
<li>T0 - study entry (baseline)</li>
<li>T1 - day of third vaccination</li>
<li>T2 - 24h after T1</li>
<li>T3 - 72h after T1</li>
<li>T4 - 2 weeks after T1</li>
<li>T5 - 5 days after challenge</li>
</ul></li>
<li><strong>parasitemia</strong> - response to malaria challenge
<ul>
<li>Protected - vaccinated and protected from challenge<br />
</li>
<li>Delay - vaccinated and delayed parasitemia<br />
</li>
<li>NonProtected - vaccinated and not protected from challenge<br />
</li>
<li>InfectivityControl - not vaccinated and challenged<br />
</li>
<li>Unknown - unknown</li>
</ul></li>
<li><strong>disease</strong> - identical to parasitemia but Delay was changed to Protected</li>
</ul>
<div id="sample-sizes" class="section level2">
<h2>Sample sizes</h2>
<p>There are 12 samples that are labeled InfectivityControl. They do not have time-points</p>
<table class="table table-striped table-hover table-condensed" style="width: auto !important; ">
<thead>
<tr>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden" colspan="1">
</th>
<th style="border-bottom:hidden; padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="5">
<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px;">
parasitemia
</div>
</th>
</tr>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
total
</th>
<th style="text-align:center;">
Delay
</th>
<th style="text-align:center;">
InfectivityControl
</th>
<th style="text-align:center;">
NonProtected
</th>
<th style="text-align:center;">
Protected
</th>
<th style="text-align:center;">
Unknown
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
T0
</td>
<td style="text-align:center;">
36
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
T1
</td>
<td style="text-align:center;">
44
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
T2
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
9
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
T3
</td>
<td style="text-align:center;">
43
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
14
</td>
<td style="text-align:center;">
12
</td>
<td style="text-align:center;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
T4
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
T5
</td>
<td style="text-align:center;">
39
</td>
<td style="text-align:center;">
11
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
15
</td>
<td style="text-align:center;">
13
</td>
<td style="text-align:center;">
0
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="quality-control" class="section level1">
<h1>Quality Control</h1>
<div id="distribution" class="section level2">
<h2>Distribution</h2>
<p>There is a clear distinction between the T5 samples and all others, probably due to the use of different chips. <img src="figure/QC.Rmd/distributions-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of distributions-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
f9dfd7e
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-26
</td>
</tr>
<tr>
<td style="text-align:left;">
c0f65f8
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-20
</td>
</tr>
</tbody>
</table>
<p></details></p>
</div>
<div id="mds" class="section level2">
<h2>MDS</h2>
<div id="chip" class="section level3">
<h3>Chip</h3>
<p>The T5 samples are a clear clustered group which would motivate the use of chip type as a covariate in the analysis.</p>
<p><img src="figure/QC.Rmd/chip-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of chip-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
f9dfd7e
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-26
</td>
</tr>
<tr>
<td style="text-align:left;">
c0f65f8
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-20
</td>
</tr>
</tbody>
</table>
<p></details></p>
</div>
<div id="visit" class="section level3">
<h3>Visit</h3>
<p><img src="figure/QC.Rmd/visit-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of visit-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
f9dfd7e
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-26
</td>
</tr>
<tr>
<td style="text-align:left;">
c0f65f8
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-20
</td>
</tr>
</tbody>
</table>
<p></details></p>
</div>
<div id="parasitemia" class="section level3">
<h3>Parasitemia</h3>
<p><img src="figure/QC.Rmd/parasitemia-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of parasitemia-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
f9dfd7e
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-26
</td>
</tr>
<tr>
<td style="text-align:left;">
c0f65f8
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-20
</td>
</tr>
</tbody>
</table>
<p></details></p>
</div>
<div id="disease" class="section level3">
<h3>Disease</h3>
<p><img src="figure/QC.Rmd/disease-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of disease-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
f9dfd7e
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-26
</td>
</tr>
<tr>
<td style="text-align:left;">
c0f65f8
</td>
<td style="text-align:left;">
carlmurie
</td>
<td style="text-align:left;">
2018-11-20
</td>
</tr>
</tbody>
</table>
<p></details></p>
</div>
<div id="chip-residuals-of-a-linear-model-with-chip-as-the-independent-variable" class="section level3">
<h3>Chip: residuals of a linear model with chip as the independent variable</h3>
</div>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.5 LTS

Matrix products: default
BLAS/LAPACK: /app/easybuild/software/OpenBLAS/0.2.18-GCC-5.4.0-2.26-LAPACK-3.6.1/lib/libopenblas_prescottp-r0.2.18.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] parallel  stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] limma_3.36.1        RNASeqUtilities_1.1 here_0.1           
 [4] data.table_1.11.8   forcats_0.3.0       stringr_1.3.1      
 [7] dplyr_0.7.99.9000   purrr_0.2.5         readr_1.1.1        
[10] tidyr_0.8.0         tibble_1.4.99.9005  ggplot2_3.0.0      
[13] tidyverse_1.2.1     GEOquery_2.48.0     Biobase_2.40.0     
[16] BiocGenerics_0.26.0 knitr_1.20         

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.0        lubridate_1.7.4   lattice_0.20-35  
 [4] assertthat_0.2.0  rprojroot_1.3-2   digest_0.6.18    
 [7] R6_2.3.0          cellranger_1.1.0  plyr_1.8.4       
[10] backports_1.1.2   evaluate_0.11     highr_0.6        
[13] httr_1.3.1        pillar_1.3.0.9001 rlang_0.3.0.9000 
[16] lazyeval_0.2.1    readxl_1.1.0      rstudioapi_0.8   
[19] whisker_0.3-2     R.utils_2.7.0     R.oo_1.22.0      
[22] DT_0.4            rmarkdown_1.10    labeling_0.3     
[25] htmlwidgets_1.2   munsell_0.5.0     broom_0.5.0      
[28] compiler_3.5.0    modelr_0.1.1      pkgconfig_2.0.2  
[31] htmltools_0.3.6   tidyselect_0.2.5  gridExtra_2.3    
[34] workflowr_1.1.1   viridisLite_0.3.0 crayon_1.3.4     
[37] withr_2.1.2       R.methodsS3_1.7.1 grid_3.5.0       
[40] nlme_3.1-137      jsonlite_1.5      gtable_0.2.0     
[43] git2r_0.23.0      magrittr_1.5      scales_1.0.0     
[46] cli_1.0.1         stringi_1.2.4     reshape2_1.4.3   
[49] xml2_1.2.0        kableExtra_0.9.0  tools_3.5.0      
[52] glue_1.3.0        hms_0.4.2         yaml_2.2.0       
[55] colorspace_1.3-2  rvest_0.3.2       haven_1.1.2      </code></pre>
</div>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
