---
title: "Protected vs Non-Protected"
author: "Carl Murie and Raphael Gottardo"
date: "April 29, 2019"
output: html_document
---


```{r load-packages, echo=FALSE, warning=FALSE, message=FALSE}
library(knitr)
knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message=FALSE)

library(GSEABase)
library(Biobase)
library(tidyverse)
library(data.table)
library(here)
library(RNASeqUtilities)
library(limma)
library(DT)
library(magrittr)
library(heatmap3)
```

```{r read}
fdr_cut <- 0.2
min_gene_set <- 5
form <- "~disease"        ## formula used in camera

vahey_eset <- readRDS(here("data", "gpl_eset.Rds"))

all_res <- NULL  ## store significant gene-sets across comparisons for heatmap

coef <- "diseaseNonProtected"
```

```{r gsea_setup}
## set up GSEA analysis
btm_gtm <- getGmt(here("data/BTM_for_GSEA_20131008.gmt"))
btm_ind <- ids2indices(geneIds(btm_gtm), rownames(vahey_eset))
btm_ind <- btm_ind[sapply(btm_ind, length) > min_gene_set]
```

# GSEA Analysis

The Blood Transcript Modules (BTM) from Shuzhao et al. Molecular signatures of antibody responses derived from a systems biological study of 5 human vaccines, 2014 were used for the GSEA analysis.
  
## Disease effect (NonProtected - Protected) for T0 to T5
  
* **visit**
    + T0 - study entry (baseline)
    + T1 - day of third vaccination
    + T2 - 24h after T1
    + T3 - 72h after T1
    + T4 - 2 weeks after T1
    + T5 - 5 days after challenge  
  
* **disease** - response to malaria challenge
    + Protected - vaccinated and protected from challenge  
    + NonProtected - vaccinated and not protected from challenge  

A FDR cutoff of <= `r fdr_cut` was used.  
  
linear model: `r form`  
coefficient: `r coef`  
  
```{r gsea1}

## filter data for relevant time points and disease status

pData(vahey_eset) %>% 
  filter(!is.na(visit)) %>%           ## remove controls
  filter(disease != "Unknown") %$%
    geo_id ->
  sample_index

gsea_eset <- vahey_eset[,sample_index] 
pData(gsea_eset) %>%
  mutate(disease=factor(disease, levels=c("Protected", "NonProtected"))) %>%
  set_rownames(pData(gsea_eset)$geo_id) ->
  pData(gsea_eset)
```

## Sample size  {.tabset .tabset-fade .tabset-pills}   
  
```{r sample_size}
tab <- countSampleSizes(pData(gsea_eset), "visit", "disease")
headerKable(tab, labels="disease", levels=2)
```
  
## Comparisons {.tabset .tabset-fade .tabset-pills}   
  
### T0  

```{r T0vsT1}

## get subset
pData(gsea_eset) %>% 
  filter(visit == "T0") %$%     
    geo_id ->
  visit_index

gsea_sub <- gsea_eset[,visit_index]

design1 <- model.matrix(formula(form), 
                        data=pData(gsea_sub))
v1 <- voom(exprs(gsea_sub), 
           design=design1)
btm_camera <- as_tibble(camera(v1, 
                               btm_ind, 
                               design1, 
                               contrast=coef),
                        rownames="geneset")
num_sig <- sum(btm_camera$FDR <= fdr_cut)

btm_camera %>% 
  mutate(time = "T0") ->
all_res

btm_camera %>% 
  filter(FDR < fdr_cut) %>% 
  headerKable() %>%
  kableExtra::scroll_box(height="300px")
 
```

### T1  

```{r T1}

## get subset
pData(gsea_eset) %>% 
  filter(visit == "T1") %$%     
    geo_id ->
  visit_index

gsea_sub <- gsea_eset[,visit_index]

design1 <- model.matrix(formula(form), 
                        data=pData(gsea_sub))
v1 <- voom(exprs(gsea_sub), 
           design=design1)
btm_camera <- as_tibble(camera(v1, 
                               btm_ind, 
                               design1, 
                               contrast=coef),
                        rownames="geneset")
num_sig <- c(num_sig, sum(btm_camera$FDR <= fdr_cut))

btm_camera %>% 
  mutate(time = "T1") %>%
  rbind(all_res) ->
  all_res

btm_camera %>% 
  filter(FDR < fdr_cut) %>% 
  headerKable() %>%
  kableExtra::scroll_box(height="300px")
```

### T2  

```{r T2}

## get subset
pData(gsea_eset) %>% 
  filter(visit == "T2") %$%     
    geo_id ->
  visit_index

gsea_sub <- gsea_eset[,visit_index]

design1 <- model.matrix(formula(form), 
                        data=pData(gsea_sub))
v1 <- voom(exprs(gsea_sub), 
           design=design1)
btm_camera <- as_tibble(camera(v1, 
                               btm_ind, 
                               design1, 
                               contrast=coef),
                        rownames="geneset")
num_sig <- c(num_sig, sum(btm_camera$FDR <= fdr_cut))

btm_camera %>% 
  mutate(time = "T2") %>%
  rbind(all_res) ->
  all_res


btm_camera %>% 
  filter(FDR < fdr_cut) %>% 
  headerKable() %>%
  kableExtra::scroll_box(height="300px")
```

### T3  

```{r T3}

## get subset
pData(gsea_eset) %>% 
  filter(visit == "T3") %$%     
    geo_id ->
  visit_index

gsea_sub <- gsea_eset[,visit_index]

design1 <- model.matrix(formula(form), 
                        data=pData(gsea_sub))
v1 <- voom(exprs(gsea_sub), 
           design=design1)
btm_camera <- as_tibble(camera(v1, 
                               btm_ind, 
                               design1, 
                               contrast=coef),
                        rownames="geneset")
num_sig <- c(num_sig, sum(btm_camera$FDR <= fdr_cut))

btm_camera %>% 
  mutate(time = "T3") %>%
  rbind(all_res) ->
  all_res


btm_camera %>% 
  filter(FDR < fdr_cut) %>% 
  headerKable() %>%
  kableExtra::scroll_box(height="300px")
```


### T4  

```{r T4}

## get subset
pData(gsea_eset) %>% 
  filter(visit == "T4") %$%     
    geo_id ->
  visit_index

gsea_sub <- gsea_eset[,visit_index]

design1 <- model.matrix(formula(form), 
                        data=pData(gsea_sub))
v1 <- voom(exprs(gsea_sub), 
           design=design1)
btm_camera <- as_tibble(camera(v1, 
                               btm_ind, 
                               design1, 
                               contrast=coef),
                        rownames="geneset")
num_sig <- c(num_sig, sum(btm_camera$FDR <= fdr_cut))

btm_camera %>% 
  mutate(time = "T4") %>%
  rbind(all_res) ->
  all_res


btm_camera %>% 
  filter(FDR < fdr_cut) %>% 
  headerKable() %>%
  kableExtra::scroll_box(height="300px")
```

### T5  

```{r T5}

## get subset
pData(gsea_eset) %>% 
  filter(visit == "T5") %$%     
    geo_id ->
  visit_index

gsea_sub <- gsea_eset[,visit_index]

design1 <- model.matrix(formula(form), 
                        data=pData(gsea_sub))
v1 <- voom(exprs(gsea_sub), 
           design=design1)
btm_camera <- as_tibble(camera(v1, 
                               btm_ind, 
                               design1, 
                               contrast=coef),
                        rownames="geneset")
num_sig <- c(num_sig, sum(btm_camera$FDR <= fdr_cut))

btm_camera %>% 
  mutate(time = "T5") %>%
  rbind(all_res) ->
  all_res


btm_camera %>% 
  filter(FDR < fdr_cut) %>% 
  headerKable() %>%
  kableExtra::scroll_box(height="300px")
```
 
 
```{r save}
## save GSEA output
write_csv(all_res, path=here("output/Vahey_GSEA_disease.csv"))
```
 
# Heatmap of significant gene-sets

```{r heat_params}
new_fdr <- 0.01
```

FDR cutoff <= `r new_fdr`  
  
```{r heatmap}

all_res %>% 
  filter(FDR < new_fdr) %>%
  filter(!str_detect(geneset, "TBA")) %$%
  geneset ->
  sigs

all_res %>%
  filter(geneset %in% sigs) %>%
  mutate(sign = as.numeric(plyr::mapvalues(Direction, 
                                           from=c("Up", "Down"), 
                                           to=c(1, -1)))) %>%
  mutate(FDR_sign = -log10(FDR)*sign) ->
  genies
  
dcast(genies, geneset~time, value.var="FDR_sign") %>%
  as_tibble() %>%
  column_to_rownames("geneset") %>%
  as.matrix() ->
  heat_mat

heatmap3(heat_mat, Rowv=TRUE, Colv=NA, scale="none", balanceColor=TRUE, 
         cexRow=0.65 ,showRowDendro=FALSE, 
         col=colorRampPalette(c("firebrick3", "white", "navy"))(1024))

genies %>%
  arrange(FDR) %>%
  select(-c(sign, FDR_sign)) %>%
  head(30) %>%
  headerKable() %>%
  kableExtra::scroll_box(height="300px")
  
```
  
  
## Summary  
  
```{r summary}
tibble(visit=sort(unique(gsea_eset$visit)), "significant genesets"=num_sig) %>%
  headerKable()

```
